<!DOCTYPE html>
<html>



<!--
!!	epitope.htm
!!-->


<!--
!!    MOE On-Line Manuals
!!    COPYRIGHT (C) CHEMICAL COMPUTING GROUP ULC.  ALL RIGHTS RESERVED.
!!-->


<head>
<meta http-equiv="x-ua-compatible" content="IE=9" />
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<link rel="stylesheet" type="text/css" href="../include/manstyle.css" />
  <link rel="icon" type="image/png" href="../images/ccgicon.png" />
  <link rel="shortcut icon" type="image/png" href="../images/ccgicon.png" />
<title>Epitope Analysis</title>
<meta name="keywords" content="panel: Epitope Analysis" />
</head>

<body>
<div class="center-page">

<!-- START MOE_HEADER --><div class="MOE_HEADER">
<a href="../index.htm"><img src="../images/ccglogo.png" /></a>
<div class="moeversion"></div>
Epitope Analysis
</div><!-- END MOE_HEADER -->
<div class="content">

<ul>
<li><a href="#Overview">Overview</a></li>
<li><a href="#Running">Running Epitope Analysis</a></li>
<li><a href="#Constraints">Applying Constraints</a></li>
<li><a href="#Viewing">Viewing Epitopes</a></li>
<li><a href="#DatabaseBrowser">Database Browser</a></li>
<li><a class="example" href="#Example1">Example: Interleukin-13</a></li>
<li><a href="#References">References</a></li>
<li><a href="#SeeAlso">See Also</a></li>
</ul>

<!-- OVERVIEW -->

<a id="Overview"></a>
<h1>Overview</h1>

<p>
The MOE <b>Epitope Analysis</b> module uses the results of a protein-protein
docking simulation to calculate the most likely residues to have a key
role in the protein-protein interface.  <b>Epitope Analysis</b> uses
<a href="plif.htm#ProteinPLIFs">Protein Interaction Fingerprints</a>
(PLIFs) calculated using
<a href="plif.htm#Surface">Surface Patch Contacts</a>
to cluster the docked poses of the "ligand" protein.  Clustering is
performed using single-link hierarchical agglomeration with the Tanimoto
similarity of the fingerprints as the distance metric.
</p><p>
Pose clusters are ranked using an ensemble free-energy score</p>
<!-- MathML
	F_i = -ln sum from{ j in i } exp(- %beta N_j S_j)
-->
<img src="epitope/eq1.png" class="center" />
<p>where <i>N</i> is the number of fingerprint bits in a pose <i>j</i> in
cluster <i>i</i>, <i>S</i> is the docking score of pose <i>j</i>, and the sum
is over all poses in cluster <i>i</i>.  The scaling factor &beta; corresponds
to an effective inverse temperature for calculating the cluster entropy.
Small values of &beta; will favor clusters with the most member poses and
poses with the most contacts whereas large values will favor contacts
in poses with the best docking scores.  The default value has been validated
to provide the best overall results on a diverse test set of protein
complexes.
</p><p>
Epitope residues are selected by picking the residues with the
highest-occupancy fingerprint bits in each cluster.  The population of
a bit is calculated using Boltzmann weighting</p>
<!-- MathML
	p_i = sum from{j} %rho_ij exp(- %beta S_j)
-->
<img src="epitope/eq2.png" class="center" />
<p>where the sum is over all poses in the cluster and &rho;<sub>ij</sub> is
equal to one if bit <i>j</i> is set in pose <i>i</i> and zero otherwise.
Fingerprint bits which correspond to strong interactions are selected
preferentially, followed by weak interactions.  Compact epitopes are
generated by applying a distance cut-off such that a new residue is only
added to an epitope if it is within the specified distance of an
existing residue.  The total number of residues in each epitope is
set to a fraction of the average number of fingerprint bits per pose
in the ensemble, such that depending on the size of the interface the
number of contacts will be correct on average.
</p><p>
The <b>Epitope Analysis</b> panel provides a convenient way to configure
and carry out the calculation, as well as a tool for browsing the
resulting epitopes and annotating the corresponding structures in MOE.
</p>

<!-- RUNNING -->

<a id="Running"></a>
<h1>Running Epitope Analysis</h1>

<p>
The <b>Epitope Analysis</b> panel can be opened from the
<a href="../mdb/db.htm">Database Viewer</a>
with <span class="menu">DBV | Compute | PLIF | Epitope</span> or from the
<a href="plif.htm#Using">PLIF Analyze</a>
panel by pressing the <b>Epitopes</b> button in the panel footer.
The panel will open initially as a panel in the <b>Database Viewer</b>
by default.  Press on the <b>Detach</b> button to
detach the panel as a separate window as shown below.  Press the
<b>Attach</b> button
to re-attach the panel to the <b>Database Viewer</b>.
</p>

<img src="epitope/open.png" alt="Epitope Analysis Panel" class="center" />

<p>To perform the calculation, select the database field containing the
docking scores (or another numerical field to be used in ranking clusters)
from the <b>Score</b> pulldown menu.  Check <b>Use Site Constraints</b>
to apply additional sequence-based scoring terms in the epitope analysis
as described in <a href="#Constraints">Applying Constraints</a> below.
Press the adjacent <b>Configure</b> icon to open the site configuration panel.
The number of sites in the current constraint specification will be indicated.
</p>

<p>For databases resulting from docking calculations there will be
additional fields in the database, usually called <em>mseq</em> and
<em>rseq</em>, which indicate the different pairs of proteins,
for example different antibodies docked with the same antigen.
In that case a <b>Receptor</b> field will be available in the Epitope
Analysis panel along with an option to select the integer field
to be used for indexing.  This enables restricting the calculation
to only those poses with the specified index.  Select zero or blank to
use all poses in the database as is the case by default.
</p>

<div class="floatright">
<img src="epitope/config.png"
alt="Epitope Configuration Panel" />
<p class="caption"> Epitope Configuration Panel </p>
</div>

<p>
The calculation parameters can be configured by pressing the <b>Configure</b>
(<img src="docking/gear_icon.png" alt="Configure Gear" class="icon" />)
icon in the upper right corner of the panel.
This will open up the Epitope Configuration panel with the following options.
</p>

<table class="titlebar">
<tr>
<th>Score Scale Factor</th>
<td>The value of the effective inverse temperature used in cluster scoring.</td>
</tr>

<tr>
<th>Cluster Threshold</th>
<td>
The cutoff value for the Tanimoto similarity above which two poses are
assigned to the same cluster.
</td>
</tr>

<tr>
<th>Epitope Fraction</th>
<td>
The fraction of the average number of fingerprint bits used to determine
the target number of residues in each epitope.
</td>
</tr>

<tr>
<th>Distance Cutoff</th>
<td>
The maximum distance (in &Aring;) between a residue in an epitope and
its closest neighbor in the same epitope.
</td>
</tr>

<tr>
<th>Epitope Limit</th>
<td>The maximum number of epitopes to calculate.</td>
</tr>
</table>

<p>Press <b>OK</b> in the Epitope Configuration panel to save the parameters
  or <b>Cancel</b> to discard the changes.  Press <b>Defaults</b> to set the
  parameters to standard values, or <b>Last</b> to restore the values used
  in the previous calculation (if applicable).  To run the calculation press
  <b>Calculate</b> in the lower left corner of the panel.
</p>

<p><span class="tip">Tip</span>
Selecting <b>Last</b> in the Configuration panel is a convenient way to inspect
the parameters which were used when re-opening a database which already contains
epitope data.
</p>

<!-- CONSTRAINTS -->

<a id="Constraints"></a>
<h1>Applying Constraints</h1>

<p>Epitope Analysis can be directed using experimental information such as
Hydrogen/Deuterium exchange spectrometry [Iacob, Huang].  Regions of the
protein which are more likely to be involved in the protein-protein
interactions can be specified in the site configuration panel shown below.
Each segment is given a type which corresponds to the weight parameter
used in the analysis (e.g. 1=strong, 2=weak, etc.)  Docking pose clusters
which contain interactions with a high weight will be ranked higher,
and those interactions will also be ranked higher among the residues
present within the cluster.</p>

<img src="epitope/site.png"
alt="Constraint Input Panel"
class="center" />

<p>Each row of the constraint table corresponds to a segment defined by a
chain ID, beginning and ending residues, and interaction type.  Multiple
segments can be assigned the same type.  Pressing
<img src="epitope/plus_icon.png" alt="Add Site Constraint" class="icon" />
brings up the option menu, allowing the user to enter Site Constraints.
Selecting an existing constraint allows the user to edit or remove it.
The option menu is described below:</p>

<table class="titlebar">
<tr>
<td><b>Residue Expression</b></td>
<td><p>
<img src="epitope/entry1.png" alt="Residue Expression pix" class="center" />
Create a new segment from the residues specified by a selection expression
applied to the specified chain.  Set the weight <b>Type</b> and press
the <b>Accept</b> icon to add the Constraint or the <b>Discard</b> icon
to discard the entry.</p></td></tr>

<tr>
<td><b>Selected&nbsp;in MOE</b></td>
<td><p>
<img src="epitope/entry2.png" alt="Selected in MOE pix" class="center" />
Create a new segment from the selected residues in MOE.  The chain ID and
residue numbers will be filled in automatically.  Set the weight <b>Type</b>
and press
the <b>Accept</b> icon to add the Constraint or the <b>Discard</b> icon
to discard the entry.  A description of the selection is provided.
</p></td></tr>

<tr>
<td><b>Load&nbsp;from File</b></td>
<td><p>
<img src="epitope/entry3.png" alt="Load from File pix" class="center" />
Load a site constraint from a file.  Enter the filename in the text field
or press the <b>Browse</b> icon to select the file.  The file format is
four space- or comma-separated values per line (duplicating the list format),
specifying the chain, first and last residues and the weight <b>Type</b>.
Press
the <b>Accept</b> icon to add the Constraint or the <b>Discard</b> icon
to discard the entry.
<p><span class="tip">Tip</span>
Cut-and-paste can be used to create a valid file for later use.</p>
</td></tr>

<tr>
<td><b>Edit</b></td>
<td><p>
<img src="epitope/entry4.png" alt="Edit Mode pix" class="center" />
Edit the selected row by entering new values directly as shown.  Press the
<b>Accept</b> icon to add the Constraint or the <b>Discard</b> icon
to discard the changes.  Press
<img src="epitope/minus_icon.png" alt="Add Site Constraint" class="icon" />
to delete the selected constraints.  Note that editing the residue
expression is disabled if multiple rows are selected.
</p></td></tr>
</table>

<p>Five <b>Types</b> are provided which can be used to create different
classes of constraints (strong, weak, etc.) with unique weights.  The
<span class="nobr"><b>Constraint Weights</b></span> can be specified
by entering a new value in the appropriate text box.
These are used
for selecting epitope residues and ranking pose clusters.  Larger values
correspond to stronger constraints.
</p>

<p>When the site configuration is complete, press <b>OK</b> or <b>Cancel</b>
to exit the configuration panel and return to the Epitope Analysis panel.
If <b>OK</b> was used to save the data, it can be used to recalculate the
epitope clusters by pressing <b>Calculate</b>.
The Site Constraints are saved in the MDB file and will
appear in the configuration panel if it is reopened from the same database.</p>
<p><span class="tip">Tip</span>
The site configuration panel can be used to inspect the constraints
and save the data at a later time.</p>

<!-- VIEWING -->

<a id="Viewing"></a>
<h1>Viewing Epitopes</h1>

<p>
After the calculation has been performed, the Epitope Analysis panel
will be populated
as shown.  One or more epitopes can be selected for display by clicking on
items in the list or by scrolling using the up and down arrow keys.  Other
navigation options correspond to those of the standard
<a href="../moe/hellomoe.htm#Listbox">Listbox</a>
with style&nbsp;2 multi-selection.
</p>

<img src="epitope/list.png"
alt="Epitope Analysis Table"
class="center" />

<p>
The bottom toolbar controls how the selected epitopes are displayed.
The protein sequences shown for the epitopes in the list correspond
to the structures in the database field selected in the <b>Database</b>
pulldown menu.  This should normally be the same as the field used
as the receptor during the fingerprint calculation.
</p>
<p>
Use the checkboxes to turn on or off showing the selected epitopes in
the main MOE window.  Either unchecking the checkboxes or deselecting
the epitopes will remove structures which were loaded from the
MOE database.
</p>

<table class="noborder">
  <tr>
    <th rowspan="2">Annotate</th>
    <th>MOE</th>
    <td>
      Show epitope residues mapped onto the structure in the MOE window.
      If the MOE window contains other protein chains besides those to be
      annotated, use the pulldown menu to specify which chains to include.
      This will apply a ribbon-only representation with the epitope residues
      highlighted in yellow.  Named sets (Epitope_1, etc.) will also be created
      for convenience in carrying out further operations on the epitope
      residues.  Pressing <b>?</b> will temporarily highlight the
      receptor that will be annotated.<br />
      <b>Note:</b> The protein sequences must be aligned to those of the
      receptors used in the docking and fingerprint calculations, otherwise
      the results may not be reliable.  For multi-chain receptors the chains
      should also appear in the correct order in the System Manager and
      Sequence Editor.<br />
      <b>Note:</b> This operation will not be performed if the chains to
      be annotated are in a different reference frame from those in the
      database.  In that case, perform the required superposition before
      annotating.
    </td>
  </tr>

  <tr>
    <th>Database</th>
    <td>
      Import the receptor structures from the MOE Database into the main
      MOE window.  The database field containing the receptor structures
      used in docking will be used automatically for annotation, however
      if the database contains multiple molecule-type fields a pulldown
      menu can be used to select the field in the MOE Database containing
      the structures to annotate.
      For each epitope, the first entry in the database for that epitope
      will be used.  The structures will have the same representation as
      described above for annotating a single structure, however the epitope
      sets will each be part of a different chain in this case.<br />
      <b>Note:</b> If the receptor structures for each epitope have a common
      backbone the imported structures will be superposed and not all
      annotations may be visible.
    </td>
  </tr>


  <tr>
    <th>Show Poses</th>
    <td colspan="2">
      Show the docked poses of the interacting protein in the MOE window.
      The database field containing the structures used as the "ligand"
      in docking will be shown automatically, however if the database
      contains multiple molecule-type fields a pulldown menu can be used
      to select the field in the MOE Database containing the poses to display.
      All entries in the database corresponding to the selected epitopes
      will be displayed, with a translucent "spaghetti" representation to
      permit viewing the ensemble of poses contributing to the epitope.
      Poses corresponding to different epitopes will be sorted into separate
      groups and tags to simplify manipulation in the System Manager.<br />
      <b>Note:</b>If the selected epitopes contain a large number of poses
      a warning will appear and the display in MOE will be limited to the
      first 50 poses matching the selection.  To view all poses (which may
      take some time), use Select Entries and import the structures into
      MOE from the Database Viewer.
    </td>
  </tr>
</table>

<p><span class="tip">Tip</span>
In order to retain the epitope annotations in MOE, close the Epitope Analysis
panel while the desired items are selected.  De-selecting all epitopes will
remove the annotations before closing.
</p>
<p>
If any problems are detected which might cause the Epitope Analysis
panel to display
incorrect or missing annotations for the indicated display mode and selection
a warning button will appear to the left of the bottom toolbar as shown.
</p>
<img src="epitope/warn.png" alt="Warning Icon" class="center" />
<p>To see an explanation of potential problems press the warning button to
bring up a Warning panel containing further details.  Not all warnings are
necessarily fatal, but particular attention should be paid to ensure that
the results are as expected.
</p><p>
Additional buttons in the panel footer allow the selected epitopes to be
displayed in other MOE applications.</p>

<table class="noborder">
<tr>
<th>Select in DBV</th>
<td>
Pressing this button will open the corresponding MOE database in a
<a href="../mdb/db.htm">Database Viewer</a>
if it is not already open.  All entries corresponding to the selected
epitopes in the Epitope Analysis panel will be selected in the Database Viewer.
This selection will be applied each time the button is pressed, but will
not have any further effect if the selections are changed explicitly in
either of the two panels until it is re-applied.<br />
<span class="tip">Tip</span>
In order to annotate all receptor structures corresponding to a given epitope,
first use <b>Select in DBV</b> to select the corresponding entries and then
use the <a href="../mdb/dbbrowse.htm#Browser">DBV Browser</a>
to view Selected Entries.  Select all chains already in MOE and choose
Unselected Chains in the Epitope Analysis panel.  The annotation will thus be
automatically applied to each browsed structure as it appears.
</td>
</tr>

<tr>
<th>PLIF Analyze</th>
<td>
Pressing this button will open the
<a href="plif.htm#Using">PLIF Analyze</a>
panel associated with the corresponding MOE database if it is not already
open.  This is equivalent to opening the MOE database and selecting
<span class="menu">DBV | Compute | PLIF | Analyze</span>.  Regardless of
which is opened first, if the PLIF Analyze and Epitope Analysis panels are
both open simultaneously the selections in the two panels will be synchronized
(unlike the case with the Database Viewer.)  Selecting in the Epitope
Analysis panel
will select all corresponding entries in the PLIF Analysis panel.
Conversely, the selection in the Epitope Analysis panel
will include all epitopes
for which any entries are selected in the PLIF Analysis panel, even if that
does not include all members of the epitope cluster.<br />
<span class="tip">Tip</span>
Selecting an entry in the PLIF Analysis panel and then clicking on the
selected epitope in the Epitope Analysis panel will select all entries in the
same cluster as the original selection.<!--Checking <b>Include Selected Only</b>
in the PLIF panel will then show only those entries.-->
</td>
</tr>
</table>

<a id="DatabaseBrowser"></a>
<h2>Database Browser</h2>

<p>The Epitope Analysis panel can be used to view previously-calculated
epitope analysis data.  When browsing the Summary Database from a <a
href="docking.htm#Protein-Protein_HPC">Protein-Protein Docking</a>
calculation, carried out for example on multiple pairs of input protein
structures, the Epitope Analysis panel is automatically integrated into
the Database Browser.
</p>

<p>To open the Database Browser plugin version of the Epitope Analysis panel,
open the docking Summary Database in a Database Viewer, then
select <span class="menu">DBV | File | Browse</span>.
The DBV Browser will show the following data in the header section:</p>

<table class="noborder">
  <tr>
    <th>Subject</th>
    <td>When set to <b>Epitope Analysis</b>, the text field will show the name
    of the docking output database corresponding to the current entry.</td>
  </tr>
  <tr>
    <th>Receptor</th>
    <td>Shows the value of <tt>rseq</tt> for the current entry.</td>
  </tr>
  <tr>
    <th>Ligand</th>
    <td>Shows the value of <tt>mseq</tt> for the current entry.</td>
  </tr>
  <tr>
    <th>Show DBV</th>
    <td>Press to open the database shown in the Subject in a separate
    Database Viewer.</td>
  </tr>
  <tr>
    <th>Select&nbsp;Entries</th>
    <td>Performs the same action as <b>Select in DBV</b> described above
    for the stand-alone viewer.</td>
  </tr>
  <tr>
    <th>Keep</th>
    <td>Press to retain the current annotation in MOE after closing the
    Database Browser.</td>
  </tr>
</table>

<p>In the Database Browser plugin, the Listbox and Annotate functions
are identical to the <a href="#Viewing">stand-alone Epitope Analysis
panel</a>, with one exception: since the Database Browser already uses
the keyboard arrow keys, to navigate between database entries, the
keyboard navigation between epitopes is accomplished by holding the
Shift key while pressing the arrow keys.</p>

<!-- EXAMPLES -->

<a id="Example1"></a>
<h1>Example: Interleukin-13</h1>

<div class="example">
<ol>
<li>
<p>This example makes use of the results of docking Interleukin-13
  to the CNTO607 Fab antibody domain (PDB: 3G6D).  Load the structure
  into MOE.
  <p class="blockquote">
  <span class="menu">MOE | File | Open | <tt>$MOE/sample/mol/3g6d.pdb.gz</tt></span>
  </p>
  View the antibody by showing the ribbons only and coloring the CDR loops.
  <p class="blockquote">
  <span class="menu">MOE | RHS | Hide | All Atoms</span><br />
  <span class="menu">MOE | Protein | Annotate | Antibody | CCG</span>
  </p>
  </li>

  <li>
  <p>Open the MOE database in a Database Viewer.
  <p class="blockquote">
  <span class="menu">MOE | File | Open | <tt>$MOE/sample/mol/3g6d_dock.mdb</tt></span>
  </p>
  Make a local working copy.
  <p class="blockquote">
  <span class="menu">DBV | File | Save As | <tt>temp.mdb</tt></span>
  </p>
  Open the Epitope Analysis panel.
  <p class="blockquote">
  <span class="menu">DBV | Compute | PLIF | Epitope</span>
  </p>
  </li>

<li>
<p>Notice that in this example the top cluster contains most of the top poses.
To generate smaller clusters, repeat the calculation with a higher cutoff value.
First, press the gear icon in the upper right corner and change the value of
<b>Cluster Threshold</b> from 0.4 to 0.5.  Press <b>OK</b> to save the change
and then Press <b>Calculate</b> at the bottom left corner of the Epitope
Analysis panel.  The epitope list should be repopulated with a new set of
epitope clusters.  Select the first row in the Epitope Browser.</p>
</li>

<li>
<img src="epitope/3g6d.jpg" alt="Interleukin-13 Epitope" class="floatright"/>

<p>Color the Interleukin-13 ribbon by selecting MOE in the <b>Annotate</b>
control and selecting <b>Selected Chains</b> in the adjacent pulldown
menu.  Activate the receptor annotation by checking the <b>Annotate</b>
checkbox.  Because no chains have been selected at this point nothing
will happen and the warning icon will appear to the left.  Open the
Sequence Editor (<span class="menu">MOE | SEQ</span>) and select the
chain <i>3G6D.A</i>.  The ribbon should now be colored, with the
epitope residues highlighted in yellow.  Now close the Epitope Analysis
panel by pressing <b>Close</b> at the bottom right.
</p></li>

<li>
<p>To view the interacting protein sidechains, open the MOE System Manager
(<span class="menu">MOE | RHS | System</span>) and expand the <b>Sets</b>
list.  Select the epitope atoms by clicking the item labeled <i>Epitope_1</i>
and display them.
<p class="blockquote">
<span class="menu">MOE | RHS | Show | Selected</span>
</p>
<p>Deselect the atoms by clicking anywhere in the MOE window.  The MOE window
should now appear similar to the image.</p>
</li>

<li><p>Return to the original epitope data by repeating Step.&nbsp;2.
Alternatively, reopen the Epitope Panel and press the gear icon to
raise the configuration pop-up.  Press <b>Defaults</b>, then <b>OK</b> to
save the values, and then <b>Calculate</b> to refresh the Epitope Browser.
</p></li>

<li><p>To see the effects of the site constraints, select residues A:85-91
in the Sequence Editor.  The highlighted loop is not part of the
top-ranked clusters, but we can see what would happen if it was known
to be involved.  Check <b>Use Site Constraints</b> in the Epitope Panel
and press the adjacent <b>Configure</b> icon.  This opens the
<span class="nobr"><b>Epitope Site Constraints panel</b></span>.
Press the
<img src="epitope/plus_icon.png" alt="Ex AddSite Constraint" class="icon" />
icon, change the <span class="nobr"><b>Input Method</b></span> to
<span class="nobr"><b>Selected in MOE</b></span> and press the <b>Accept</b>
icon to add the Site Constraint.  Increase the weight
for <b>Type</b> 1 by entering 200 in the text box and press <b>OK</b> to
save the Constraint and close the panel.</p>
</li>

<li><p>Press <b>Calculate</b> to apply the constraints.  Notice that now
the poses which contain interactions in the selected loop are ranked first
in the Epitope Browser.  Select the top three epitopes and then open the
PLIF panel by pressing <b>PLIF Analyze...</b>.  The top poses are selected,
and it can be seen that they are the only entries with interactions for
residues 90 and 91, in the selected loop.</p>
</li>

</ol>

</div>

<a id="References"></a>
<h1>References</h1>
  <table class="ref">
    <tr>
      <td>[Iacob]</td>
      <td>Iacob, Roxana E; Krystek, Stanley; Huang, Richard; Wei, Hui;
      Tao, Li; Lin, Zheng; Morin, Paul; Doyle, Michael; Tymiak, Adrienne A;
      Engen, John R; Chen, Guodong. (2015). Hydrogen/Deuterium Exchange Mass
      Spectrometry Applied to IL-23 Interaction Characteristics: Potential
      Impact for Therapeutics. Expert review of proteomics. 12. 1-11.</td>
    </tr>
    <tr>
      <td>[Huang]</td>
      <td>Huang, Richard; Krystek, Stanley; Felix, Nathan; Graziano,
      Robert F; Srinivasan, Mohan; Pashine, Achal; Chen, Guodong. (2017).
      Hydrogen/deuterium exchange mass spectrometry and computational modeling
      reveal a discontinuous epitope of an antibody/TL1A Interaction.
      mAbs. 10. 1-9.</td>
    </tr>
  </table>

<a id="SeeAlso"></a>
<h1>See Also</h1>
<p>
  <a href="docking.htm#Protein_Method">Protein-Protein Docking</a><br />
  <a href="plif.htm">Protein Ligand Interaction Fingerprints</a><br />
  <a href="surfmap.htm#ProteinPatches">Protein Patches</a><br />
</p>

<!-- START MOE_FOOTER --><div class="MOE_FOOTER">
<img src="../images/ccgicon.png" />
<a href="../index.htm"></a>
<a href="../legal.htm"></a> &copy; <span class="versionyear"></span>
<a href="http://www.chemcomp.com"></a>.  All rights reserved.
<br /><a href="mailto:info@chemcomp.com"></a>
</div><!-- END MOE_FOOTER -->

</div></div>
</body>
</html>

