<!DOCTYPE html>
<html>

<!--
!!    MOE On-Line Manuals
!!    COPYRIGHT (C) CHEMICAL COMPUTING GROUP ULC.  ALL RIGHTS RESERVED.
!!-->
<head>
  <meta http-equiv="x-ua-compatible" content="IE=9" />
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <link rel="stylesheet" type="text/css" href="../include/manstyle.css" />
  <link rel="icon" type="image/png" href="../images/ccgicon.png" />
  <link rel="shortcut icon" type="image/png" href="../images/ccgicon.png" />
  <title>Protein Ligand Interaction Fingerprints</title>
  <meta name="keywords" content="panel: PLIF Calculate" />
  <meta name="keywords" content="panel: PLIF Analyze" />
</head>
<body>
<div class="center-page">
  <!-- START MOE_HEADER -->
  <div class="MOE_HEADER">
    <a href="../index.htm"><img src="../images/ccglogo.png" /></a>
    <a href="../index.htm"><div class="moeversion"></div></a>
    Protein Ligand Interaction Fingerprints
  </div>
<!-- END MOE_HEADER -->
<div class="content">
  <!-- <div id="useMathJax"></div> -->
  <ul>
    <li><a href="#Overview">Overview</a>
      <ul>
        <li><a href="#Potential">Potential Contacts</a></li>
        <li><a href="#Surface">Surface Contacts</a></li>
      </ul>
    </li>
    <li>
      <a href="#Generate">Generating Fingerprints</a>
      <ul>
        <li><a href="#GeneralOpts">General Options</a></li>
        <li><a href="#LigandPLIFs">Protein-Ligand Fingerprints</a></li>
        <li><a href="#ProteinPLIFs">Protein-Protein Fingerprints</a></li>
      </ul>
    </li>
    <li>
      <a href="#Using">Using Fingerprints</a>
      <ul>
        <li><a href="#UsingBarcodes">Barcode Display</a></li>
        <li><a href="#UsingPopulation">Population Display</a></li>
        <li><a href="#UsingSignificance">Significance Display</a></li>
        <li><a href="#UsingLigand">Ligand Display</a></li>
        <li><a href="#UsingFieldSel">Field Selection</a></li>
        <li><a href="#UsingFingSel">Fingerprint Selection</a></li>
        <li><a href="#UsingSelByFing">Entry Selection by Fingerprint</a></li>
      </ul>
    </li>
    <li>
      <a href="#Tools">Fingerprint Tools</a>
      <ul>
        <li><a href="#ToolsBitSel">Bit Selector</a></li>
        <li><a href="#ToolsPoseSel">Pose Selector</a></li>
        <li><a href="#ToolsQuery">Query Generator</a></li>
        <li><a href="#ToolsSimil">Similarity</a></li>
      </ul>
    </li>
  </ul>
  <!-- ============================== OVERVIEW ============================= -->
<a id="Overview"></a>
<h1>Overview</h1>

<p>The Protein Ligand Interaction Fingerprints (PLIF) tool is a method for
  summarizing the interactions between ligands and proteins using a fingerprint
  scheme. Interactions such as hydrogen bonds, ionic interactions and surface
  contacts are classified according to the residue of origin, and built into a
  fingerprint scheme which is representative of a given database of
  protein-ligand complexes.</p>

<p>The input data for PLIF can be from a variety of sources, which most
  commonly include X-ray crystal structures and docking results. Using
  fingerprints to collectively represent protein-ligand interactions for a
  large database is an effective way of dealing with databases which are noisy
  and error-prone due to the many difficulties involved in modeling ligands
  bound to proteins.</p>

<p>Fingerprints generated using PLIF are compatible with other fingerprint
  tools found in MOE. Standard fingerprint tools such as clustering and diverse
  subsets can be applied to fingerprints generated by PLIF. There is a
  specialized visualization interface which is designed to take into account
  the specific structural meaning of each fingerprint bit.</p>

<p>In order to classify interaction fingerprints, each amino acid residue of
  a protein is broken down into categories. The general scheme is illustrated
  below for a few sample interactions:</p>
  <div align="center"><img src="plif/scheme.png" /></div>

<p>There are two categories of interaction in which a residue may participate:
  potential (energy-based) contacts and surface (patch) contacts.  For potential
  contacts the value is that of the strongest interaction between any pair of
  atoms in the residue and ligand, whereas for surface contacts the value is the
  total contact area of each type between a residue and the other molecule.</p>

<p>If no interactions of a particular category are found, or none pass the
  thresholds, no bits are set for that category. If the strongest interaction
  passes the lower interaction threshold, the low order fingerprint bit is set.
  If the strongest interaction passes the higher interaction threshold, then
  the low order and high order bits are both set. Therefore the bit patterns
  for each category can take on values of 00, 10 or 11, respectively.</p>

<p>Generally speaking, the potential contacts are used to characterize the
  binding of a small molecule in a well-defined binding site, whereas surface
  contacts are used to characterize protein-protein interfaces.
  Thresholds for each class of interaction are based on energy values or
  surface area respectively.  The default values are shown in the following
  sections.</p>

<a id="Potential"></a><h2>Potential Contacts</h2>

<p>There are currently nine types of potential interactions in which a residue
  may participate:
  sidechain hydrogen bonds (donor or acceptor), backbone hydrogen bonds (donor or
  acceptor), solvent hydrogen bonds (donor or acceptor), ionic interactions,
  metal binding interactions and &pi; interactions.</p>

<a id="Surface"></a><h2>Surface Contacts</h2>

<p>Surface contact interactions are determined by calculating the solvent
  exposed surface area of each protein chain and performing a
  <a href="../proteins/propatch.htm">Patch Analysis</a> calculation for each.
  For each residue, the total contact area with the other surface is calculated
  as well as the Protein Patch types of each surface in contact.  The contacts
  are divided into five categories according to the patches involved:
  hydrophobic (both surfaces are in hydrophobic patches), charged (either one
  of the surfaces is a charged patch), partial hydrophobic (one surface is
  hydrophobic, the other has no patch type), other (neither surface has
  a patch type), and total (the sum of all contact area).  For all categories
  other than the total, an additional threshold is applied to the fraction
  of the total contact represented by a specific patch contact.  For example,
  a hydrophobic contact is defined as being at least 8 &Aring;<sup>2</sup> and
  also at least 50% of the total contact area of the residue.  This
  requirement helps avoid double-counting contacts consisting of parts of
  multiple patches.</p>

  <!-- ============================== GENERATE ============================= -->
<a id="Generate"></a>
<h1>Generating Fingerprints</h1>

<p>Before PLIF can be used, it is necessary to assemble one or more proteins
  to serve as the receptor species, and some number of ligands or proteins
  with bound conformations. There are three main ways to go about this:</p>
  <ol>
    <li>Load a receptor into the MOE main window, and provide a database which
    contains a molecule-type field for which each entry contains a single ligand
    or protein pose, the coordinates of which correspond to its orientation within
    the active site.</li>
    <li>Provide a database which has a molecule-type field which contains both the
    ligand and the receptor together in each entry.  In this case the receptor and
    ligand must be identified by their respective Residue Class as indicated in
    the <a href="../moe/gui/system.htm#GroupControls">System Manager</a>.</li>
    <li>Provide a database which has two separate molecule-type fields, one for
    the receptor and one for the ligand, which taken together correspond to a
    bound complex.  In the case where the receptor has been stored in the
    database environmment (e.g. from rigid-receptor docking or a previous
    PLIF calculation as in method 1) the integer field <em>Rec_index</em>
    can be used as the receptor.</li>
  </ol>

<p>The first method is most commonly applicable when the source data comes
  from rigid-receptor docking results, in which a single protein structure
  applies to many poses of many ligands for which bound conformations have been
  calculated. The second method is most commonly applicable when the source
  data comes from a collection of crystal structures, for each of which the
  receptor is similar, but not necessarily identical.  The third method would
  apply to flexible-receptor (induced fit) docking results where the refined
  receptor structures are not identical, or to a database of protein-protein
  complexes.  Note that for the latter two cases the receptor structures are
  assumed to be aligned and have a common residue-numbering scheme.</p>

<p>Open the MOE database, and select
  <b>DBV&nbsp;|&nbsp;Compute&nbsp;|&nbsp;PLIF&nbsp;|&nbsp;Generate</b>. The
  following panel will appear:</p>
  <img src="plif/gen_lig.png" class="center"/>

<p>The default options are configured to be reasonable for a database of
  bound poses which are assumed to be complementary to a receptor which is
  currently loaded in MOE.</p>
<p>The <b>Fingerprint</b> selection provides a choice between the two
  fingerprint modes: <a href="#LigandPLIFs"><b>Ligand</b></a> or
  <a href="#LigandPLIFs"><b>Protein</b></a>.  The image shown
  above corresponds to the default Ligand mode.  The fingerprint selection
  switches the display of fingerprint bits between potential and surface
  interactions as described above, as well as setting appropriate default
  values for certain options.  The Fingerprint Generation calculation
  will be carried out using the set of cutoffs which are visible in the
  panel when the calculation is launched.  The following image shows the
  default options that are displayed in Protein mode.</p>

  <div align="center"><img src="plif/gen_prot.png"></div>

<a id="GeneralOpts"></a><h2>General Options</h2>
<p>The following options determine how the raw interactions are calculated
  and apply to either Ligand or Protein modes of the panel.</p>

  <table class="noborder">
    <tr>
      <td><b>Receptor<br />
      Ligand</b></td>
      <td>
        For each entry in the underlying database, the preparation step which
        calculates the raw interactions needs both a ligand and a receptor
        definition. For databases which have many poses which are relative to a
        single receptor, e.g. docking runs, the receptor can be specified as
        the <em>Rec_index</em> field.  Otherwise, the receptor can be loaded
        into MOE (using the DBV docking browser or from another source)
        and it must apply to all the entries which are being
        processed during preparation.  For databases of complexes where the
        receptor varies for each entry, it is necessary to store the receptor
        for each of the bound ligands within the same molecule field. For the
        <b>Receptor</b>, select <b>MOE</b> to use a currently loaded protein as
        the receptor for all database entries for which interactions are to be
        calculated. Select <b>Database</b> if the protein and ligand atoms are
        both stored in the database.
        <ol>
          <li>When <b>MOE</b> is selected, a choice of atom sets to use for the
          receptor is offered:
            <ul>
              <li>Receptor Atoms</li>
              <li>Receptor+Solvent</li>
              <li>Selected Atoms</li>
              <li>Selected Residues</li>
              <li>Selected Chains</li>
            </ul>

<img src="plif/prosource1.png" class="center" />

<p>The default selection is <tt>Receptor+Solvent</tt>. For this
            setting, it is important to ensure that just one protein is loaded,
            and that all of the ligands in the underlying database have 3D
            coordinates relative to the active site of this protein.</p>

<p>For <b>Ligand</b>, one of the molecule fields in the database
            must be selected. This field is not modified by the generation of
            interaction data or fingerprints.</p>
          </li>
	  <li>If the receptor type is changed to <b>Database</b>, then a
	  molecule-type field (to immediate right) must be selected, or in the
	  case where the database was created from rigid-body docking results
	  and the receptor structure is saved in the database using the special
	  field <tt>Rec_index</tt>, this field can also be
	  selected as the receptor.

<img src="plif/prosource2.png" class="center" />

<p>When the receptor is a molecule-type field in the database, the <b>Ligand</b>
	  field must also be specified, and is indicated as a text box.  There are three
	  possible situations depending on the choice of ligand field.</p>
	  <ul>
	    <li>The ligand field is the same as the receptor.  In this case the receptor
	    and ligand chains will be determined by their residue class.</li>
	    <li>The ligand field does not exist.  In this case the receptor and ligand
	    are determined as above, but the ligand extracted from the receptor field
	    will be written to the new ligand field.</li>
	    <li>The ligand field is a different molecule-type field from the receptor.
	    In this case the chains in the two fields will be used as specified for
	    the receptor and ligand.</li>
	  </ul>
          </li>
        </ol>
      </td>
    </tr>
    <tr>
      <td><b>Selected&nbsp;Only</b></td>
      <td>If this option is checked, only entries which are
      selected in the Database Viewer will be used to generate interactions or
      fingerprints.</td>
    </tr>
    <tr>
      <td><b>Interactions</b></td>
      <td>The Interactions checkboxes control whether or not
      different types of raw interactions are calculated.
        <ul>
          <li><b>Atomic:</b> Calculate atom-atom energy values.  These
            interactions are required to calculate the energy-based
	    fingerprints.</li>
          <li><b>Surface:</b> Calculate surface-surface contact areas.  These
            interactions are required to calculate the surface-patch
	    fingerprints.</li>
       </ul>
      By default, Atomic interactions are enabled in Ligand mode and Surface
      interactions in Protein mode.  It is possible however to calculate both
      types of interactions and thus be able to produce both types of fingerprint
      in subsequent steps.  In particular, if the total surface contact is used
      in Ligand mode the Surface interactions must be enabled.  In cases where
      the receptor proteins are all different (i.e. retrieved from a field in
      the database) calculating the surface patches for all entries can add
      significantly to the computational cost.</td>
    </tr>
    <tr>
      <td><b>Prepare</b></td>
      <td>This button initiates the calculation of selected raw
      interactions, which is the most resource-intensive part of generating
      fingerprints. The raw interactions are written to the underlying database,
      where they can be retrieved in order to construct fingerprints. If the
      definition of the <b>Protein</b> field indicates a molecule object in
      the MOE main window, it must be opened before this button is pressed.
      By default, all of the entries in the database are considered, and only
      those which have no interaction data are recalculated.</td>
    </tr>
    <tr>
      <td><b>Cutoffs</b></td>
      <td>By default, each interaction category can occupy as
      many as two interaction bits: low and high. It is possible to disable
      the generation of these bit types by manually setting the minimum
      threshold to an impossibly high value, but for convenience it is also
      possible to disable the two-bit scheme by un-checking <b>Weak</b> or
      <b>Strong</b>.</td>
    </tr>
    <tr>
      <td><b>Generate</b></td>
      <td>This button causes the generation of fingerprint bits,
      using the previously calculated raw interaction data. By default, the
      fingerprint scheme is rebuilt, and the contributions of all database
      entries are considered when constructing it. Once the fingerprints have
      been calculated and written to the database, the setup panel will
      automatically close, to be replaced by the results panel.</td>
    </tr>
    <tr>
      <td><b>Analyze</b></td>
      <td> If some or all protein ligand interaction fingerprints
      have already been calculated, this button can be used to open the
      results panel.</td>
    </tr>
  </table>

  <a id="LigandPLIFs"></a><h2>Protein-Ligand Fingerprints</h2>
  <p>The following options are available for fingerprint generation
    in Ligand mode.  If any of these fingerprint bits are selected the
    Atomic interactions must be calculated.</p>

  <table class="noborder">
    <tr>
      <td><b>Sidechain&nbsp;H-donor<br />
      H-acceptor</b></td>
      <td>Up to two fingerprint bits per residue can be assigned
      for interactions between protein sidechain atoms acting as H-bond donors,
      and two more fingerprint bits for protein sidechain atoms acting as
      H-bond acceptors. For convenience, the generation of either of these
      types of interactions can be disabled by un-checking the checkbox. The
      minimum thresholds are set by default to 0.5 kcal/mol for the low bit,
      and 1.5 kcal/mol for the high bit, and may be modified as necessary.</td>
    </tr>
    <tr>
      <td><b>Backbone&nbsp;H-donor<br />
      H-acceptor</b></td>
      <td>Up to two fingerprint bits per residue can be assigned
      for interactions between protein backbone atoms acting as H-bond donors,
      and two more fingerprint bits for protein backbone atoms acting as H-bond
      acceptors.</td>
    </tr>
    <tr>
      <td><b>Solvent&nbsp;H-donor<br />
      H-acceptor</b></td>
      <td>Up to two fingerprint bits per solvent molecule can be
      assigned for interactions between solvent atoms acting as H-bond donors,
      and two more fingerprint bits for solvent atoms acting as H-bond
      acceptors.</td>
    </tr>
    <tr>
      <td><b>Ionic&nbsp;Attraction</b></td>
      <td>Up to two fingerprint bits per residue can be assigned
      for ionic interactions between some charged atom within the residue, and
      some oppositely charged atom on the ligand.</td>
    </tr>
    <tr>
      <td><b>Metal&nbsp;Ligation</b></td>
      <td>Up to two fingerprint bits per residue can be assigned
      for metal interactions. The minimum thresholds are set by default to 0.5
      kcal/mol for the low bit, and 3.5 kcal/mol for the high bit.</td>
    </tr>
    <tr>
      <td><b>Arene&nbsp;Attraction</b></td>
      <td>Up to two fingerprint bits per residue can be assigned
      for arene interactions. The minimum thresholds are set by default to 0.5
      kcal/mol for the low bit, and 1 kcal/mol for the high bit.</td>
    </tr>
  </table>

  <a id="ProteinPLIFs"></a><h2>Protein-Protein Fingerprints</h2>
  <p>The following options are available for fingerprint generation
    in Protein mode.  If any of these fingerprint bits are selected the
    Surface interactions must be calculated.</p>

  <table class="noborder">

    <tr>
      <td><b>Hydrophobic Surface Contact</b></td>
      <td>Up to two fingerprint bits per residue can be assigned for
	contacts between hydrophobic patches on each surface.  The first
	column contains the cutoff values for the Weak interactions, which
	consist of both a minimum contact area between the two patches and
	a minimum fractional value of the patch contact as a percentage of
	the total surface of the residue in contact with the other molecule.
	The second column contains the minimum contact area and percentage
	of the total contact required for a Strong interaction.
      </td>
    </tr>
    <tr>
      <td><b>Charged Surface Contact</b></td>
      <td>Up to two fingerprint bits per residue can be assigned
	for contacts which involve a charged patch on either surface.
	The strong and weak cutoffs are as described above.</td>
    </tr>
    <tr>
      <td><b>Partial Hydrophobic Contact</b></td>
      <td>Up to two fingerprint bits per residue can be assigned
	for contacts in which one surface is part of a hydrophobic patch
	and the other surface is not part of any patch.
	The strong and weak cutoffs are as described above.</td>
    </tr>
    <tr>
      <td><b>Other Surface Contact</b></td>
      <td>Up to two fingerprint bits per residue can be assigned
	for surface contacts in which neither surface is part of any patch.
	The strong and weak cutoffs are as described above.</td>
    </tr>
    <tr>
      <td><b>Total Surface Contact</b></td>
      <td>Up to two fingerprint bits per residue can be assigned
	for the total area per residue in contact with the other surface,
	independent of any patches on either surface.  For this interaction,
	only the minimum surface area for strong and weak interactions is
	required as the fractional value is necessarily 100%.  The total
	surface contact appears in the panel in both Ligand and Protein
	modes, but refers to the same fingerprint bits in either case.</td>
    </tr>

  </table>

  <!-- ================================ USING ============================== -->
<a id="Using"></a>
<h1>Using Fingerprints</h1>

<p>After the protein ligand interaction fingerprints have been generated and
  written to the database, the field <tt>FP:PLIF</tt> will be populated with
  data, and the fingerprints available to generic MOE fingerprint
  functionality, such as clustering and diverse subset calculation. The
  remainder of this page describes visualization and derivative tools which are
  designed especially for use with PLIF.</p>

<p>To bring up the PLIF results panel, open the MOE database, and select
  <b>DBV&nbsp;|&nbsp;Compute&nbsp;|&nbsp;PLIF&nbsp;|&nbsp;Analyze</b>. The
  panel which appears should resemble the following:</p>
  <div align="center"><img src="plif/results.png" /></div>

<p>The results panel is divided up into two main parts:</p>
  <ul>
    <li>Graphic display (top), which reflects a particular view of the
    currently selected entries.</li>
    <li>Entry list (center), which shows all of the fingerprints and other
    fields in a listbox, allowing them to be selected. Various display options
    affect the fields shown in the entry list, but at a minimum, the
    composition of the visible fingerprint bits is shown. Each fingerprint bit
    is denoted by a character to indicate its meaning:
      <table class="noborder">
          <tr>
            <td><tt>-</tt></td>
            <td>bit not set</td>
          </tr>
          <tr>
            <td><tt>D</tt></td>
            <td>sidechain hydrogen bond donor</td>
          </tr>
          <tr>
            <td><tt>A</tt></td>
            <td>sidechain hydrogen bond acceptor</td>
          </tr>
          <tr>
            <td><tt>d</tt></td>
            <td>backbone hydrogen bond donor</td>
          </tr>
          <tr>
            <td><tt>a</tt></td>
            <td>backbone hydrogen bond acceptor</td>
          </tr>
          <tr>
            <td><tt>O</tt></td>
            <td>solvent hydrogen bond</td>
          </tr>
          <tr>
            <td><tt>I</tt></td>
            <td>ionic attraction</td>
          </tr>
          <tr>
            <td><tt>M</tt></td>
            <td>metal ligation</td>
          </tr>
          <tr>
            <td><tt>R</tt></td>
            <td>arene attraction</td>
          </tr>
	  <tr>
	    <td><tt>H</tt></td>
	    <td>hydrophobic surface contact</td>
          </tr>
	  <tr>
	    <td><tt>Q</tt></td>
	    <td>charged surface contact</td>
          </tr>
	  <tr>
	    <td><tt>P</tt></td>
	    <td>partial hydrophobic contact</td>
          </tr>
	  <tr>
	    <td><tt>X</tt></td>
	    <td>other surface contact</td>
          </tr>
	  <tr>
	    <td><tt>C</tt></td>
	    <td>total surface contact</td>
          </tr>

        </table>
    </li>
  </ul>

<p>The graphic display is automatically adjusted to show only the selected
  entries and populated fingerprint bits.  Right-clicking in the list area
  provides additional selection functions, and the keyboard up/down arrows
  can be used to step through the list.  The panel is also
  adorned with a number of options and buttons, which control the
  presentation and interaction with the underlying data.</p>
  <table class="noborder">
    <tr>
      <td><b>Display Mode</b></td>
      <td>The graphical display shown at the top of the panel
      contains a representation which is based on the entries currently
      selected in the entry list, and the selected display mode. The available
      display modes are <a href="#UsingBarcodes"><b>Barcodes</b></a>, <a href=
      "#UsingPopulation"><b>Population</b></a>, <a href=
      "#UsingSignificance"><b>Significance</b></a> and <a href=
      "#UsingLigand"><b>Ligands</b></a>, each of which is described
      subsequently.  The Ligands display mode is only available for viewing
      PLIFs generated in Ligand mode.</td>
    </tr>
    <tr>
      <td><b>Export...</b></td>
      <td>Allows the current state of the graphical display to be
      exported as a picture, using a number of different formats.</td>
    </tr>
    <tr>
      <td><b>Activity</b></td>
      <td>
        For some visualization and calculation purposes, it is necessary to
        define entries as having a binary activity classification, which
        evaluates to 1 (active) or 0 (inactive). If such information is not
        available, activity will not be used. The Activity field consists of
        three parts: field, comparison and value. The field list contains
        numeric fields in the underlying database. The comparison can be
        greater or less than, and the value can be any number.

<p>For the sake of convenience, the results panel checks to see if
        there is a numeric field called <tt>is_active</tt>, and if so, selects
        this field and defines entries to be active if the value is greater
        than 0.5, which is appropriate if all entries contain a value of 1 for
        actives and 0 for inactives. If activity is stored in some other form,
        such as pKi or IC50, then a threshold may be customized as necessary.
        If none of the fields contain activity, select the field labeled
        <tt>&lt;none&gt;</tt>.</p>

<p>Note that in the entry list, entries which are classified as active
        are labeled with a <tt>*</tt> character under the column heading
        <tt>A</tt>.</p>
      </td>
    </tr>
    <tr>
      <td><b>Verbose List</b></td>
      <td>
        This option controls the level of detail displayed in the entry list.
        If on (default), the bit patterns will be split up into groups of
        residues, each of which is labeled by residue number in the column
        header:

<img src="plif/verbose1.png" class="center" />

<p>If off, then the fingerprints for each entry are displayed in a
        concise char-packed form, with no labels to indicate which residues are
        associated with each bit:</p>

<img src="plif/verbose0.png" class="center" />
      </td>
    </tr>
    <tr>
      <td><b>Show&nbsp;Ligand&nbsp;in&nbsp;MOE</b></td>
      <td>If on, or whenever changed, the ligands corresponding to
      the selected entries in the entry list are created in the main MOE
      window, and colored by chain color. When the entry selection changes, or
      this option is turned off, or the panel closes, these molecules will be
      removed.  This option is only available for viewing PLIFs generated in
      Ligand mode.</td>
    </tr>
    <tr>
      <td><b>Fields...</b></td>
      <td>Brings up the <a href="#UsingFieldSel">field
      Selection</a> dialog.</td>
    </tr>
    <tr>
      <td><b>Fingerprints...</b></td>
      <td>Brings up the <a href="#UsingFingSel">Fingerprint
      Selection</a> dialog.</td>
    </tr>
    <tr>
      <td><b>Select...</b></td>
      <td>Brings up the <a href="#UsingSelByFing">Entry Selection
      by Fingerprint</a> dialog.</td>
    </tr>
    <tr>
      <td><b>Tools</b></td>
      <td>When clicked, offers a selection of <a href=
      "#Tools">external tools</a> designed for use with PLIF.</td>
    </tr>
    <tr>
      <td><b>Recalculate</b></td>
      <td>Brings up the <a href="#Generate">Fingerprint Setup</a>
      dialog, which can be used to modify the fingerprints in the
      database.</td>
    </tr>
    <tr>
      <td><b>Update from DBV</b></td>
      <td>Re-reads the underlying data from the source database,
      and replaces the current content. This is necessary if the content of the
      database has been changed by external means, and it is desired that the
      results panel reflect these changes. This will also copy the selection
      state from the <b>DBV</b> so that the same entries will be selected in
      the PLIF display. If <b>Include Selected Only</b> is checked, then only
      the selected entries will be shown.</td>
    </tr>
    <tr>
      <td><b>Select in DBV</b></td>
      <td>Synchronizes the selection in the entry list with the
      corresponding database viewer. The database viewer will be opened if it
      is not already.</td>
    </tr>
    <tr>
      <td><b>View</b></td>
      <td>For the selected entries in the entry list, the
	ligands and the corresponding proteins will be created in the
	MOE main window, deleting any preexisting content. The ensemble
	will then be viewed using the Ligand Interactions panel.  If
	there are multiple selected entries, then the concurrent
	alignment method will be used by default.  <b>Note:</b> This
	option will appear if the fingerprints are generated in Ligand
	mode.</td>
    </tr>
    <tr>
      <td><b>Epitopes...</b></td>
      <td>Brings up the
	<a href="epitope.htm">Epitope Analysis</a> Panel,
	which can be used to browse pose clusters.  <b>Note:</b> This
	option will appear if the fingerprints are generated in Protein
	mode.</td>
    </tr>
    <tr>
      <td><b>Print...</b></td>
      <td>
        Brings up the standard MOE <a href="print.htm#PrintImage">Print
        Preview</a> dialog, with the following options in the Content section:

<img src="plif/print.png" class="center" />

<p>If <b>Bit Descriptions</b> is on, the page layout begins with a
        summary of the fingerprint scheme, which provides the fingerprint
        indices, residue IDs and interaction types for each. <b>Font Size</b>
        may be adjusted as necessary.</p>

<p>If any or all of <b>Barcode Form</b>, <b>Population Histogram</b>,
        <b>Activity Significance</b> or <b>Ligands</b> are checked, then each
        of these graphical display types will be included in the page layout.
        The manner in which they are drawn is the same as for the panel.</p>
      </td>
    </tr>
  </table>

<a id="UsingBarcodes"></a>
<h1>Barcode Display</h1>

<p>The &quot;barcode&quot; display represents the selected entries and
  selected fingerprints as a matrix (one row per entry, one column per
  fingerprint bit) in which a set bit is drawn as a black rectangle:</p>
  <div align="center"><img src="plif/gfx_bc.png" /></div>

<p>The X-axis shows the residue numbers that correspond to each group of
  fingerprint bits, and is coded with an arbitrary sequence of colors.
  Information about the amino acid codes which compose the residues is
  presented when available. If all of the proteins involved have the same
  residue composition with regards to the interactions, then the 3-letter amino
  acid codes are drawn above the residue ID numbers (e.g. His, Asp, Met, etc.).
  If there is a mixture of residue compositions, then single letter codes will
  be used (e.g. H, D, M, etc). If a particular interaction fingerprint involves
  two different types of residues, the two letters will be concatenated (e.g.
  HD). If there are three or more types of residues involved, the description
  will be replaced by <tt>*</tt>. If the residue code does not correspond to a
  naturally occurring amino acid, the description will be <tt>?</tt>.</p>

<a id="UsingPopulation"></a>
<h1>Population Display</h1>

<p>The population histogram displays the frequency of occurrence amongst the
  selected entries for each of the selected fingerprint bits. The X-axis is
  constructed in the same way as for the barcode display, while the Y-axis
  shows a black bar which shows the relative counts for the bits:</p>
  <div align="center"><img src="plif/gfx_pop.png" /></div>

<p>At the top of the largest bar is shown the bit count, the number of
  entries involved, and the percentage of the entries which feature this
  bit.</p>

<p>When the population display is selected, there is an additional option
  called <b>Normalize</b>. If this is checked, the base peak is scaled to take
  up the maximum amount of size in the graphic. If not checked, then the base
  peak will be scaled in proportion to the fraction of the entries in which it
  occurs (e.g. if 100 entries are selected, and the most populous bit occurs in
  50 entries, then the largest bar will be approximately half the size of the
  picture).</p>

<p>If activity information is available, then each of the frequency bars are
  further annotated with a qualitative indication of the overabundance of
  actives vs. inactives. The fraction of actives containing the bit (<b>a</b>)
  is compared with the fraction of actives overall (<b>b</b>). If
  <b>a&nbsp;&gt;&nbsp;b</b>, then the extent to which it is greater is
  indicated by drawing a black line above the bar. If
  <b>a&nbsp;&lt;&nbsp;b</b>, then the extent to which it is less is indicated
  by drawing an inverse (white) line below the top of the bar:</p>
  <div align="center"><img src="plif/overabund.png" /></div>

<p>The combination of height and overabundance is often an indication that
  the corresponding interaction is important for activity. If an interaction
  occurs frequently, but no more often in active compounds than inactive, then
  it is unlikely to be of interest as a distinguishing feature of actively
  binding ligands; similarly if an interaction occurs much more often in active
  compounds than in inactive, but its overall occurrence is rare, it is
  unlikely to be a crucial determining factor.</p>

<a id="UsingSignificance"></a>
<h1>Significance Display</h1>

<p>The significance chart is related to the population histogram, except that
  the heights of the bars are constructed to indicate the extent to which each
  fingerprint bit is an important prerequisite for activity. The display
  differs in that the bars can have positive or negative values, and make use
  of grayscale hues:</p>
  <div align="center"><img src="plif/gfx_sig.png" /></div>

<p>The height of the bars for each fingerprint bit are a product of the
  frequency of the bit and the normalized probability of an entry being active
  if it contains the bit. The probability is calculated as:
  <!-- <span class='LaTeX'>
\begin{align}
P_{b} = \frac{1}{ 1 + \frac{I_{0}}{A_{0}} \cdot \frac{I_{b}}{A_{b}} }
\end{align}
</span> --></p>

<img src="plif/math_plif_eq0.png" class="center" />

<p>where <i>A</i><sub>0</sub> and <i>I</i><sub>0</sub> are the number of
  actives and inactives within the set overall, and <i>I<sub>b</sub></i> and
  <i>I<sub>b</sub></i> are the number of actives and inactives within the
  subset of entries for which the particular bit is set.</p>

<p>The height of the bar for the particular bit, <i>Q<sub>b</sub></i>, is
  calculated as: <!-- <span class='LaTeX'>
\begin{align}
Q_{b} = \frac{C_{b}}{C_{0}} \cdot 2 \left( P_{b} - \frac{1}{2} \right)
\end{align}
</span> --></p>

<img src="plif/math_plif_eq1.png" class="center" />

<p>where <i>C<sub>b</sub></i>is the number of entries within the set which
  have the bit set, and <i>C</i><sub>0</sub> is the size of the set. The value
  of <i>Q<sub>b</sub></i> lies between -1 and 1. A positive value indicates
  that the bit occurs frequently in general, and even more frequently within
  active compounds, relative to the proportion of actives within the set. A
  negative value indicates that the bit occurs more often in the inactives.</p>

<p>The significance of each bit is calculated by proposing the hypothesis
  that if the bit is set, then the compound is active. This hypothesis is the
  compared to a random model based on the overall fraction of entries which are
  marked as active: <!-- <span class='LaTeX'>
\begin{align}
N_{b} & = \frac{\left[ TP, TN, FN, FP \right]}{C_{0}}
\\\\
E_{b} & = \left[
        \frac{A_{0}}{C_{0}}
        \frac{C_{b}}{C_{0}},
        \frac{I_{0}}{C_{0}}
        \left( 1 - \frac{C_{b}}{C_{0}} \right),
        \frac{A_{0}}{C_{0}}
        \left( 1 - \frac{C_{b}}{C_{0}} \right),
        \frac{I_{0}}{C_{0}}
        \frac{C_{b}}{C_{0}}
    \right]
\\\\
\chi_{b}^{2} & = C_{0} \sum \left( \frac{N_{b} - E_{b}}{E_{b}} \right)^{2}
\end{align}
</span> --></p>

<img src="plif/math_plif_eq2.png" class="center" />

<p>The properties of the hypothesis, <i>N<sub>b</sub></i>, are summarized by
  the number of true positives, true negatives, false positives and false
  negatives (<i>TP, TN, FP, FN</i>). The properties of the random model,
  <i>E<sub>b</sub></i>, are obtained from the fraction of the set which
  contains the bit, and the fraction of the set which is classified as
  active.</p>

<p>The bars are color-coded so that highly significant fingerprint bits (low
  <i>&chi;<sub>b</sub></i><sup>2</sup>) are drawn in black/dark gray, while
  less significant bits are drawn in lighter shades of gray.</p>
<a id="UsingLigand"></a>
<h1>Ligand Display</h1>

<p>The ligand display graphic shows the ligands of the selected entries, in
  2D diagrammatic form, with a concurrent alignment that is representative of
  the bound conformation, and further annotated with an indication of the
  interactions in which it participates.  A navigation bar is shown at the
  top which allows selecting the number of ligands to show at once with
  <b>Prev</b> and <b>Next</b> buttons to page through the selected entries.
  <b>Note:</b> Expanding the window size will enable showing multiple rows
  if desired.</p>
  <div align="center"><img src="plif/gfx_lig.png" /></div>

<p>The 3D coordinates of the ligands are first converted into 2D coordinates
  using the 2D depiction method [Clark&nbsp;2006], with a flattening
  constraint, which forces the layout algorithm to compromise pure aesthetics
  in order to find a planar representation which resembles the 3D form
  [Clark&nbsp;2007]. Once all of the poses have been flattened, they are
  superposed onto their original 3D positions, then collectively rotated onto
  the plane of the page. In order to compose this graphic, they are shown
  alongside each other.</p>

<p>The relative layout of the ligands will be meaningful only if the ligand
  coordinates for the selected entries are relative to the same protein, or are
  relative to multiple proteins which have been superimposed onto a common
  space.</p>

<p>For each ligand, the fingerprints are examined, and those which correspond
  to a hydrogen bond are correlated to the individual ligand atoms which were
  found to be involved in the interaction. These ligand atoms are annotated
  with a circle, which is colored in the same way as the X-axis for the
  barcode, population and significance graphics. Surface contact interactions
  are annotated differently, as the number of ligand atoms affected is
  typically many; a dotted outline is drawn around the ligand atoms which are
  close to the residue.  A scrollable legend is shown to the right indicating
  which colors correspond to which residue for the ligands currently displayed.
  <b>Note:</b> This option is only available for small-molecule ligands.</p>
  <div class="floatright"><img src="plif/dlg_fields.png" class="floatright" /></div>
<a id="UsingFieldSel"></a>
<h1>Field Selection</h1>

<p>By default, the main entry list shows index, activity and fingerprint bits
  for each entry, and sorts these by database order. It is possible to show
  additional fields in this display.  Pressing the <b>Fields...</b>
  button brings up the adjacent panel.
</p>

<p>The fields listed underneath <b>Fingerprint Table Fields</b> can be toggled
  on or off, and will be updated accordingly in the main entry list.  Once
  shown in the entry list, selecting the column headings will sort the list
  by the corresponding property values.  Likewise, the fields listed under
  <b>Ligand Depiction Fields</b> can be selected and
  the values will be shown for each ligand in the Ligand Display.
</p>

<a id="UsingFingSel"></a>
<h1>Fingerprint Selection</h1>

<p>Groups of fingerprint bits can be selected or deselected as necessary,
  which can clear up room in the main entry list and graphical display.
  Pressing the <b>Fingerprints...</b> button brings up the following panel.</p>
  <div align="center"><img src="plif/dlg_fprints.png" /></div>

<div class="floatright">
<table class="titlebar">
  <tr>
    <th>Filter Description</th>
    <th>Bit Codes</th>
  </tr>
  <tr>
    <td>Surface Contacts</td>
    <td>CHQPX</td>
  </tr>
  <tr>
    <td>Solvent Interactions</td>
    <td>O</td>
  </tr>
  <tr>
    <td>Hydrogen Bonds</td>
    <td>aAdDO</td>
  </tr>
  <tr>
    <td>Sidechain H-bonds</td>
    <td>AD</td>
  </tr>
  <tr>
    <td>Backbone H-bonds</td>
    <td>ad</td>
  </tr>
</table>
</div>

<p>Each table entry corresponds to a particular residue ID. If
  deselected, none of the corresponding fingerprint bits will be shown,
  and more space will be available to visualize the remaining bits.</p>

<p>The selection can be filtered using the tools at the bottom of
  the panel.  Individual Fingerprint Bit codes can be specified in the
  text field, with the left option menu used to control whether those
  bits are to be excluded or required.  Use the Clear button
  <img src="plif/cleartext.png" alt="Clear" class="icon" />
  to clear the filter text.</p>

<p>The Fingerprint Bit codes can be typed in manually, or for convenience
  several common filters can be selected from the shortcut menu (small
  triangle).  These choices are described in the adjacent table.</p>

<p>The right option menu also allows filtering based on the strength
  of interaction, with <b>Strong</b> only showing those interactions with
  both bits set (<em>i.e.</em> appearing twice in the table.)  Selecting
  <b>Weak</b> will show interactions with the first bit only, with
  the default <b>All</b> showing all bits not filtered by type.</p>

<p><b>Select All</b> and <b>Clear All</b> can be used for convenience
  to either select or deselect all residues currently visible in the
  table.  Residues which have been entirely filtered out are not shown
  and will be automatically deselected.</p>

<p>By default, only fingerprint bits which appear in at least one percent
  of selected entries will be shown.  Unselect the <b>Frequency</b>
  checkbox to disable filtering by frequency, or use the adjecent field
  to modify the cut-off value used by the filter.</p>

<a id="UsingSelByFing"></a>
<h1>Entry Selection by Fingerprint</h1>

<p>Entries in the main list can be selected based on the presence of selected
  fingerprint bits, using the selection panel:</p>
  <div align="center"><img src="plif/dlg_selby.png" /></div>

<p>Some combination of fingerprint specifiers must be selected in the main
  list in this dialog panel. For each residue in the fingerprint scheme, there
  is one entry labeled &quot;any interaction&quot;, which translates to a
  positive match if any fingerprint bit related to that residue is set. For
  each of the distinct bit types, an additional entry is provided.</p>

<p>Each entry either matches or does not, based on the fingerprints selected,
  and the value of <b>Logical Operation</b>. In the case of <tt>OR</tt>, an
  entry matches if any of the selected fingerprints applies. In the case of
  <tt>AND</tt>, an entry only matches if all of the selected fingerprints
  apply.</p>

<p>The changes to the selection in the main results panel are then modified
  by the <b>Current Selection</b> state. If set to <b>Clear</b>, then the
  selection is completely replaced whether or not the entry matched. If <b>Add
  To</b>, then any entry that is matched is selected, and anything previously
  selected remains so. If <b>Subtract</b>, then any entry which matches is
  removed from the selection, if it was selected previously. If
  <b>Restrict</b>, then an entry must match and already be selected in order to
  remain so.
  <!-- ================================ TOOLS ============================== --></p>
<a id="Tools"></a>
<h1>Fingerprint Tools</h1>

<p>The tools described in this section are not an integral part of PLIF, but
  can be accessed conveniently from the <b>Tools</b> menu in the PLIF results
  panel. Each of the tools makes use of existing fingerprint data, and may
  derive a particular property, or modify the content of the database.</p>

<p>Each tool brings up an independently operated panel. Any changes to the
  data which are made by an external tool will not be automatically reflected
  in the corresponding PLIF results panel, so it may be necessary to use the
  <b>Update&nbsp;from&nbsp;DBV</b> button in order to refresh the data view.
  Similarly, tools which require a set of entries to be provided do not obtain
  this selection from the PLIF result panel, but may instead use the selection
  state in the database viewer, which may be synchronized using the
  <b>Select&nbsp;in&nbsp;DBV</b> button.</p>
<a id="ToolsBitSel"></a>
<h2>Bit Selector</h2>

<p>Detailed information about the nature and statistics of each fingerprint
  is presented by this tool:</p>
  <div align="center"><img src="plif/tool_bitsel.png" /></div>

<p>The fingerprint bits may be sorted and selected using particular criteria,
  which may then be used to remove unwanted bits from the fingerprint scheme.
  Also, the raw data used to generate the fingerprint bits can be
  extracted.</p>
  <table class="noborder">
    <tr>
      <td><b>Activity</b></td>
      <td>The definition of active vs. inactive may be specified
      with this option, by selecting a field and a cutoff threshold. If a
      boolean field called <tt>is_active</tt> is defined, this will be set by
      default. The definition of activity affects some of the statistics which
      are presented.</td>
    </tr>
    <tr>
      <td><b>List</b></td>
      <td>
        In the main list, each of the fingerprint bits is shown on one line, in
        the currently sorted order. The bit position, residue ID and
        interaction type are presented in the first three columns. The final
        six columns display statistical information about the participation of
        each of the bits within the database as a whole:
        <div class="indent">
          <dl>
            <dd></dd>
            <dt><tt>%abund</tt></dt>
            <dd>The percentage abundance of the fingerprint bit throughout the
            database. If the fingerprint bit occurs in every entry, the value
            will be 100.</dd>
            <dt><tt>%active</tt></dt>
            <dd>The percentage abundance of the fingerprint bit within the
            entries which are classified as active.</dd>
            <dt><tt>%relabn</tt></dt>
            <dd>The percentage abundance of the fingerprint bit, scaled so that
            the most abundant bit has a value of 100%, e.g. if the most
            populous fingerprint bit occurred in 30 entries, a fingerprint bit
            which occurred in 15 entries would have a <tt>%relabn</tt> value of
            50%, regardless of the total number of entries.</dd>
            <dt><tt>%relact</tt></dt>
            <dd>The percentage abundance of the fingerprint bit within the
            entries classified as active, scaled so that the most populous
            active bit is set to 100%. The calculation is identical as for
            <tt>%relabn</tt>, but using only active entries.</dd>
            <dt><tt>%ovract</tt></dt>
            <dd>The over-activity, which is given as the difference between
            <tt>%active</tt> and <tt>%abund</tt>. A positive value means that
            the bit occurs more often in active compounds than it does in the
            database overall.</dd>
            <dt><tt>%wovact</tt></dt>
            <dd>The value of <tt>%ovract</tt> multiplied by <tt>%abund</tt>,
            expressed as a percentage. Scaling the over-activity by the overall
            abundance gives a number which gives a rough indication of when the
            bit frequently selects in favor of activity. Bits with higher
            values are likely to be necessary for activity, whereas bits with
            small or negative values are likely to be unimportant or actually
            anti-selective.</dd>
          </dl>
        </div>
        By default, the bits are listed in their natural order.  By selecting
        the column headers, the table can be sorted highest first according to
        any of the calculated statistical values presented in the list above.
      </td>
    </tr>
    <tr>
      <td><b>Where</b></td>
      <td>Minimum or maximum ranges of a selected property can be
      used to select or deselect entries in the list. Select first the desired
      property, then enter a minimum and/or maximum bound, and press the
      <b>Select</b> or <b>Deselect</b> buttons to apply the effect to the
      list.</td>
    </tr>
    <tr>
      <td><b>Select&nbsp;All<br />
      Clear&nbsp;Selection</b></td>
      <td>For convenience, the entire set of bits can be selected
      or deselected with a single button click.</td>
    </tr>
    <tr>
      <td><b>Prune</b></td>
      <td>Given that some number of bits have been selected in the
      list, the fingerprint scheme will be modified so that <i>only</i> the
      selected bits remain in the scheme. All other bits will be removed, and
      each entry in the database will be updated accordingly. Next time the
      PLIF results panel is updated, the bit subset will be reflected in the
      display.</td>
    </tr>
    <tr>
      <td><b>Extract</b></td>
      <td>Since the interaction data upon which the PLIF
      fingerprints are based is stored internally within the database, it is
      possible to extract this data, and for each residue/interaction type,
      create a new column in the database and output the strength of the
      corresponding interaction with the ligand (or 0 if none, maximum if more
      than one). The resulting columns can then be used as continuous
      descriptors, e.g. for QSAR studies.</td>
    </tr>
  </table>
<a id="ToolsPoseSel"></a>
<h2>Pose Selector</h2>

<p>One of the possible uses for interaction fingerprints is to disambiguate
  the many poses generated by computational docking algorithms, by studying the
  binding modes of a number of ligands, and deciding which of their putative
  poses is most likely to be the correct one, in the context of all the
  ligands. The <b>Pose Selector</b> tool uses activity data for the ligands to
  discover a common binding motif which is indicative of activity. The
  interaction patterns which most commonly emerge from active ligands should
  contain a structural motif which, if discovered, will lead to a way to
  eliminate many of the superfluous poses.</p>

<p>This collective method is quite orthogonal to conventional docking scoring
  functions, which typically consider each ligand in isolation, often resulting
  in many poses which score similarly well but have quite different binding
  orientations. The method used by this tool is however based on the
  assumptions that (1) there is just one binding mode, and that (2) for most of
  the ligands, at least one pose exists which demonstrates the important
  interactions.</p>

<p>The algorithm operates by postulating a series of hypotheses of the
  form:</p>
  <div class="indent">

  <p>ACTIVE = {contains bits <b>A</b>} AND {does not contain bits
      <b>B</b>}</p>
    </div>

  <p>where <b>A</b> and <b>B</b> are arbitrary sets of bits. A ligand is
    considered as being predicted to be active if any of its constituent poses is
    classified as active by the hypothesis. The efficacy of each hypothesis is
    determined by:</p>
    <div class="indent">

  <p>SCORE = {% true positives} + {% true negatives} - {% false positives} -
      {% false negatives}</p>
    </div>

<p>Hypotheses are initially generated at random, where the set <b>A</b> is
  weighted in favor of selecting bits which are frequently observed in active
  compounds. After a predetermined time, the best randomly determined
  hypotheses are subjected to iterative refinement in order to improve the
  efficacy scores.</p>

<p>If the final hypothesis has a score close to zero, it means that no
  pattern was found which distinguishes actives from inactives. If the final
  score is close to 100%, then the selected motif has a near perfect ability to
  distinguish actives from inactives.</p>

<p>For each group of poses belonging to an active ligand, if there is only
  one pose which satisfies the hypothesis, then that pose is selected as being
  the most probable active pose. If none or multiple poses satisfy the
  hypothesis, then that with the highest Tanimoto similarity to the set
  <b>A</b> is retained, or in the event of a draw, the first entry.</p>
  <div align="center"><img src="plif/tool_posesel.png" /></div>

<p>When the <b>Fit</b> button is selected, the algorithm for selecting active
  poses begins. When it is complete, the <b>Write</b> button becomes available,
  and the results may be written to the underlying database. A database field
  will be created if necessary: <tt>PLIF_apose</tt>, the value of which will be
  0 for any entry which is classified as inactive. For each group of poses from
  active ligands, one entry will have a value of 1, and the others will have a
  value of 0. The database will thence be split into three divisions: active
  poses, inactive poses, and inactive ligands.</p>
<a id="ToolsQuery"></a>
<h2>Query Generator</h2>

<p><i>Available only for small-molecule ligands.</i></p>

<p>Because ligand interaction fingerprints are directly derived from actual
  structural information, it is possible to at any time backtrack the
  fingerprint data to actual 3D interactions. The <b>Query Generator</b> tool
  operates on the principle that a modest selection of poses with a homogeneous
  set of interaction fingerprints will most likely also possess a homogeneous
  set of pharmacophoric feature points. The pharmacophoric feature points can
  be grouped and filtered according to which residues they interact with, and
  those with a sufficiently tight grouping can be converted into a
  pharmacophore query feature.</p>

<p>The tightness of the resulting collection of query features is highly
  dependent on the uniformity of the selected poses. While not all queries
  generated in such a way are suitable for screening databases of conformations
  without binding modes, when applied in absolute mode to poses already docked
  into the protein, or as a constraint on further docking calculations, the
  resultant queries are a very effective way to make use of interaction
  information gleaned from PLIF.</p>
  <div class="center">
    <img src="plif/tool_querygen.png" class="icon"/>
    <img src="plif/ph4query.png" class="icon"/>
  </div>

<p>The main dialog box allows control of the various parameters that go into
  the construction of the 3D query (above left). The query features are shown
  in the MOE main window (above right).</p>
  <table class="noborder">
    <tr>
      <td><b>Entries</b></td>
      <td>
        Before any queries can be generated, the subset of poses to operate on
        must be established. This subset should ideally be between 10 and 100
        poses, and should have a high degree of commonality in binding modes,
        i.e. the interaction fingerprint bits which are important to activity
        should be conserved throughout the set to the extent possible.

<p>If set to <b>All</b>, then the entire database will be considered as
        the input. If set to <b>Selected</b>, then all entries which are
        selected in the Database Viewer will be considered (note that this is
        <i>not</i> the selection within the main PLIF results panel; use the
        <b>Select&nbsp;in&nbsp;DBV</b> button to synchronize these). If set to
        <b>Active&nbsp;Poses</b>, then all entries for which the value of the
        field <tt>PLIF_apose</tt> is true will be considered.</p>
      </td>
    </tr>
    <tr>
      <td><b>Scheme</b></td>
      <td>The pharmacophore query scheme to use for generating the
      query features. All of the schemes available in MOE are presented in this
      list.</td>
    </tr>
    <tr>
      <td><b>List of fingerprints</b></td>
      <td>The fingerprint bits which correspond to at least one
      pharmacophore feature are listed in this list box, along with the type
      and number of poses in which they occur. These fingerprint bits are all
      selected by default, which means any of them may be combined into the
      final query features. They may be deselected, in which case they will be
      removed from consideration.</td>
    </tr>
    <tr>
      <td><b>Max.&nbsp;Radius</b></td>
      <td>Defines the maximum radius of a query feature. If a
      group of interaction feature points cannot be grouped together with a
      radius of less than this value, the query feature will be split into
      smaller groups, or excluded.</td>
    </tr>
    <tr>
      <td><b>Feature Coverage</b></td>
      <td>Each proposed query feature must hit at least this
      fraction of the provided input poses, e.g. if 10 poses are provided, and
      the threshold is set to 60%, a proposed query feature which only matches
      the pharmacophore points of 5 or less poses will be rejected, but one
      which matches 6 or more will be included in the final results.</td>
    </tr>
    <tr>
      <td><b>List of query features</b></td>
      <td>All of the query features which qualify according to the
      thresholds will be shown in this list box, and display in the MOE main
      window. By default all are selected, but they can be deselected, in which
      case they will not be displayed, nor will they be included when the
      pharmacophore query is saved.</td>
    </tr>
    <tr>
      <td><b>Show Original Features</b></td>
      <td>If on, the individual pharmacophoric features of all the
      input poses will be displayed in the MOE main window, in addition to the
      query features derived from them.</td>
    </tr>
    <tr>
      <td><b>Excluded Volumes</b></td>
      <td>If on, then the region of space occupied by the protein
      will be included in the query in the form of excluded volumes. This is of
      interest when the query is intended to be used in screening mode, for
      matching conformations without explicitly binding to the protein.</td>
    </tr>
    <tr>
      <td><b>Build</b></td>
      <td>Constructs the pharmacophore interaction points using
      the indicated input pose and the selected scheme. From this point on, the
      pharmacophore query result is updated dynamically.</td>
    </tr>
    <tr>
      <td><b>Evaluate</b></td>
      <td>Runs through the source database and evaluates the
      extent to which the pharmacophore query matches the contents, in both
      absolute and relative modes, and provides information as to the ratio of
      actives and inactives which match. An entry is considered to match in
      absolute mode if the bound pose matches the pharmacophore query without
      any rotation or translation. It is considered a match in relative mode if
      the conformation is capable of fitting the query by applying any rigid
      transformation. This is a convenient way of approximately evaluating the
      efficacy of the query, without proper statistical validation. The output
      is written to the SVL Commands window.</td>
    </tr>
    <tr>
      <td><b>Create</b></td>
      <td>Creates the pharmacophore query, closes the dialog, then
      opens the pharmacophore query editor in its stead.</td>
    </tr>
  </table>
<a id="ToolsSimil"></a>
<h2>Similarity</h2>

<p>Using the interaction fingerprint bits to compare similarity between a
  selected subset of the database and all the rest can be a useful way to
  estimate properties or segregate the data in various ways. For instance, if
  one or more entries have been obtained from crystal structures of known
  active ligands, the activity of the remaining data might be estimated by
  means of comparing the Tanimoto coefficients between their fingerprints and
  those of the remaining entries.</p>
  <div align="center"><img src="plif/tool_simil.png" /></div>
  <table class="noborder">
    <tr>
      <td><b>Compare&nbsp;To</b></td>
      <td>The subset of entries to which the remaining entries are
      to be compared to is defined by this field. The default value,
      <tt>selected</tt>, implies that the selected entries in the Database
      Viewer define that set. The other entries consist of the numerical fields
      in the database, and these may be used in the form of a cutoff. Generally
      the set should be small, e.g. a handful of known-active correct binding
      poses.</td>
    </tr>
    <tr>
      <td><b>Similarity</b></td>
      <td>The similarity metric may be either the Tanimoto
      coefficient or the Euclidean distance. Note that for Tanimoto
      coefficients, larger numbers indicate more similarity, while for the
      Euclidean the opposite is true.</td>
    </tr>
    <tr>
      <td><b>Collective</b></td>
      <td>The collective behavior of comparison must be selected.
      If <b>max</b> is selected, then each entry is compared with the selection
      set, and the highest value is chosen (for Tanimoto, this means most
      similar, for Euclidean this means least similar). Conversely, <b>min</b>
      causes the lowest value to be chosen. If <b>mean</b> is selected, then
      the similarities between a particular entry and all of the selected set
      are computed, then the mean average taken.</td>
    </tr>
    <tr>
      <td><b>Output To</b></td>
      <td>The field to which the resulting similarity value is
      written, which defaults to <tt>PLIF_sim</tt>.</td>
    </tr>
  </table>

<p>To conduct the calculation and write the results to the underlying
  database, press the <b>Calculate</b> button.
  <!-- ============================ REFERENCES ============================= --></p>

<h1>See Also</h1>

<p><a href="surfmap.htm">Surfaces and Maps</a><br />
  <a href="sitefind.htm">Site Finder</a><br />
  <a href="docking.htm">Molecular Docking</a><br />
  <a href="epitope.htm">Epitope Analysis</a><br /></p>
<a id="Ref"></a>
<h1>References</h1>
  <table class="ref">
    <tr>
      <td>[Labute&nbsp;2001]</td>
      <td>Labute, P.; Probabilistic Receptor Potentials;
      <i>Journal of the Chemical Computing Group</i> (<b>2001</b>) <a href=
      "http://www.chemcomp.com/journal/cstat.htm">http://www.chemcomp.com/journal/cstat.htm</a>.</td>
    </tr>
    <tr>
      <td>[Clark&nbsp;2006]</td>
      <td>(a) Clark, A.M., Labute, P., Santavy, M.; 2D Structure
      Depiction; <i>J. Chem. Inf. Model. 46</i> (<b>2006</b>)
      1107&ndash;1123.<br />
      (b) <a href=
      "http://www.chemcomp.com/journal/depictor.htm">http://www.chemcomp.com/journal/depictor.htm</a></td>
    </tr>
    <tr>
      <td>[Clark&nbsp;2007]</td>
      <td>(a) Clark, A.M., Labute, P.; 2D Depiction of
      Protein-Ligand Complexes; <i>J. Chem. Inf. Model. 47</i> (<b>2007</b>)
      1933&ndash;1944.<br />
      (b) <a href=
      "http://www.chemcomp.com/journal/ligintdia.htm">http://www.chemcomp.com/journal/ligintdia.htm</a></td>
    </tr>
  </table><!-- START MOE_FOOTER -->
  <div class="MOE_FOOTER">
    <img src="../images/ccgicon.png" /> <a href="../index.htm"></a> <a href=
    "../legal.htm"></a> &copy;<span class="versionyear"></span> <a href="http://www.chemcomp.com"></a>. All
    rights reserved.<br />
    <a href="mailto:info@chemcomp.com"></a>
  </div><!-- END MOE_FOOTER -->
</div>
</div>
</body>
</html>

