<!DOCTYPE html>
<html>

<!--
!!    MOE On-Line Manuals
!!    COPYRIGHT (C) CHEMICAL COMPUTING GROUP ULC.  ALL RIGHTS RESERVED.
!!-->
<head>
  <meta http-equiv="x-ua-compatible" content="IE=9" />
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <link rel="stylesheet" type="text/css" href="../include/manstyle.css" />
  <link rel="icon" type="image/png" href="../images/ccgicon.png" />
  <link rel="shortcut icon" type="image/png" href="../images/ccgicon.png" />
  <title>Protein Design</title>
  <meta name="keywords" content="panel: Protein Design" />
  <meta name="keywords" content="panel: Alanine Scan" />
  <meta name="keywords" content="panel: Disulfide Scan" />
  <meta name="keywords" content="panel: Residue Scan" />
  <meta name="keywords" content="panel: Resistance Scan" />
  <meta name="keywords" content="panel: Sequence Scan" />
  <meta name="keywords" content="main_keywords: protein design, scan" />
  <meta name="keywords" content="main_keywords: alanine, residue sequence" />
  <meta name="keywords" content=
  "main_keywords: disulfide, resistance sequence" />
  <meta name="keywords" content=
  "main_keywords: sequence design" />
  <meta name="keywords" content="mutation, design" />
  <style type="text/css">
  /*<![CDATA[*/
  td { vertical-align: top; }
  /*]]>*/
  </style>
  <style type="text/css">
  /*<![CDATA[*/
  .subsup_f { line-height: 1.8; }
  .subsup_f sub { margin-right: 1ex; }
  .subsup_f sup { margin-left: -1ex; }
  /*]]>*/
  </style>
</head>
<body>
<div class="center-page">
  <!-- START MOE_HEADER -->
  <div class="MOE_HEADER">
    <a href="../index.htm"><img src="../images/ccglogo.png" /></a>
    <a href="../index.htm"><div class="moeversion"></div></a>
    Protein Design
  </div>
<!-- END MOE_HEADER -->
<div class="content">
  <!-- <div id="useMathJax"></div> -->
  <ul>
    <li><a href="#Introduction">Introduction</a></li>
    <li>
      <a href="#Methodology">Methodology Overview</a>
      <ul>
        <li><a href="#Searching">Searching Conformational and Sequence
        Space</a></li>
        <li><a href="#Refinement">Refining Mutants Generated by UQO</a></li>
        <li><a href="#Scoring">Scoring Changes in Protein Stability</a></li>
        <li><a href="#Ensemble">Generation of Conformational Ensembles</a></li>
        <li>
          <a href="#Variants">Protein Design Variants</a>
          <ul>
            <li><a href="#AlanineScanMethod">Alanine Scan Methodology</a></li>
            <li><a href="#DisulfideScanMethod">Disulfide Scan
            Methodology</a></li>
            <li><a href="#ResistanceScanMethod">Resistance Scan
            Methodology</a></li>
            <li><a href="#ResidueScanMethod">Residue Scan Methodology</a></li>
            <li><a href="#SampleSequenceMethod">Sample Sequence
            Methodology</a></li>
            <li><a href="#SequenceDesignMethod">Sequence Design
            Methodology</a></li>
          </ul>
        </li>
      </ul>
    </li>
    <li>
      <a href="#Run">Running the Protein Design Application</a>
      <ul>
        <li><a href="#Preparing_the_Data">Preparing the Data</a></li>
        <li>
          <a href="#Using_the_Panel">Using the Protein Design Panel</a>
          <ul>
            <li>
              <a href="#MutationEditor">Using the Mutation List Editor</a>
              <ul>
                <li><a href="#ConstantMutation">Defining a Constant
                Mutation</a></li>
                <li><a href="#MultipleMutations">Defining Multiple
                Mutations</a></li>
                <li><a href="#NonNaturalAA">Mutating to/from Non-Natural Amino
                Acids</a></li>
              </ul>
            </li>
            <li>
              <a href="#panel_variants">Using Protein Design Variants</a>
              <ul>
                <li><a href="#AlanineScan">Alanine Scan</a></li>
                <li><a href="#DisulfideScan">Disulfide Scan</a></li>
                <li><a href="#ResistanceScan">Resistance Scan</a></li>
                <li><a href="#ResidueScan">Residue Scan</a></li>
                <li><a href="#SampleSequence">Sample Sequence</a></li>
                <li><a href="#SequenceDesign">Sequence Design</a></li>
              </ul>
            </li>
            <li><a href="#panel_input">Input/Output Options</a></li>
            <li><a href="#display_opt">Display Options</a></li>
            <li><a href="#ensemble_opt">Ensemble Options</a></li>
            <li><a href="#affinity_opt">Affinity Options</a></li>
            <li><a href="#properties_opt">Properties Options</a></li>
          </ul>
        </li>
        <li>
          <a href="#Output">Working with Output</a>
          <ul>
            <li><a href="#Output_Database">The Output Database</a></li>
            <li><a href="#Browsing_the_Output_Database">Browsing the Output
            Database</a></li>
          </ul>
        </li>
        <li><a href="#Using_MOEsmp">Running Protein Design with MOE/cluster
        (MOE/smp)</a></li>
        <li><a href="#Using_GPU">Running Protein Design with GPU</a></li>
        <li><a href="#Continuing">Continuing an Incomplete Protein Design
        Job</a></li>
        <li><a href="#Reloading">Reload/Start from a Protein Design Output
        File</a></li>
      </ul>
    </li>
    <li>
      <a class="example" href="#Examples">Examples</a>
      <ul>
        <li><a class="example" href="#Example1_Data_Preparation">Example 1:
        Data Preparation for Protein Design</a></li>
        <li><a class="example" href="#Example2_DisulfideScan">Example 2:
        Disulfide Scan on 2ST1</a></li>
        <li><a class="example" href="#Example3_ResidueScan">Example 3: Residue
        Scan on 1VFB</a></li>
      </ul>
    </li>
    <li><a href="#References">References</a></li>
    <li><a href="#See_Also">See Also</a></li>
  </ul>
<a id="Introduction"></a>
<h1>Introduction</h1>

<p>Biologics are human or animal protein derived compounds. Such substances
  play an ever-increasing role in everyday life, from laundry to therapeutics,
  and their great promise has given rise to an urgent need for efficient
  methods for engineering new biomolecules. Computational techniques can assist
  in suggesting modifications to existing structures for obtaining desirable
  properties and functions. In MOE, the Protein Design application permits
  unlimited simultaneous residue mutations of target structures, and then
  calculates the properties of the resultant mutant, with the purpose of
  examining how mutations of a protein modulate its protein properties.</p>
<a id="Methodology"></a>
<a id="Methodology_Overview"></a>

<h1>Methodology Overview</h1>

<p>Computational approaches to protein design face a difficult problem, as
  illustrated in the figure below. Given a protein structure, the generation of
  an arbitrary mutant in principle requires taking into consideration all
  possible residue mutations and all rotamers of each mutation state. A
  selection process must then choose a single state for each residue, which
  ideally should coincide with the minimum global energy structure as a
  whole.</p>

<img src="prodesign/size_of_problem.png" alt=
  "Protein Design Problem" class="center" />

<p><br clear="all" /></p>
  <div class="floatleft"><img src="prodesign/method.png" alt=
  "Protein Design Method" style="margin: 0 15px 0 0" class="floatleft" /></div>

<p>The Protein Design methodology reduces the size of the problem by
  partitioning the system and then pruning unpropitious states. To make the
  search for optimal solutions computationally tractable, mutation/rotamer
  states are encoded into an energy matrix that lends itself to solution by
  Unary Quadratic Optimization (UQO) [Labute&nbsp;2009]. The Protein Design
  algorithm is as follows:</p>
  <ol>
    <li><b>Partition the System.</b> The atoms of the system are divided into
    two groups: <i>sidechains</i> and <i>environment</i>. The sidechain atoms
    are collected after the specified residue mutations have been made. All
    other atoms, including backbone atoms, are considered part of the
    environment.</li>
    <li><b>Read mutation and rotamer states.</b> The desired mutations and
    associated rotamers are obtained from the rotamer library (see <span class=
    "nobr"><span class="menu">MOE | Settings | Protein Defaults</span>).</span>
    For example, for histidine, four tautomer and protomer forms are read out.
    The rotamer library also contains the self energy and strain energy for
    each rotamer of the desired mutation(s). The backbone atoms of the rotamers
    are superposed onto the backbone atoms of the residues to be mutated.</li>
    <li><b>Matrix Setup.</b> The interaction energies between each pair of
    mutation and rotamer states are calculated and stored in a matrix, U. High
    energy states are removed during construction of the matrix, garnering
    savings in both computational and memory requirements.</li>
    <li><b>Unary Quadratic Optimization (UQO)</b>. First, Dead-End Elimination
    [Goldstein&nbsp;1994] is used to reduce the complexity of the optimization
    procedure in subsequent steps. This step is followed by a search for the
    particular configuration of states (mutation and/or rotamer) that minimizes
    a quadratic energy function, which is described in more detail <a href=
    "#binvec">below</a>.</li>
    <li><b>Refinement of Mutations</b>. A final energy minimization of the UQO
    solution is performed to relax the structure.</li>
    <li><b>Stability/Affinity Scoring</b>. A final scoring for stability and
    affinity is performed on the refined structure. Additionally some common
    protein property descriptors (see <a href="#Output">Working with
    Output</a>) are automatically calculated. The units of affinity and
    stability are kcal/mol.</li>
    <li><b>Ensemble Generation</b>. the Protein Design application can
    optionally generate an ensemble of protein conformations using any of the
    <a href="../proteins/probuild.htm#RotamerExplorer">Rotamer Explorer</a>,
    <a href="confsrch.htm#LowModeMD">LowMode&nbsp;MD</a> or <a href=
    "md.htm">Molecular Dynamics</a> methods.</li>
    <li><b>Ensemble Refinement</b>. If an ensemble is being generated, the
    conformations generated by the ensemble method are refined using the same
    method as in Step 5, &quot;Refinement of Mutations&quot;. The ensemble is
    then pruned to remove similar and high energy conformations.</li>
    <li><b>Ensemble Scoring</b>. If an ensemble is being generated, each
    conformation can be scored for stability, affinity, and protein properties.
    The values are averaged using a Boltzmann distribution.</li>
  </ol>

<p><br clear="all" />
<a id="Searching"></a>
</p>

<h2>Searching Conformational and Sequence Space</h2>
  <div class="floatright"><img src="prodesign/prodes_partition.png" alt=
  "Protein Design Partition" class="floatright" /></div>

<p>In the first step of the Protein Design algorithm, the system is
  partitioned by implicitly cutting the bond joining the <i>&alpha;</i> and
  <i>&beta;</i> carbons in the residues to be mutated. The result of the
  partitioning is a collection of non-intersecting atom sets (sidechains) each
  associated with a particular residue. Each such atom set will have an
  associated collection of one or more mutation and rotamer states which have
  been read in from the rotamer library (see <span class="nobr"><span class=
  "menu">MOE | Settings | Protein Defaults</span>).</span> Rotamer and mutation
  backbone atoms are superposed onto the backbone atoms of the residue to be
  mutated. Since the rotamers in the library do not include any backbone atoms,
  the strain energy due to a backbone carbonyl oxygen is added to the strain
  energy of the sidechain. A <b>Repack Environment</b> option allows nearby
  residues to be included in the conformational search.</p>
<a id="binvec"></a>
<p>A binary vector <i>x</i> is used to encode a particular selection of
  mutation/conformational states of the residues to be mutated <i>i</i>,
  <i>j</i>, <i>k</i>,&nbsp;&hellip;. The bits of <i>x</i> are arranged in
  blocks, where each block represents a residue, and each element of the block
  represents one of the possible mutation/rotamer configurations of that
  residue. Consider the image at right in which residue <i>i</i>, an alanine,
  is to be mutated into one of three possible residues, with one conformation
  (green) for the first mutation, and two for each of the second (blue) and
  third (purple), for a total of five possible mutation/rotamer configurations.
  These states would correspond to a block of 5 bits in the binary vector.
  Similarly, residue <i>j</i> is depicted as having two possible mutations with
  one conformation each, for a total of two bits in the binary vector, and
  residue <i>k</i> has 3 mutations and is represented by 4 bits in the binary
  vector. The vector <i>x</i> with the following sequences of bits:</p>

<p class="center"><span class="nobr"><i>x</i> = [ 0 1 0 0 0 | 1 0 | 0 0 1 0 |
  ... ]</span></p>

<p>would then specify the configuration of state 2 for <i>i</i>, state 1 for
  <i>j</i> and state 3 for <i>k</i>. In this representation, a value of 1 in
  <i>x</i> marks the presence of a particular state, and 0, its absence.
  Admissible values of <i>x</i> are those in which there is exactly one bit of
  value 1 in each block (all the rest are 0's). This constraint on the
  values of <i>x</i> is called a <i>unary constraint</i> since the individual
  states of the mutation are encoded in base 1 or unary notation.</p>

<p>The energy of a configuration is determined from a quadratic function:</p>

<p class="center"><i>E</i>(<i>x</i>) =
  &frac12;&middot;<i>x</i>&middot;<i>U</i><i>x</i> + <i>u</i>&middot;<i>x</i> +
  <i>u</i><sub>0</sub></p>

<p>which is taken to be a multidimensional quadratic. The function is
  determined by an interaction matrix <i>U</i> and a &quot;self&quot; energy
  vector <i>u</i>.</p>

<img src="prodesign/prodes_matrix.png" alt=
  "Protein Design Partition" class="center" />

<p>The matrix contains the interaction energy between each pair of residue
  states. Within a block, the matrix contains 0's and 1's, since
  exactly one state can be chosen within each block. The vector <i>u</i>
  contains interactions with the rest of the system, for instance, backbone
  atoms. For a given vector, subject to the unary constraint, the function E
  returns the free energy of the system. The Protein Design application uses
  van der Waals, hydrogen bond, pKa and a disulfide bond terms to populate the
  matrix. The functional form for an individual conformer would therefore
  be:</p>

<p class="center">S = <i>E<sub>self</sub></i> + <i>E<sub>vdw</sub></i> +
  <i>E<sub>hb</sub></i> + &Delta;<i>E<sub>pH-pK<sub>a</sub></sub></i> +
  <i>E<sub>SS</sub></i></p>

<p style="text-align:left;">where</p>

  <table class="noborder">
    <tr>
      <td>
      <i>E<sub>self</sub></i></td>
      <td>The self energy of the
      specific conformation of the mutation. <i>E<sub>self</sub></i> is
      comprised of the AMBER angle, torsion and van der Waals terms.</td>
    </tr>
    <tr>
      <td>
      <i>E<sub>vdw</sub></i></td>
      <td>The AMBER van der Waals
      interaction energy from the matrix <i>u</i>.</td>
    </tr>
    <tr>
      <td>
      <i>E<sub>hb</sub></i></td>
      <td>
        A dipole-based hydrogen bond term from the matrix <i>u</i>. A component
        of this term is a hydrogen bond radial 12-10 distribution that is
        consistent with contact statistics derived from the PDB and dipole
        interaction based energy. The 12-10 distribution has the following
        formalism:

<img src="prodesign/formula_1012dist.png" alt=
        "10-12 distribution" class="center" />

<p>where <i>r<sub>ij</sub></i> is the distance between the polar heavy
        atoms in the sidechain and the environment.</p>

<p>Each polar hydrogen-bearing heavy atom is given a point dipole, with
        a direction consistent with its hybridization, and with a magnitude
        consistent with the TIP5P water model. Dipole-dipole electrostatic
        interactions are evaluated for each such pair according to:</p>

<img src="prodesign/formula_edipdip.png" alt=
        "dipole-dipole hydrogen bond energy" class="center" />

<p>where <i>d<sub>i</sub></i> and <i>d<sub>j</sub></i> are the dipole
        vectors of the heavy atoms and <i>r<sub>ij</sub></i> is the vector
        between the atom positions.</p>

<p>The total hydrogen bond interaction energy <i>E<sub>hb</sub></i>,
        which takes directional effects into account, is the product of the
        radial term <i>P</i> and the dipole-dipole energy term
        <i>E<sub>dip-dip</sub></i>.</p>
      </td>
    </tr>
    <tr>
      <td><span class=
      "nobr">&Delta;<i>E<sub>pH-pK<sub>a</sub></sub></i></span></td>
      <td>
        A term used to account for <i>pK<sub>a</sub></i> shifts due to
        conformational and environmental effects. This allows for prediction of
        the correct protonation state for a given conformation.
        &Delta;<i>E<sub>pH-pK<sub>a</sub></sub></i> is given by:

<p class="center">&Delta;<i>E<sub>pH-pK<sub>a</sub></sub></i> =
        (<i>kT</i> log 10)(<i>pH</i> - <i>pK<sub>a</sub></i>)</p>

<p>where <i>pK<sub>a</sub></i> for a particular mutation and its
        conformation is calculated using <a href=
        "residuepka.htm">PROPKA</a>.</p>
      </td>
    </tr>
    <tr>
      <td>
      <i>E<sub>SS</sub></i></td>
      <td>
        A geometry-based energy term for disulfide bonds. In the creation of
        the energy matrix <i>U</i>, whenever two cysteine sulfurs are within a
        specified distance cutoff, they are treated as if they were forming a
        disulfide bond. <i>E<sub>SS</sub></i> is equal to 0 when the geometry
        is distant from that of the ideal disulfide bond. The definition of an
        ideal disulfide bond in the Protein Design application is:
        <ul>
          <li>The distance between the two cysteine sulfurs atoms is in the
          range <span class="nobr">1.99&Aring;&ndash;2.44&Aring;.</span></li>
          <li>The angles between the &beta; carbons and both sulfurs are within
          45&deg; of the ideal angle of 103.7&deg;.</li>
          <li>The dihedral angle &Chi;<sub>S-S</sub> of the disulfide bond is
          within the 30&deg; of the ideal dihedral angle of 90&deg;.</li>
        </ul>

<p>When matching these geometry constraints, <i>E<sub>SS</sub></i> is
        equal to:</p>

<p class="center"><i>E<sub>SS</sub></i> = -300 + abs(2.04 -
        <i>d<sub>SS</sub></i>) + max abs(103.7 -
        <i>&Theta;<sub>C<sub>&beta;i</sub>S<sub>i</sub>S<sub>j</sub></sub></i>)
        + 6&middot;abs(90 - <i>&Chi;<sub>S-S</sub></i>)</p>with
        &Chi;<sub>S-S</sub> = 0 if 60&deg; &lt; &Chi;<sub>S-S</sub> &lt;
        120&deg;.
      </td>
    </tr>
  </table>

<p>Following the creation of the energy matrix <i>U</i>, the protein design
  problem can now be reduced to finding the binary vector <i>x</i>, subject to
  the unary constraint, that minimizes <span class=
  "nobr"><i>E</i>(<i>x</i>).</span> This is done with a Unary Quadratic
  Optimization algorithm. For more information on the Unary Quadratic
  Optimization itself see the <a href=
  "../apps/protonate3d.htm#Method">Protonate3D Methodology</a> section which
  describes the optimization technique in detail.</p>
<a id="Refinement"></a>
<h2>Refining Mutations Generated by UQO</h2>

<p>After producing a mutant using the above methodology, it is beneficial to
  perform an energy minimization on the mutated structure to relax it prior to
  scoring. Before the energy minimization is done, the structure is checked for
  severe clashes at the mutation site. If clashes are detected, the mutation is
  repeated with slightly different starting parameters to avoid the
  clashes.</p>

<p>The refinement protocol allows atoms in the mutated residue(s) to move
  freely and tethers the backbone atoms of neighboring residues
  (residues with atoms within the 4.5 &Aring; repack cutoff of the mutated
  residues) with a weight of 10 kcal/mol and an allowed deviation of
  0.25 &Aring;. All other atoms are kept fixed.
  Atoms in residues which are farther away than 11 &Aring;
  (<a href="../moe/pot.htm#Forcefield">non-bonded cutoff</a> + 1 &Aring;</a>)
  from the mutated residues are set to be inert (i.e. ignored)
  to speed up the minimization.</p>

<p>If the <a href="#PreserveRestraints">Preserve Restraints</a> option
  is enabled, additional restraints can be applied:
  <ul>
  <li>Atoms of residues which have all atoms fixed in the input structure
  are kept fixed. Atoms of the mutated residues are free to move.</li>
  <li>Heavy sidechain atoms of residues in which all of them are tethered
  in the input structure are tethered with their maximum tether weight.
  The side chain hydrogens are not tethered. The atoms of the mutated
  residues are free to move as well.</li>
  <li>Existing force field restraints (distance, angle, torsions) are applied
  if an atom name match to the input structure can be established.</li>
  </ul>
</p>  
<p>The all-fixed, all-tethered, and restrained residues are excluded from
  the environment repacking.
  The refinement protocol and forcefield setup for ensemble conformation
  generation differs slightly and is described in the <a href="#Ensemble">
  ensemble</a> section below.</p>

<a id="Scoring"></a>
<h2>Scoring Protein Stability</h2>
  <div class="floatright"><img src="prodesign/stability_prediction.png" alt=
  "Protein Design Prediction of Stability" class="floatright" /></div>

<p>Accurate modeling of the impact of a mutation on a protein must capture
  both the conformation change upon mutation as well as the change in stability
  from wild type to mutant protein. The change in stability due to mutation is
  measured experimentally as the difference in free energy of folding of the
  mutant (Mut), and wild type (WT). Here, we predict this change from the
  difference in stability (<i>s</i>) between the mutant and wild type in the
  folded (<i>f</i>) and unfolded (<i>u</i>) states of the protein:</p>

<p class="center">&Delta;&Delta;<i>G<sub>s</sub></i> =
  &Delta;<i>G<span class="subsup_f"><sub>f</sub><sup>Mut</sup></span></i> -
  &Delta;<i>G<span class="subsup_f"><sub>f</sub><sup>WT</sup></span></i> =
  &Delta;<i>G<span class=
  "subsup_f"><sub>s<sub>f</sub></sub><sup>WT&rarr;Mut</sup></span></i> -
  &Delta;<i>G<span class=
  "subsup_f"><sub>s<sub>u</sub></sub><sup>WT&rarr;Mut</sup></span></i></p>

<p>Using this formalism, we can create a model based on Linear Interaction
  Energy (LIE). According to LIE, &Delta;<i>G</i> can be expressed as the
  change in the residue environment interaction energy going from wild-type to
  mutant, where the interaction energy is the change between the energy of the
  protein and the energy of the protein without the residue of interest. LIE is
  usually expressed as:</p>

<p class="center">&Delta;<i>G</i> = &Delta;<i>E<sub>vdw</sub></i> +
  &Delta;<i>E<sub>ele</sub></i> + <i>T</i>&Delta;<i>S</i></p>

<p>In the case of the folded state, this is calculated using the following
  equation, applying the ideal theoretical LIE values of 1 for
  &Delta;<i>E<sub>vdw</sub></i> and 0.5 for &Delta;<i>E<sub>ele</sub></i>.</p>

<p class="center">&Delta;<i>G<span class=
  "subsup_f"><sub>s</sub><sup>WT&rarr;Mut</sup></span></i> =
  <i>&alpha;</i>[&Delta;<i>E<sub>vdw</sub></i> +
  0.5(&Delta;<i>E<sub>coul</sub></i> + &Delta;<i>E<sub>sol</sub></i>)] +
  &beta;&Delta;<i>E<sub>SS</sub></i> +
  &gamma;&middot;&Delta;<i>SA<sub>sc</sub></i> +
  &epsilon;&middot;&Delta;<i>SA<sub>pol</sub></i></p>

<p>where &alpha; is a scaling factor accounting for configurational entropy
  effects, &Delta;<i>E<sub>vdw</sub></i> is the AMBER van der Waals interaction
  energy, &Delta;<i>E<sub>coul</sub></i> is the AMBER Coulomb interaction
  energy, &Delta;<i>E<sub>sol</sub></i> is the change in solvation energy
  calculated using <a href="../moe/pot.htm#Equation">GBVI</a>, and
  &Delta;<i>E<sub>SS</sub></i> is the change in energy due to the presence of a
  disulfide bond. This value is calculated using the same formalism as
  described in the <a href="#Searching">Sequence and Conformational Search</a>
  section above. &Delta;<i>SA<sub>sc</sub></i> and
  &Delta;<i>SA<sub>pol</sub></i> are the changes in the side-chain and polar
  surface areas.</p>

<p>Predicting the change in stability of the unfolded states proves to be
  more difficult since there are typically no structures available of this
  state. It may be possible to use a computational method to predict the
  unfolded state but this would require a significant investment in time, both
  real and computational. In addition, the unfolded state may, in fact,
  comprise multiple conformations, making this prediction even more
  difficult.</p>

<p>Using the structure of the unfolded step can be side-stepped if it is
  assumed that the stability change of the unfolded state is in fact a
  constant. This assumption can be made on the basis that this unfolded state
  is unrelated to the sequence of the protein and its environment. In other
  words, the stability is only dependent on the change of wild type residue to
  mutant. Therefore the change in stability in the unfolded state was broken
  down into 9 terms:</p>

<p class="center">&Delta;<i>G<span class=
  "subsup_f"><sub>s<sub>u</sub></sub><sup>WT&rarr;Mut</sup></span></i> =
  <i>&kappa;<sub>1</sub></i>&middot;&Delta;<i>E<sub>self</sub></i> +
  <i>&kappa;<sub>2</sub></i>&middot;&Delta;<i>S<sub>ema</sub></i> +
  <i>&kappa;<sub>3</sub></i>&middot;&Delta;<i>S<sub>emd</sub></i> +
  <i>&kappa;<sub>4</sub></i>&middot;&Delta;<i>&delta;<sub>acc</sub></i> +
  <i>&kappa;<sub>5</sub></i>&middot;&Delta;<i>&delta;<sub>don</sub></i><br />
  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; +
  <i>&kappa;<sub>6</sub></i>&middot;&Delta;<i>&delta;<sub>rot</sub></i> +
  <i>&kappa;<sub>7</sub></i>&middot;&Delta;<i>&delta;<sub>H</sub></i> +
  <i>&kappa;<sub>8</sub></i>&middot;&Delta;<i>&delta;<sub>heavy</sub></i> +
  <i>&kappa;<sub>9</sub></i>&middot;&Delta;<i>&delta;<sub>gly</sub></i> +
  <i>&kappa;<sub>10</sub></i></p>

<p>In this equation &Delta;<i>E<sub>self</sub></i> is the change in the self
  energy of the mutated sidechain conformations. &Delta;<i>S<sub>ema</sub></i>
  and &Delta;<i>S<sub>emd</sub></i> are the differences in the sums of the
  H&uuml;ckel theory hydrogen bond acceptor and donor strengths.
  &Delta;<i>&delta;<sub>acc</sub></i> and &Delta;<i>&delta;<sub>don</sub></i>
  are the change in number of acceptor and donor atoms due to the mutation.
  &Delta;<i>&delta;<sub>rot</sub></i> quantifies the difference in rotatable
  bonds between the wild-type and mutated residue.
  &Delta;<i>&delta;<sub>H</sub></i> and &Delta;<i>&delta;<sub>heavy</sub></i>
  are the change in number of hydrogens and heavy atoms respectively. And
  finally &Delta;<i>&delta;<sub>gly</sub></i> is the change in number of
  glycine residues..
<a id="stability_scoring_function"></a>
</p>

<p>The stability scoring function therefore has the following functional
  form:</p>

<p class="center">&Delta;&Delta;<i>G<sub>s</sub></i> =
  <i>&alpha;</i>[&Delta;<i>E<sub>vdw</sub></i> +
  0.5(&Delta;<i>E<sub>coul</sub></i> + &Delta;<i>E<sub>sol</sub></i>)] +
  &beta;&Delta;<i>E<sub>SS</sub></i> +
  &gamma;&middot;&Delta;<i>SA<sub>sc</sub></i> +
  &epsilon;&middot;&Delta;<i>SA<sub>pol</sub></i> - &Delta;<i>G<span class=
  "subsup_f"><sub>s<sub>u</sub></sub><sup>WT&rarr;Mut</sup></span></i></p>

<p>In the Protein Design application, the stability scoring function has been
  trained on over 3000 single point mutations taken from the FoldX
  [Guerois&nbsp;2002] and PoPMuSiC-2.0 [Dehouck&nbsp;2009] datasets.</p>

<a id="Ensemble"></a>
<h2>Generation of Conformational Ensembles</h2>

<p>The ensemble option is used to generate an ensemble of protein
  conformations and to calculate average stability, affinity and protein
  properties using a Boltzmann distribution. There are three different methods
  for generating an ensemble: Rotamer Explorer, LowMode and Dynamics.</p>

<p>The <a href="../proteins/probuild.htm#RotamerExplorer">Rotamer
  Explorer</a> uses a similar search method as that used for Protein Design:
  the mutation along with residues within a cutoff are partitioned into
  sidechains atoms and an environment atom set. The conformations for residues
  in the sidechain set are read in from the rotamer library. A matrix is built
  and searched using UQO. This is followed by Monte Carlo search performed on
  the matrix around an optimal solution. The search finishes when the number of
  conformations found reaches twice the desired ensemble size.</p>

<p>The LowMode ensemble generation method uses <a href=
  "confsrch.htm#LowModeMD">LowMode MD</a> to search the conformational space of
  the mutant. It should be used when a mutation may cause larger changes in the
  backbone conformation. All atoms of the mutated residue are free to move,
  while residues whose sidechains are repacked or are within the default
  cutoff of 4.5 &Aring; are tethered (see cutoff option: 
  <a href="#ensemble_opt">Fix Residues Farther than</a>).
  By default the backbone atoms of these residues are tethered
  with a weight of 1 kcal/mol, while the sidechain atoms are not tethered.
  Both tether types can be configured in the <a href="#ensemble_opt">
  LowMode options</a> as well. Atoms of non-repacked residues outside
  the cutoff are kept fixed.</p>

<p>If the <a href="#PreserveRestraints">Preserve Restraints</a> option
  is enabled, additional restraints can be applied:
  <ul>
  <li>Atoms of residues which have all atoms fixed in the input structure
  are kept fixed. Atoms of the mutated residues are free to move.</li>
  <li>Heavy sidechain atoms of residues in which all of them are tethered
  in the input structure are tethered with their maximum tether weight.
  The atoms of the mutated residues are free to move.</li>
  <li>Existing force field restraints (distance, angle, torsions) are applied
  if an atom name match to the input structure can be established.</li>
  </ul>
</p>  
<p>
  To speed up the simulation, atoms of residues that are farther than 13 &Aring;
  (<a href="../moe/pot.htm#Forcefield">non-bonded cutoff</a> + 3&nbsp;&Aring;)
  are marked as inert unless explicit solvent is enabled in the LowMode
  options. The number of iterations is a parameter that can be modified.
  Each LowMode iteration minimizes the current conformation to a root
  mean square gradient of 1 to enable the search to escape small local
  energy wells.</p>
<p>The Molecular Dynamics ensemble generation method uses <a href=
  "md.htm">Molecular Dynamics</a> to create a conformational ensemble. It is
  recommended to use the explicit solvent option with this ensemble method.
  The Molecular Dynamics protocol uses the same tethering and fixed scheme as
  LowMode MD. A 1ps heating phase is performed followed by another 0.5ps
  heating phase in which the heavy atom restraints are gradually reduced. This
  is followed by a production phase whose length is a parameter that can be
  specified (default 50ps). All phases use a time step of 2ps. A
  conformation is saved every 5ps.</p>

<p>After an ensemble is generated, the conformations are minimized using
  the protocol described in the <a href="#Refinement">refinement</a>
  section. For conformations created by LowMode MD or Molecular
  Dynamics the same tethering and fixed scheme as described in the
  LowMode MD section above is used.</p>

<p>After minimization the conformations within an RMSD threshold of
  each other (i.e. similar conformations) are removed. The remaining
  conformations are then scored and sorted using the <a href=
  "#stability_scoring_function">stability scoring function</a>. Conformations
  outside of the energy window of the lowest energy conformation are removed.
  Additionally, the best <i>N</i> conformations, where <i>N</i> is a settable
  parameter, are kept. The affinity and protein properties of these
  conformations can optionally be calculated. All properties and scores are
  then averaged using a Boltzmann distribution.</p>

<a id="Variants"></a>
<h2>Protein Design Variants</h2>

<p>The Protein Design application has 6 variants, each tailored to solving
  different yet similar problems. In all cases, the goal of the experiment is
  to modulate some physical protein properties, such as stability or affinity,
  with each variant restricting the search in a different way. In each variant,
  the desired mutations are specified in a mutation list which can be applied
  either to chains currently loaded in the system or to a database containing a
  conformational ensemble. The following sections explain the distinct aspects
  of each of the Protein Design variants.</p>
<a id="AlanineScanMethod"></a>
<h3>Alanine Scan Methodology</h3>

<p>Alanine Scanning is a technique used to determine the importance of a
  residue to the stability, affinity or property of a given protein. Alanine is
  used because its sidechain is non-bulky and chemically inert while retaining
  secondary structure preferences similar to most amino acids.</p>

<p>The Alanine Scan variant of Protein Design sequentially mutates all
  residues, specified in the mutation list, to alanine.</p>
  <div class="floatright"><img src="prodesign/cys_distance.png" alt=
  "Protein Design Prediction of Stability - Unfolded State" class="floatright" /></div>
<a id="DisulfideScanMethod"></a>
<h3>Disulfide Scan Methodology</h3>

<p>Disulfide bonds play an important role in the folding and stability of
  proteins. An easy way to increase the stability of a protein is to add a new
  disulfide bond.</p>

<p>The Disulfide Scan variant of the Protein Design application analyzes the
  residues specified in the mutation list and determines whether there is
  another residue close enough to create a disulfide bond. Two residues are
  said to be close enough to form a disulfide bond if their &beta; carbons are
  within 5&Aring; of each other. With such a pair identified, the Disulfide
  Scan mutates both residues to Cysteine using the <a href=
  "#Searching">Sequence and Conformational Search</a> methodology described
  above.</p>

<p><br clear="all" /></p>
  <div class="floatleft"><img src="prodesign/codon_table.png" alt=
  "Codon Table" class="floatleft" /></div>
<a id="ResistanceScanMethod"></a>
<h3>Resistance Scan Methodology</h3>

<p>Resistance Scanning is a technique used to identify single point
  mutations, typically in viral and bacterial proteins, which cause a loss of
  affinity toward a ligand. This decreased affinity is also known as
  <i>resistance</i>. These single point mutations represent a low genetic
  barrier to resistance and cause the target to become resistant quicker.
</p>

<p>In Resistance Scanning, mutations are limited to single nucleotide
  polymorphisms (SNP) of the wild type sequence. A SNP mutation is where there
  is a single nucleotide change in the codon encoding an amino acid, inducing a
  change in the protein sequence. For example, if the wild-type sequence is
  Ala, the possible mutations for this residue are Asp, Glu, Gly, Pro, Ser,
  Thr, and Val. The mutations are limited to SNPs to emulate the mutations that
  are more likely to happen naturally over the evolution of the protein.</p>

<p>When using Resistance Scan, all mutations specified in the mutation list
  are created sequentially, and their affinities are calculated automatically.
  Large positive increases in affinity indicate that the target may become
  resistant to the ligand easily.</p>

<p><br clear="all" />
<a id="ResidueScanMethod"></a>
</p>

<h3>Residue Scan Methodology</h3>

<p>Residue Scanning, also known as site-directed mutagenesis, is used to
  generate mutations that may produce rationally designed proteins with
  modulated properties.</p>

<p>Residue Scanning mutates in a sequential fashion all residues specified in
  the mutation list. The mutants of a particular residue can be edited with a
  Mutation Editor. It is also possible to perform multiple point mutations
  and define distance constraints between the residues of such a multi point
  mutations.</p>
<a id="SampleSequenceMethod"></a>
<h3>Sample Sequence Methodology</h3>

<p>Sample Sequence randomly searches the sequence space, as opposed to
  enumerating all possibilities as in Residue Scanning. This provides the
  ability to search and optimize larger sequence spaces in conjunction with
  the <a href="mutanal.htm">Mutation Analysis application</a> through
  accumulation of statistics on the search space.</p>

<p>Sample Sequence mutates in a random fashion all residues specified in the
  mutation list. The mutants of a particular residue can be edited with a
  Mutation Editor. This option should be used when a mutation space becomes
  large enough to prohibit enumeration (for example, larger than
  10&nbsp;000).</p>
<a id="SequenceDesignMethod"></a>
<h3>Sequence Design Methodology</h3>

<p>When dealing with multiple point mutations, the number of possible
  mutations grows exponentially. For example, there are 8000 possible mutations
  for 3 residues if they are allowed to be any of the 20 standard amino acids.
  In these cases, it is beneficial to optimize the sequence space. This can be
  done similarly to optimizing the conformation of a protein.</p>

<p>Sequence Design is similar in principle to Residue Scan, but is meant to
  handle situations where the combinatorial explosion due to the number of
  residues and mutations reaches a point where producing all mutants is not
  feasible. Sequence Design uses the algorithm as described in <a href=
  "#Searching">Sequence and Conformational Search</a>, but unlike the other
  variants which restrict the UQO search to one mutant per residue, multiple
  mutants for each residue are allowed to compete with each other during the
  UQO search. This enables an efficient search of sequence space without the
  need to explicitly create all mutations. In essence, Sequence Design
  optimizes the sequence for a given backbone structure. Please note that when
  using Sequence Design, the wild type mutation is always allowed to compete.
  In some cases, if no better mutant is possible, the best mutant found will be
  simply the wild type.</p>
<a id="Run"></a>
<h1>Running the Protein Design Application</h1>
<a id="Preparing_the_Data"></a>
<h2>Preparing the Data</h2>

<p>The Protein Design application can operate on either a protein structure
  loaded in MOE or on a database of containing a conformational ensemble, such
  as a Molecular Dynamics trajectory. When working on a database, the first
  entry of the first molecule field in the database file must be loaded into
  MOE, and the residues to mutate specified. The structure data under
  consideration are typically crystallographic data read from a <a href=
  "../proteins/pdb_mdb.htm#OpeningPDBFiles">PDB file</a>, or obtained from the
  MOE <a href="../proteins/pdb_mdb.htm#MOEPDB">protein database</a> or directly
  from the <a href="../moe/gui/fileopen.htm#PSILODownload">PDB or PSILO</a>.
  The data might also be loaded from a <a href="../mdb/db.htm">MOE database</a>
  or pasted from the <a href="../moe/gui/mainwin.htm#Copy">clipboard</a>.</p>

<p>Structural data often contain errors. Some additional preparatory steps
  are commonly required, such as capping, completing residues with missing
  atoms and selecting appropriate alternate locations. In MOE, structural
  issues can be automatically corrected using the Structure Preparation
  application <span class="nobr">(<span class="menu">MOE | Compute | Prepare |
  Structure Preparation</span></span> or <span class="nobr"><span class=
  "menu">MOE | Protein | Structure Preparation</span>).</span> Once the panel
  is raised, press <b>Correct</b>. This will result in an attempt to correct
  all issues detected by the application. For more information on data
  preparation, please see <a href="structprep.htm">Structure
  Preparation</a>.</p>

<p>Although not required by the Protein Design application, it may be
  desirable to optimize the hydrogen bond network <span class=
  "nobr">(<span class="menu">MOE | Compute | Prepare | <a href=
  "protonate3d.htm">Protonate&nbsp;3D</a></span>)</span> and calculate charges
  <span class="nobr">(<span class="menu">MOE | Compute | Prepare | <a href=
  "../moe/pot.htm#Charges">Partial&nbsp;Charges</a></span>)</span> as optimal
  hydrogens positions and charges are necessary if any molecular mechanics
  refinement is to be done on the mutants produced. Note: If molecular
  mechanics refinements are to be performed, an appropriate molecular mechanics
  <a href="../moe/pot.htm">forcefield</a> should be selected.</p>

<p>It is also recommended to perform tethered energy minimization prior to
  using the Protein Design application to relieve bad crystallographic contacts
  or other bad geometries. Note that in this case, the <a href=
  "../apps/emin.htm">Energy Minimize</a> application itself can be used to
  adjust hydrogens and lone pairs and to calculate partial charges. Tethering,
  which prevents receptor atoms from deviating too far from their initial
  coordinates, can be done through the Energy Minimize panel. Alternatively,
  the <span class="menu">MOE | RHS | <a href=
  "../apps/ligx.htm#Prepare">QuickPrep</a></span> panel can be used to add
  tether restraints to receptor, ligand, and/or solvent atoms (<b>Receptor</b>,
  <b>Ligand</b>, <b>Solvent</b>) and then perform energy minimization on the
  system (enable <b>Refine</b>).</p>

<p>An additional step that may be taken is to delete extraneous co-factors or
  unbound water molecules. Water molecules which play structural roles within
  the protein, such as bridging waters, and important cofactors should be
  retained.</p>

<p>Finally, the <a href="progeom.htm">Protein Geometry</a> application can be
  used to assess the stereochemical quality of the protein.</p>
  <ul class="help">
    <li class="help" style="padding-left: 25px;">For more information on
    protein structure preparation, please see <a href=
    "structprep.htm">Structure Preparation</a>.</li>
  </ul>
<a id="Using_the_Panel"></a>
<h2>Using the Protein Design Panel</h2>

<p>To use the Protein Design application, select <span class="nobr"><b>MOE |
  Protein | Design | <i>Variant</i></b></span>. The following panel will
  open:</p>

<img src="prodesign/prodes_panel.png" alt=
  "Protein Design Panel" class="center" />

<p>After specifying the Protein Design options, the calculation is
started with the <b>OK</b> or <b>Build</b> button at the bottom of the
panel. <b>OK</b> will start the calculation and close the Protein
Design panel, while <b>Build</b> keeps the panel open.  The
<b>Batch</b> button will open the
<a href="../moe/hpc.htm#BatchPanel">Batch</a> job submit
panel, which allows the user to create
a batch job that can be run locally outside of MOE,
or on a compute cluster via a queueing system such as slurm:
</p>

<p>
<img src="prodesign/prodesign_batchpanel.png" alt="Batch Panel"
class="center" /></p>

<p>The following sections describe details of the Protein Design panel:</p>
  <ul>
    <li>
      <a href="#MutationEditor">Using the Mutation List Editor</a>
      <ul>
        <li><a href="#ConstantMutation">Defining a Constant Mutation</a></li>
        <li><a href="#MultipleMutations">Defining Multiple Mutations</a></li>
        <li><a href="#NonNaturalAA">Mutating to/from Non-Natural Amino
        Acids</a></li>
      </ul>
    </li>
    <li>
      <a href="#panel_variants">Using Protein Design Variants</a>
      <ul>
        <li><a href="#AlanineScan">Alanine Scan</a></li>
        <li><a href="#DisulfideScan">Disulfide Scan</a></li>
        <li><a href="#ResistanceScan">Resistance Scan</a></li>
        <li><a href="#ResidueScan">Residue Scan</a></li>
        <li><a href="#SampleSequence">Sample Sequence</a></li>
        <li><a href="#SequenceDesign">Sequence Design</a></li>
      </ul>
    </li>
    <li><a href="#panel_input">Input/Output Options</a></li>
    <li><a href="#display_opt">Display Options</a></li>
    <li><a href="#ensemble_opt">Ensemble Options</a></li>
    <li><a href="#affinity_opt">Affinity Options</a></li>
    <li><a href="#properties_opt">Properties Options</a></li>
  </ul><!-- Mutation Editor -->
<a id="MutationEditor"></a>
<h3>Using the Mutation List Editor</h3>

<p>There are two variations of the <b>Mutation List Editor</b>: one for
  defining a constant mutation, and the other for defining multiple
  mutations.</p>
<a id="ConstantMutation"></a>
<h4>Defining a Constant Mutation</h4>

<p>The first variation of the <b>Mutation List Editor</b> is used for
  <a href="#AlanineScan">Alanine Scanning</a>, <a href=
  "#DisulfideScan">Disulfide Scanning</a>, and <a href=
  "#ResistanceScan">Resistance Scanning</a>. For these three <a href=
  "#Variants">Protein Design Variants</a>, the mutation for each residue is
  constant and therefore only the following options are available. Note: When
  no mutations are listed for a residue, that residue will not be used for the
  search.</p>

<img src="prodesign/panel_edit1.png" alt=
  "Protein Design Edit Options" class="center" />
  <table class="noborder">
    <tr>
      <td><b>Clear All</b></td>
      <td>Remove mutations for all
      residues in the <b>Mutation List</b>.</td>
    </tr>
    <tr>
      <td><b>Mutate All</b></td>
      <td>Add default mutations to
      residues that have no mutations listed in their <b>Mutation</b>
      column.</td>
    </tr>
    <tr>
      <td valign="top"><b>Clear/Mutated</b></td>
      <td valign="top">If a residue cannot be mutated, it will be identified in
      the bubble help of the <b>Clear/Mutated</b> button.</td>
    </tr>
  </table>
<a id="MultipleMutations"></a>
<h4>Defining Multiple Mutations</h4>

<p>The second variation of the <b>Mutation List Editor</b> is used for
  <a href="#ResidueScan">Residue Scanning</a>, <a href="#SampleSequence">Sample
  Sequence</a>, and <a href="#SequenceDesign">Sequence Design</a>. For these
  three <a href="#Variants">Protein Design Variants</a>, the mutation list can
  be edited using the mutation list editor. Note: When no mutations are listed
  for a residue, that residue will not be used for the search.</p>

<img src="prodesign/panel_edit2.png" alt=
  "Protein Design Edit Options" class="center" />
  <table class="noborder">
    <tr>
      <td><b>Clear/Mutate</b></td>
      <td>Remove mutations from, or
      add mutations to, residues selected in the <b>Mutation List</b>.</td>
    </tr>
    <tr>
      <td><b>Edit Box</b></td>
      <td>Manually define a
      mutation using either a sequence of single letter codes (such as
      <tt>AKW</tt>) or a list of three letter codes separated by a space or
      comma (such as <tt>Arg Lys Trp</tt>). Three letter codes must be entered
      with the first letter capitalized and the other two letters lowercase;
      for example, &quot;Arg&quot;, but not &quot;ARG&quot; or &quot;arg&quot;.
      Non-natural amino acids must be defined in the database chosen as the
      Rotamer Library and specified using its full name in parentheses (such as
      <tt>A,(HYP),K</tt>). In addition to the 20 standard amino acids
      (<tt>Xxx</tt>), other supported mutations include <tt>J</tt> (Non-polar),
      <tt>O</tt> (Polar), <tt>B</tt> (<tt>N</tt>, <tt>D</tt>) and <tt>Z</tt>
      (<tt>Q</tt>, <tt>E</tt>).</td>
    </tr>
    <tr>
      <td><b>Apply</b></td>
      <td>Apply the mutation in the
      edit box to the residues selected in the <b>Mutation List</b>.</td>
    </tr>

    <tr>
      <td><a id="LoadMutationFile"></a><b>Load Mutation File</b></td>
      <td>Pressing <img src="prodesign/ico_browse.png" alt=
       "Browse" class="icon" /> will open a file browser to
       select a file which specifies the mutations to be done.
       This is especially useful if one has a list of specific
       mutated sequences which one wants to generate with the
       <a href="#SiteLimit0">Site Limit = 0</a> option.
       Three file formats are supported: <tt>Fasta</tt>, <tt>raw</tt>,
       and <tt>mutation expression</tt>.
       <p>
       The sequences with the mutations in the <tt>fasta</tt> or <tt>raw</tt>
       file must have the same length as the protein chain of the input
       structure. If the input consists of multiple chains
       (multimer) the chain names in the file must match the input
       structure or be the input chain number, so that the sequences can be
       associated correctly. Chain names are specified in <tt>fasta</tt> files
       in the header line which starts with a <tt>"&gt;"</tt> character.
       In the raw file format, each line specifies one sequence and the
       chain name is separated at the beginning of the line by a <tt>"\t"</tt>
       tab character from the sequence. <tt>Fasta</tt> files only support
       the standard 20 amino acids as mutations, while in <tt>raw</tt> files
       non-standard amino acids can be specified by their 3-letter code
       within round brackets <tt>"()"</tt>.
       </p>
       <p>
       The <tt>mutation expression</tt> format specifies the mutations of each
       sequence in one line. The mutation expressions are separated by
       commas and have the following form: E.g.
       1:Y61N (1 = Chain number, Y61 = residue to mutate, N = mutation).
       3-letter residue names can specified in round brackets.
       The following example demonstrates the three file formats,
       with the mutations colored <span style="color:red">red</span> for clarity:
       </p>
       <p>
       <tt>fasta</tt> file:
<pre>
>1CRN.A
TTCCP<span style="color:red">GGIG</span>RSNFNVCRLPGTPEAICATYTGCIIIPGATCPGDYAN
>1CRN.A
TTCCP<span style="color:red">TSLS</span>RSNFNVCRLPGTPEAICATYTGCIIIPGATCPGDYAN
>1CRN.A
TTCCP<span style="color:red">L</span>IV<span style="color:red">V</span>RSNFNVCRLPGTPEAICATYTGCIIIPGATCPGDYAN
</pre>
       </p>
       <p>
       <tt>raw</tt> file:
<pre>
1CRN.A	TTCCP<span style="color:red">GGIG</span>RSNFNVCRLPGTPEAICATYTGCIIIPGATCPGDYAN
1CRN.A	TTCCP<span style="color:red">TSLS</span>RSNFNVCRLPGTPEAICATYTGCIIIPGATCPGDYAN
1CRN.A	TTCCP<span style="color:red">L</span>IV<span style="color:red">V</span>RSNFNVCRLPGTPEAICATYTGCIIIPGATCPGDYAN
</pre>
       </p>
       <p>
       <tt>mutation expression</tt> file:
<pre>
1:S6<span style="color:red">G</span> 1:I7<span style="color:red">G</span> 1:V8<span style="color:red">I</span> 1:A9<span style="color:red">G</span>
1:S6<span style="color:red">T</span> 1:I7<span style="color:red">S</span> 1:V8<span style="color:red">L</span> 1:A9<span style="color:red">S</span>
1:S6<span style="color:red">L</span> 1:A9<span style="color:red">V</span>
</pre>
       </p>
      .</td>
    </tr>

    <tr>
      <td>
<a id="MutationEditorShortcut"></a>
<b>Mutation
      Editor Shortcut</b></td>
      <td>
        Pressing <img src="prodesign/panel_edit2_shortcut.png" alt=
        "Mutation Editor Shortcut" class="icon" /> will open the <b>Mutation
        Editor Shortcut</b> panel. This shortcut provides an easy way to
        browse, add, and remove mutations for the residues selected in the
        <b>Mutation List</b>. By default, the <b>Mutation Editor Shortcut</b>
        will select all of the common amino acids defined as mutations for each
        of the residues, and will start with the <b>Class</b> filter set to
        <i>All</i>. Selecting/deselecting an amino acid will add/remove it from
        the list of mutations for the selected residues in the <b>Mutation
        List</b>.
        <table class="noborder">
          <tr>
            <td style="padding: 1em;"><img src=
            "prodesign/prodes_edit_popup.png" alt=
            "Edit Short Cut" /></td>
            <td>

<p class="center"></p>
              <table class="noborder">
                <tr>
                  <td>
                  <b>Class</b></td>
                  <td>Restrict the
                  display of amino acids in the amino acid list to a particular
                  amino acid class. The <i>All</i> option displays all
                  classes.</td>
                </tr>
                <tr>
                  <td>
                  <b>Filter</b></td>
                  <td>
                    Filter the display of amino acids in the list. If no filter
                    is specified, then no restriction is made other than the
                    <b>Class</b> restriction. If a filter is specified, then in
                    addition to the restrictions imposed by <b>Class</b>, only
                    those lines in the list with text containing each of the
                    words in the <b>Filter</b> text field will be displayed.

<p>For example, if the filter text is &quot;Standard&quot;,
                    the display is restricted to lines that contain the
                    consecutive letters (case insensitive)
                    &quot;standard&quot;. If the filter text is &quot;Standard
                    Polar&quot;, the display is restricted to lines that
                    contain the substring &quot;standard&quot; as well as the
                    substring &quot;polar&quot; (not necessarily in that
                    order).</p>

<p>Typing in the <b>Filter</b> text field will
                    automatically update the list. The red <b>X</b> button
                    deletes all characters from the <b>Filter</b> text.</p>
                  </td>
                </tr>
                <tr>
                  <td>
                  <b>Select</b></td>
                  <td>
                    Select amino acids in the listbox that correspond to the
                    chosen category. If the red <b>x</b> is pressed, it will
                    deselect the amino acids that correspond to that category.
                    <ul>
                      <li><b>Reset</b>: Sets the selection in the listbox to
                      the initial selection when the <b>Mutation Editor
                      Shortcut</b> opened.</li>
                      <li><b>Aromatic</b>: Selects aromatic amino acids (Phe,
                      Tyr, Trp).</li>
                      <li><b>Polar</b>: Selects polar amino acids (Arg, Asn,
                      Asp, Cys, Gln, Glu, His, Lys, Ser, Thr).</li>
                      <li><b>Hydrophobic</b>: Selects hydrophobic amino acids
                      (Ala, Gly, Ile, Leu, Met, Phe, Pro, Trp, Tyr, Val).</li>
                      <li><b>Positive</b>: Selects positively charged amino
                      acids (Arg, Lys).</li>
                      <li><b>Negative</b>: Selects negatively charged amino
                      acids (Asp, Glu).</li>
                      <li><b>Self</b>: Selects the respective residue type(s)
                      to do a self mutation.</li>
                      <li><b>SNP</b>: Mutate to amino acids represented by
                      single nucleotide polymorphisms of the current amino
                      acid type.</li>
                    </ul>
                  </td>
                </tr>
              </table>
            </td>
          </tr>
        </table>
      </td>
    </tr>

  </table><!-- Non-Natural Amino Acids -->
<a id="NonNaturalAA"></a>
<h4>Mutating to/from Non-Natural Amino Acids</h4>

<p>If the parent residue or mutation is a non-natural amino acid it must be
  added to the Rotamer Library. The <a href="../proteins/buildrotlib.htm">Build
  Rotamer Library</a> application can be used to create MDB files which can
  then be added to the Rotamer Library (see <b>MOE | Settings
  | Protein Defaults | Rotamer Library</b>). After a non-natural amino acid
  is added to the Rotamer Library, the Protein Design application will need to
  be restarted.</p>

<p>Non-natural amino acids will appear in the Editor Shortcut when the class
  is changed to <i>All</i>. Mutations to a non-natural amino acid can be
  performed via the Mutation List, by first selecting the residue to be mutated
  and then editing the mutation string in the mutation list editor. Non-natural
  amino acids must be specified using the residue name (rName) in parentheses;
  for example, <tt>A,(HYP),K</tt>.</p><!-- Protein Design Variants -->
<a id="panel_variants"></a>
<h3>Using Protein Design Variants</h3>

<p>The Protein Design application has 6 different variants each of which is
  optimized for a specific application. They can be selected by clicking on the
  desired variant. For more information, see <a href="#Variants">Protein Design
  Variants</a>. A site is defined as the residues to be mutated. For example,
  in the following mutation &quot;Asp24[Ala;Ser], Ala25[Asp,Ser]&quot;, the
  site would be defined as Asp24 and Ala25. A mutation combination refers to
  the specific mutations for a site. For the example &quot;Asp24[Ala,Ser],
  Ala25[Asp,Ser]&quot;, there would be 4 mutation combinations: &quot;Asp24Ala,
  Ala25Asp&quot;; &quot;Asp24Ser, Ala25Asp&quot;; &quot;Asp24Ala,
  Ala25Ser&quot;; &quot;Asp24Ser, Ala25Ser&quot;.</p>
  <div class="floatleft"><img src="prodesign/prodes_variants.png" alt=
  "Protein Design Variants" style="padding: 1em;" class="floatleft" /></div>
  <ul>
    <li>
      <a href="#AlanineScan"><i>Alanine Scan</i></a> <span class=
      "nobr">(<span class="menu">MOE | Protein | Design | Alanine
      Scan</span>)</span>

<p>All residues displayed in the mutation list will be mutated to
      Alanine, as shown in the Mutation column in the list. All mutations are
      done sequentially, also known as single point mutations.</p>
    </li>
    <li>
      <a href="#DisulfideScan"><i>Disulfide Scan</i></a> <span class=
      "nobr">(<span class="menu">MOE | Protein | Design | Disulfide
      Scan</span>)</span>

<p>All residues displayed in the mutation list will be assessed for the
      possibility of creating a disulfide bond with a nearby residue. If
      disulfide bond creation is possible, the implicated residues and their
      partner(s) will be mutated to cysteine systematically, followed by an
      attempt to create the disulfide bonds.</p>
    </li>
    <li>
      <a href="#ResistanceScan"><i>Resistance Scan</i></a> <span class=
      "nobr">(<span class="menu">MOE | Protein | Design | Resistance
      Scan</span>)</span>

<p>Single nucleotide polymorphism (SNP) mutations will be performed on
      residues displayed in the mutation list. All mutations will be single
      point mutations and will be built sequentially.</p>
    </li>
    <li>
      <a href="#ResidueScan"><i>Residue Scan</i></a> <span class=
      "nobr">(<span class="menu">MOE | Protein | Design | Residue
      Scan</span>)</span>

<p>Mutations listed in the mutation column of the mutation list will be
      performed on residues displayed in the mutation list. Mutations for a
      given residue can be edited using the Mutation List Editor. The
      permissible number of concurrent N point mutations can be controlled
      using the Site Limit option. All mutation combinations, as defined by the
      Mutation list and Site Limit will be created and output.</p>
    </li>
    <li>
      <a href="#SampleSequence"><i>Sample Sequence</i></a> <span class=
      "nobr">(<span class="menu">MOE | Protein | Design | Sample
      Sequence</span>)</span>

<p>Mutations listed in the mutation column of the mutation list will be
      used to define the mutation space for residues displayed in the mutation
      list. When using the Sample Sequence variant, the mutation space will be
      randomly searched, creating the number of mutants specified by Sequences.
      Mutations for a given residue can be edited using the Mutation List
      Editor. The permissible number of concurrent N point mutations can be
      controlled using the Site Limit option.</p>
    </li>
    <li>
      <a href="#SequenceDesign"><i>Sequence Design</i></a> <span class=
      "nobr">(<span class="menu">MOE | Protein | Design | Sequence
      Design</span>)</span>

<p>Mutations listed in the mutation column of the mutation list will be
      performed on residues displayed in the mutation list. Mutations for a
      given residue can be edited using the Mutation List Editor. The
      permissible number of concurrent N-point mutations can be controlled
      using the Site Limit option. Unlike in Residue Scan, here the sequence is
      optimized, with mutations being allowed to compete with each other for a
      specific site instead of all possible mutations being produced. Only the
      best scoring mutations per mutation site will be output. This is
      useful if the number of combinations is too high for the Residue Scan
      method.</p>
    </li>
  </ul>

<p><br clear="all" />
  <!-- Alanine Scan -->
<a id="AlanineScan"></a>
</p>

<h4>Alanine Scan</h4>

<p>For more information on input options for Alanine Scan, please see
  <a href="#panel_input">Input/Output Options</a>. For details concerning the
  Alanine Scan method, please see its <a href=
  "#AlanineScanMethod">Methodology</a>.</p>

<h5>Mutation List and Display Options</h5>
  <table class="noborder">
    <tr>
      <td valign="top"><b>Mutation List</b></td>
      <td valign="top">
        All residues as defined by the <a href="#display_opt">Display
        Options</a> appear in this list. For each residue, the following
        details are given:
        <ul>
          <li><b>Chain</b>: Chain number and name.</li>
          <li><b>Res</b>: Residue Type.</li>
          <li><b>UID</b>: Residue UID and Insertion code.</li>
          <li><b>Mutation</b>: If this field is blank, the residue will not be
          mutated. An &quot;A&quot; marks the residue for mutation to alanine.
          If no mutations are listed for a residue, no mutation will be
          applied.</li>
        </ul>
        <p>When the <a href="#display_opt">display option</a> <b>Contour in
        MOE Window</b> is enabled, a single click in the <b>Mutation List</b>
        will display the contours for the selected residues in the list. A
        double click will toggle the selection state of the associated atoms of
        the selected residues.</p>
      </td>
    </tr>
  </table>

<h5>Mutation Editor</h5>

<img src="prodesign/panel_edit1.png" alt=
  "Protein Design Edit Options" class="center" />

<p>The <a href="#ConstantMutation">first variation</a> of the <b>Mutation
  List Editor</b> is used for Alanine Scanning.</p>

<h5>Generation Options</h5>
  <table class="noborder">
    <tr>
      <td><b>Site Limit</b></td>
      <td>A site refers to the
      residues being mutated. <b>Site Limit</b> gives the maximum number of
      residues which may constitute a mutation site. In Alanine Scan, <b>Site
      Limit</b> is fixed at a value of 1. To allow for multi-point alanine
      mutations, use <a href="#ResidueScan">Residue Scan</a>.</td>
    </tr>
    <tr>
      <td><b>Mutation
      Space</b></td>
      <td>
        This is the number of mutations that will be searched. The value will
        change color depending on the size of the space:
        <ul>
          <li>Yellow = 0</li>
          <li>Green &gt; 0 and &lt; 10,000,000</li>
          <li>Red &ge; 10,000,000</li>
        </ul>When the mutation space is large, it is recommended to use
        <a href="#SequenceDesign">Sequence Design</a>.
      </td>
    </tr>
    <tr>
      <td><b>Settings</b></td>
      <td>
              Pressing the Settings icon <img src="prodesign/settings.png" alt=
              "settings" style="vertical-align: middle" /> will raise the popup
              menu shown below.
<img src="prodesign/config_alanine_scan.png" alt="options" class="center" />
              <ul>
                <li><b>Repack environment</b>. When enabled, residues within
                &quot;Cutoff&quot; of the mutation will be included in
                conformational search.</li>
                <li><b>Cutoff</b>. The distance (&Aring;) used to determine
                which residues are added to the conformational search in
                addition to the residue being mutated.</li>
                <li><b>Alternate protomers</b>. Alternate protonation states of
                residues to be mutated are considered when <b>Repack
                environment</b> is enabled. For example, when enabled and
                mutating to Asp both the charged and neutral forms are allowed
                to compete against each other. Disabling would allow only the
                charged form. This option is by default disabled.</li>
                <li><b>pH</b>. Specifies the <i>pH</i> of the system when
                performing the mutation. This, along with <a href=
                "../apps/residuepka.htm">PROPKA</a>, is used for scoring
                alternate protonation states to account for local
                <i>pK<sub>a</sub></i> shifts.</li>
                <li><b>Refine Mutations</b>. When enabled, the mutations will
                be refined following the conformational search using UQO. For
                more information on the refinement protocol, see <a href=
                "#Searching">Searching Conformation and Sequence Space</a> and 
                <a href="#Refinement">Refining Mutations Generated by UQO</a>.
                This option is enabled by default.</li>
                <li><b>RMS Gradient</b>. If <b>Refine Mutations</b> is
                enabled, the final refinement will be performed until the RMS
                of the gradient of the potential energy is less than a
                specified value.</li>
                <li>
                <b><a id="PreserveRestraints"></a>Preserve Restraints</a></b>.
                Try to preserve existing fixed states, tethers, and restraints
                during refinement and ensemble calculations:
                Hold all-atom-fixed evironment residues fixed,
                apply tethers of all-tethered environment sidechains,
                and apply existing force field restraints.
                The all-fixed, all-tethered, and restrained residues
                are excluded from the repacking of the environment.
                See further details in the
                <a href="#Refinement">refinement</a> and
                <a href="#Ensemble">ensemble</a> section.
                </li>
              </ul>
      </td>
    </tr>
  </table><!-- Disulfide Scan -->
<a id="DisulfideScan"></a>
<h4>Disulfide Scan</h4>

<p>For more information on input options for Disulfide Scan, please see
  <a href="#panel_input">Input/Output Options</a>. For details concerning the
  Disulfide Scan method, please see its <a href=
  "#DisulfideScanMethod">Methodology</a>.</p>

<h5>Mutation List and Display Options</h5>
  <table class="noborder">
    <tr>
      <td><b>Mutation List</b></td>
      <td>
        All residues as defined by the <a href="#display_opt">Display
        Options</a> appear in this list. For each residue, the following
        details are given:
        <ul>
          <li><b>Chain</b>: Chain number and name.</li>
          <li><b>Res</b>: Residue Type.</li>
          <li><b>UID</b>: Residue UID and Insertion code.</li>
          <li><b>Mutation</b>: Shows the possible disulfide bond partners for a
          given residue. The chain is separated from the partners by a colon
          (:). For example, 1:S24C specifies that Serine 24 of Chain 1 will be
          mutated to Cysteine. If no mutations are listed for a residue, no
          mutation will be done.</li>
        </ul>
        <p>When the <a href="#display_opt">display option</a> <b>Contour in
        MOE Window</b> is enabled, a single click in the <b>Mutation List</b>
        will display the contours for the selected residues in the list. A
        double click will toggle the selection state of the associated atoms of
        the selected residues.</p>
      </td>
    </tr>
  </table>

<h5>Mutation Editor</h5>

<img src="prodesign/panel_edit1.png" alt=
  "Protein Design Edit Options" class="center" />

<p>The <a href="#ConstantMutation">first variation</a> of the <b>Mutation
  List Editor</b> is used for Disulfide Scanning.</p>

<h5>Generation Options</h5>
  <table class="noborder">
    <tr>
      <td><b>Site Limit</b></td>
      <td>A site refers to the
      residues being mutated. <b>Site Limit</b> gives the maximum number of
      residues which may constitute a mutation site. In Disulfide Scan, <b>Site
      Limit</b> has a default value of 2, signifying that a pair of
      residues is mutated to cysteine, potentially forming a new disulfid bond.
      If set to 4 or 6, multiple (2 or 3) pair mutations will be done at the
      same time, i.e. the possible double or triple combination of the
      residue pair mutations are created as well. Thus exploring how multi pair
      mutations might influence the forming of disulfide bonds.
      <p>To allow for even more complex varying multi-point mutations,
      including Disulfide mutations, use
      <a href="#ResidueScan">Residue Scan</a>.</td>
    </tr>
    <tr>
      <td><b>Mutation
      Space</b></td>
      <td>
        This is the number of mutations that will be searched. The value will
        change color depending on the size of the space:
        <ul>
          <li>Yellow = 0</li>
          <li>Green &gt; 0 and &lt; 10,000,000</li>
          <li>Red &ge; 10,000,000</li>
        </ul>When the mutation space is large, it is recommended to use
        <a href="#SequenceDesign">Sequence Design</a>.
      </td>
    </tr>
    <tr>
      <td><b>Settings</b></td>
      <td style="vertical-align:top;">
              Pressing the Settings icon <img src="prodesign/settings.png" alt=
              "settings" style="vertical-align: middle" /> will raise the popup
              menu shown below.
<img src="prodesign/config_disulfide_scan.png" alt="options" class="center" />
              <ul>
                <li><b>Repack environment</b>. When enabled, residues within
                <b>Cutoff</b> of the mutation will be included in the
                conformational search.</li>
                <li><b>Cutoff</b>. The distance (in &Aring;) used to determine
                which residues are added to the conformational search in
                addition to the residue being mutated.</li>
                <li><b>Alternate protomers</b>. When enabled, alternate
                protonation states of residues to be mutated are considered.
                For example, when enabled and mutating to Asp both the charged
                and neutral forms are allowed to compete against each other.
                Disabling would allow only the charged form. By default, this
                option is disabled.</li>
                <li><b>pH</b>. Specifies the <i>pH</i> of the system when
                performing the mutation. This, along with <a href=
                "../apps/residuepka.htm">PROPKA</a>, is used for scoring
                alternate protonation states to account for local
                <i>pK<sub>a</sub></i> shifts.</li>
                <li><b>Refine Mutations</b>. When enabled, the mutations will
                be refined, using UQO, following the conformational search. For
                more information on the refinement protocol, see <a href=
                "#Searching">Searching Conformation and Sequence Space</a> and 
                <a href="#Refinement">Refining Mutations Generated by UQO</a>.
                This option is by default enabled.</li>
                <li><b>RMS Gradient</b>. Specifies the cutoff RMS of the
                gradient of the potential energy. If <b>Refine Mutations</b> is
                enabled, the final refinement will be performed until the RMS
                of the gradient of the potential energy is less than this
                specified value.</li>
                <li><b>Preserve Restraints</b>.
                Try to preserve existing fixed states, tethers, and restraints
                during refinement and ensemble calculations:
                Hold all-atom-fixed evironment residues fixed,
                apply tethers of all-tethered environment sidechains,
                and apply existing force field restraints.
                The all-fixed, all-tethered, and restrained residues
                are excluded from the repacking of the environment.
                See further details in the
                <a href="#Refinement">refinement</a> and
                <a href="#Ensemble">ensemble</a> section.
                </li>
                <li><b>Inter-Chain Only</b>. If enabled, only mutations which
                can result in disulfide bridges between two different protein
                chains are performed (e.g. in a protein-protein interface).
                This option also affects the list of possible disulfide
                partners in the <b>Mutation List</b>. If enabled, only possible
                disulfide bride partners in different chains are listed.</li>
                <li><b>Disulfide Cutoff</b>. Specifies the maximum distance
                allowed between the &beta; carbon atoms of residues to be
                included in the Mutation List of the Disulfide Scan.</li>
                <li><b>Exclude Adjacent</b>. If enabled, residues which are
                direct neighbors in the sequence chain are excluded from the
                Mutation List and Disulfide Scan. </li>
              </ul>
      </td>
    </tr>
  </table><!-- Resistance Scan -->
<a id="ResistanceScan"></a>
<h4>Resistance Scan</h4>

<p>For more information on input options for Resistance Scan, please see
  <a href="#panel_input">Input/Output Options</a>. For details concerning the
  Resistance Scan method, please see its <a href=
  "#ResistanceScanMethod">Methodology</a>.</p>

<h5>Mutation List and Display Options</h5>
  <table class="noborder">
    <tr>
      <td><b>Mutation List</b></td>
      <td>
        All residues as defined by the <a href="#display_opt">Display
        Options</a> appear in this list. For each residue, the following
        details are given:
        <ul>
          <li><b>Chain</b>: Chain number and name.</li>
          <li><b>Res</b>: Residue Type.</li>
          <li><b>UID</b>: Residue UID and Insertion code.</li>
          <li><b>Mutation</b>: List of possible mutations, given as single
          letter residue codes. Only <a href="#ResistanceScan">single
          nucleotide polymorphism</a> (SNP) mutations are allowed. If no
          mutations are listed for a residue, no mutation will be done.</li>
        </ul>
        <p>When the <a href="#display_opt">display option</a> <b>Contour in
        MOE Window</b> is enabled, a single click in the <b>Mutation List</b>
        will display the contours for the selected residues in the list. A
        double click will toggle the selection state of the associated atoms of
        the selected residues.</p>
      </td>
    </tr>
  </table>

<h5>Mutation Editor</h5>

<img src="prodesign/panel_edit1.png" alt=
  "Protein Design Edit Options" class="center" />

<p>The <a href="#ConstantMutation">first variation</a> of the <b>Mutation
  List Editor</b> is used for Resistance Scanning.</p>

<h5>Generation Options</h5>
  <table class="noborder">
    <tr>
      <td><b>Site Limit</b></td>
      <td>A site refers to the
      residues being mutated. <b>Site Limit</b> gives the maximum number of
      residues which may constitute a mutation site. In Resistance Scan,
      <b>Site Limit</b> is fixed at a value of 1. To allow for multi-point
      Resistance Scan mutations, use <a href="#ResidueScan">Residue
      Scan</a>.</td>
    </tr>
    <tr>
      <td valign="top"><b>Mutation Space</b></td>
      <td valign="top">
        This is the number of mutations that will be searched. The value will
        change color depending on the size of the space:
        <ul>
          <li>Yellow = 0</li>
          <li>Green &gt; 0 and &lt; 10,000,000</li>
          <li>Red &ge; 10,000,000</li>
        </ul>When the mutation space is large, it is recommended to use
        <a href="#SequenceDesign">Sequence Design</a>.
      </td>
    </tr>
    <tr>
      <td><b>Settings</b></td>
      <td>
              Pressing the Settings icon <img src="prodesign/settings.png" alt=
              "settings" style="vertical-align: middle" /> will raise the popup
              menu shown below.
<img src="prodesign/config_resistance_scan.png" alt="options" class="center" />
              <ul>
                <li><b>Repack environment</b>. When enabled, residues within
                <b>Cutoff</b> of the mutation will be included in the
                conformational search.</li>
                <li><b>Cutoff</b>. The distance (in &Aring;) used to determine
                which residues are added to the conformational search in
                addition to the residue being mutated.</li>
                <li><b>Alternate protomers</b>. When enabled, alternate
                protonation states of residues to be mutated are considered.
                For example, when enabled and mutating to Asp both the charged
                and neutral forms are allowed to compete against each other.
                Disabling would allow only the charged form. By default, this
                option is disabled.</li>
                <li><b>pH</b>. Specifies the <i>pH</i> of the system when
                performing the mutation. This, along with <a href=
                "../apps/residuepka.htm">PROPKA</a>, is used for scoring
                alternate protonation states to account for local
                <i>pK<sub>a</sub></i> shifts.</li>
                <li><b>Refine Mutations</b>. When enabled, the mutations will
                be refined, using UQO, following the conformational search. For
                more information on the refinement protocol, see <a href=
                "#Searching">Searching Conformation and Sequence Space</a>.  and 
                <a href="#Refinement">Refining Mutations Generated by UQO</a>.
                This option is by default enabled.</li>
                <li><b>RMS Gradient</b>. Specifies the cutoff RMS of the
                gradient of the potential energy. If <b>Refine Mutations</b> is
                enabled, the final refinement will be performed until the RMS
                of the gradient of the potential energy is less than this
                specified value.</li>
                <li><b>Preserve Restraints</b>.
                Try to preserve existing fixed states, tethers, and restraints
                during refinement and ensemble calculations:
                Hold all-atom-fixed evironment residues fixed,
                apply tethers of all-tethered environment sidechains,
                and apply existing force field restraints.
                The all-fixed, all-tethered, and restrained residues
                are excluded from the repacking of the environment.
                See further details in the
                <a href="#Refinement">refinement</a> and
                <a href="#Ensemble">ensemble</a> section.
                </li>
              </ul>
      </td>
    </tr>
  </table><!--Residue Scan -->
<a id="ResidueScan"></a>
<h4>Residue Scan</h4>

<p>For more information on input options for Residue Scan, please see
  <a href="#panel_input">Input/Output Options</a>. For details concerning the
  Residue Scan method, please see its <a href=
  "#ResidueScanMethod">Methodology</a>.</p>

<h5>Mutation List and Display Options</h5>
  <table class="noborder">
    <tr>
      <td><b>Mutation List</b></td>
      <td>
        All residues as defined by the <a href="#display_opt">Display
        Options</a> appear in this list. For each residue, the following
        details are given:
        <ul>
          <li><b>Chain</b>: Chain number and name.</li>
          <li><b>Res</b>: Residue Type.</li>
          <li><b>UID</b>: Residue UID and Insertion code.</li>
          <li><b>Mutation</b>: List of possible mutations, given as single
          letter residue codes for the 20 standard amino acids. For non-natural
          amino acids, the full residue name within parentheses is required;
          For example, <tt>(SEP)</tt> specifies phosphorylated serine. If no
          mutations are listed for a residue, no mutation will be carried
          out.</li>
        </ul>
        <p>When the <a href="#display_opt">display option</a> <b>Contour in
        MOE Window</b> is enabled, a single click in the <b>Mutation List</b>
        will display the contours for the selected residues in the list. A
        double click will toggle the selection state of the associated atoms of
        the selected residues.</p>
      </td>
    </tr>
  </table>

<p>To create a single multi-point mutation using the Residue Scan variant,
  first ensure that all residues listed in <b>Mutation List</b> have only one
  mutation. If this condition is met, <b>Site Limit</b> can be set to 0, which
  enables the creation of the exact mutation described in <b>Mutation List</b>.
  For example, if 2 residues are present in <b>Mutation List</b> and these
  residues have only one mutation each, then, when <b>Site Limit</b> is set to
  0, only a single 2-point mutant (as described by the mutation) will be
  created in addition to the wild type protein.</p>

<h5>Mutation Editor</h5>

<img src="prodesign/panel_edit2.png" alt=
  "Protein Design Edit Options" class="center" />

<p>The <a href="#MultipleMutations">second variation</a> of the <b>Mutation
  List Editor</b> is used for Residue Scanning.</p>

<a id="SiteLimit0"></a>
<h5>Generation Options</h5>
  <table class="noborder">
    <tr>
      <td><b>Site Limit</b></td>
      <td>A site refers to the
      residues being mutated. <b>Site Limit</b> gives the maximum number of
      residues which may constitute a mutation site. For Residue Scan, all
      sites of sizes 1 to <b>Site Limit</b> will be generated.
      <p>If the special value 0 is specified, the exact mutations specified
      for all sites in the <b>Mutation List</b> are created. In this case,
      the number of mutations for each site must be equal.
      A list of mutation sequences can be
      <a href="#LoadMutationFile">loaded from a file</a>
      with the <img src="prodesign/ico_browse.png" class="icon" /> button.
      </p></td>
    </tr>
    <tr>
      <td valign="top"><b>Mutation Space</b></td>
      <td valign="top">
        This is the number of mutations that will be searched. The value will
        change color depending on the size of the space:
        <ul>
          <li>Yellow = 0</li>
          <li>Green &gt; 0 and &lt; 10,000,000</li>
          <li>Red &ge; 10,000,000</li>
          <li>Cyan: Upper estimate with enabled <b>Pair Constraint</b>.
        </ul>When the mutation space is very large, it is recommended to use
        the <a href="#SampleSequence">Sample Sequence variant</a> or <a href=
        "#SequenceDesign">Sequence Design variant</a>.
      </td>
    </tr>
    </tr>
    <tr>
      <td><b>Settings</b></td>
      <td>
              Pressing the Settings icon <img src="prodesign/settings.png" alt=
              "settings" style="vertical-align: middle" /> will raise the popup
              menu shown below.
<img src="prodesign/config_residue_scan.png" alt="options" class="center" />
              <ul>
                <li><b>Repack environment</b>. When enabled, residues within
                <b>Cutoff</b> of the mutation will be included in the
                conformational search.</li>
                <li><b>Cutoff</b>. The distance (in &Aring;) used to determine
                which residues are added to the conformational search in
                addition to the residue being mutated.</li>
                <li><b>Alternate protomers</b>. When enabled, alternate
                protonation states of residues to be mutated are considered.
                For example, when enabled and mutating to Asp both the charged
                and neutral forms are allowed to compete against each other.
                Disabling would allow only the charged form. By default, this
                option is disabled.</li>
                <li><b>pH</b>. Specifies the <i>pH</i> of the system when
                performing the mutation. This, along with <a href=
                "../apps/residuepka.htm">PROPKA</a>, is used for scoring
                alternate protonation states to account for local
                <i>pK<sub>a</sub></i> shifts.</li>
                <li><b>Refine Mutations</b>. When enabled, the mutations will
                be refined, using UQO, following the conformational search. For
                more information on the refinement protocol, see <a href=
                "#Searching">Searching Conformation and Sequence Space</a> and 
                <a href="#Refinement">Refining Mutations Generated by UQO</a>.
                This option is by default enabled.</li>
                <li><b>RMS Gradient</b>. Specifies the cutoff RMS of the
                gradient of the potential energy. If <b>Refine Mutations</b> is
                enabled, the final refinement will be performed until the RMS
                of the gradient of the potential energy is less than this
                specified value.</li>
                <li><b>Preserve Restraints</b>.
                Try to preserve existing fixed states, tethers, and restraints
                during refinement and ensemble calculations:
                Hold all-atom-fixed evironment residues fixed,
                apply tethers of all-tethered environment sidechains,
                and apply existing force field restraints.
                The all-fixed, all-tethered, and restrained residues
                are excluded from the repacking of the environment.
                See further details in the
                <a href="#Refinement">refinement</a> and
                <a href="#Ensemble">ensemble</a> section.
                </li>
                <li><b>Enumerate Site Limit</b>. When enabled, the number of
                mutated sites will be enumerated from 1 to N (where N = <b>Site
                Limit</b>). When disabled, only mutations of size N will be
                created.</li>
                <li><b>Lower Limit</b>. Specify site enumeration with an optional
                lower limit other than the default of 1 to restrict
                the mutation space.</li>
                <li><b>Pair Constraint</b>. If enabled, multi-site mutations
                residue pairs must satisfy distance constraints to be included
                in the Residue Scan. This will reduce the number of mutations
                created if the <b>Site Limit</b> is larger than 1.
                </li>
                <li><b>Pair Cutoff</b>. Specifies the maximum distance allowed
                between &alpha; carbon atoms of residues pairs in a multi-site
                mutation.
                </li>
                <li><b>Inter-Chain Only</b>. If enabled, multi-site mutation
                residue pairs are only included, if the residues are in
                different chains. This option is useful to restrict mutations
                to inter-chain interactions in protein interfaces.
                </li>
                <li><b>Strict Pair Cutoff</b>. If on, all residues of a
                multi-site mutation must be within the cutoff distance.
                If off, each residue must have another residue
                within the cutoff distance, but the pairs can
                be farther away.
                </li>
              </ul>
      </td>
    </tr>
  </table><!--Sample Sequence -->
<a id="SampleSequence"></a>
<h4>Sample Sequence</h4>

<p>For more information on input options for Sample Sequence, please see
  <a href="#panel_input">Input/Output Options</a>. For details concerning the
  Sample Sequence method, please see its <a href=
  "#SampleSequenceMethod">Methodology</a>.</p>

<h5>Mutation List and Display Options</h5>
  <table class="noborder">
    <tr>
      <td><b>Mutation List</b></td>
      <td>
        All residues as defined by the <a href="#display_opt">Display
        Options</a> appear in this list. For each residue, the following
        details are given:
        <ul>
          <li><b>Chain</b>: Chain number and name.</li>
          <li><b>Res</b>: Residue Type.</li>
          <li><b>UID</b>: Residue UID and Insertion code.</li>
          <li><b>Mutation</b>: List of possible mutations, given as single
          letter residue codes for the 20 standard amino acids. For non-natural
          amino acids, the full residue name within parentheses is required.
          For example, <tt>(SEP)</tt> specifies phosphorylated serine. If no
          mutations are listed for a residue, no mutation will be carried
          out.</li>
        </ul>
        <p>When the <a href="#display_opt">display option</a> <b>Contour in
        MOE Window</b> is enabled, a single click in the <b>Mutation List</b>
        will display the contours for the selected residues in the list. A
        double click will toggle the selection state of the associated atoms of
        the selected residues.</p>
      </td>
    </tr>
  </table>

<h5>Mutation Editor</h5>

<img src="prodesign/panel_edit2.png" alt=
  "Protein Design Edit Options" class="center" />

<p>The <a href="#MultipleMutations">second variation</a> of the <b>Mutation
  List Editor</b> is used for Sample Sequence.</p>

<h5>Generation Options</h5>
  <table class="noborder">
    <tr>
      <td><b>Site Limit</b></td>
      <td>A site refers to the
      residues being mutated. <b>Site Limit</b> gives the maximum number of
      residues which may constitute a mutation site. For Sample Sequence, the
      mutation space can be randomly searched, creating mutants with 1 to N
      mutations, where N is defined by <b>Site Limit</b>.</td>
    </tr>
    <tr>
      <td valign="top"><b>Mutation Space</b></td>
      <td valign="top">
        This is the number of mutations that will be searched. The value will
        change color depending on the size of the space:
        <ul>
          <li>Yellow = 0</li>
          <li>Green &gt; 0 and &lt; 10,000,000</li>
          <li>Red &ge; 10,000,000</li>
          <li>Cyan: Upper estimate with enabled <b>Pair Constraint</b>.
        </ul>
      </td>
    </tr>
    <tr>
      <td><b>Sample</b></td>
      <td>The mutation space will be searched randomly.
      <b>Sequences</b> determines the number of random mutants that
      will be generated.
<img src="prodesign/sample_sequence_option.png"
    alt="Sample Sequence Option" class="center" />
      </td>
    </tr>
    <tr>
      <td><b>Settings</b></td>
      <td>
              Pressing the Settings icon <img src="prodesign/settings.png" alt=
              "settings" style="vertical-align: middle" /> will raise the popup
              menu shown below.
<img src="prodesign/config_residue_scan.png" alt="options" class="center" />
              <ul>
                <li><b>Repack environment</b>. When enabled, residues within
                <b>Cutoff</b> of the mutation will be included in the
                conformational search.</li>
                <li><b>Cutoff</b>. The distance (in &Aring;) used to determine
                which residues are added to the conformational search in
                addition to the residue being mutated.</li>
                <li><b>Alternate protomers</b>. When enabled, alternate
                protonation states of residues to be mutated are considered.
                For example, when enabled and mutating to Asp both the charged
                and neutral forms are allowed to compete against each other.
                Disabling would allow only the charged form. By default, this
                option is disabled.</li>
                <li><b>pH</b>. Specifies the <i>pH</i> of the system when
                performing the mutation. This, along with <a href=
                "../apps/residuepka.htm">PROPKA</a>, is used for scoring
                alternate protonation states to account for local
                <i>pK<sub>a</sub></i> shifts.</li>
                <li><b>Refine Mutations</b>. When enabled, the mutations will
                be refined, using UQO, following the conformational search. For
                more information on the refinement protocol, see <a href=
                "#Searching">Searching Conformation and Sequence Space</a> and 
                <a href="#Refinement">Refining Mutations Generated by UQO</a>.
                This option is by default enabled.</li>
                <li><b>RMS Gradient</b>. Specifies the cutoff RMS of the
                gradient of the potential energy. If <b>Refine Mutations</b> is
                enabled, the final refinement will be performed until the RMS
                of the gradient of the potential energy is less than this
                specified value.</li>
                <li><b>Preserve Restraints</b>.
                Try to preserve existing fixed states, tethers, and restraints
                during refinement and ensemble calculations:
                Hold all-atom-fixed evironment residues fixed,
                apply tethers of all-tethered environment sidechains,
                and apply existing force field restraints.
                The all-fixed, all-tethered, and restrained residues
                are excluded from the repacking of the environment.
                See further details in the
                <a href="#Refinement">refinement</a> and
                <a href="#Ensemble">ensemble</a> section.
                </li>
                <li><b>Enumerate Site Limit</b>. When enabled, the number of
                mutated sites will be enumerated from 1 to N (where N = <b>Site
                Limit</b>). When disabled, only mutations of size N will be
                created.</li>
                <li><b>Lower Limit</b>. Specify site enumeration with an optional
                lower limit other than the default of 1 to restrict
                the mutation space.</li>
                <li><b>Pair Constraint</b>. If enabled, multi-site mutations
                residue pairs must satisfy distance constraints to be included.
                This will reduce the number of mutations
                created if the <b>Site Limit</b> is larger than 1.
                </li>
                <li><b>Pair Cutoff</b>. Specifies the maximum distance allowed
                between &alpha; carbon atoms of residues pairs in a multi-site
                mutation.
                </li>
                <li><b>Inter-Chain Only</b>. If enabled, multi-site mutation
                residue pairs are only included, if the residues are in
                different chains. This option is useful to restrict mutations
                to inter-chain interactions in protein interfaces.
                </li>
                <li><b>Strict Pair Cutoff</b>. If on, all residues of a
                multi-site mutation must be within the cutoff distance.
                If off, each residue must have another residue
                within the cutoff distance, but the pairs can
                be farther away.
                </li>
              </ul>
      </td>
    </tr>
  </table><!-- Sequence Design -->
<a id="SequenceDesign"></a>
<h4>Sequence Design</h4>

<p>For more information on input options for Sequence Design, please see
  <a href="#panel_input">Input/Output Options</a>. For details concerning the
  Sequence Design method, please see its <a href=
  "#SequenceDesignMethod">Methodology</a>.</p>

<h5>Mutation List and Display Options</h5>
  <table class="noborder">
    <tr>
      <td><b>Mutation List</b></td>
      <td>
        All residues as defined by the <a href="#display_opt">Display
        Options</a> appear in this list. For each residue, the following
        details are given:
        <ul>
          <li><b>Chain</b>: Chain number and name.</li>
          <li><b>Res</b>: Residue Type.</li>
          <li><b>UID</b>: Residue UID and Insertion code.</li>
          <li><b>Mutation</b>: List of possible mutations, given as single
          letter residue codes for the 20 standard amino acids. For non-natural
          amino acids, the full residue name within parentheses is required.
          For example, <tt>(SEP)</tt> specifies phosphorylated serine. If no
          mutations are listed for a residue, no mutation will be carried
          out.</li>
        </ul>
        <p>When the <a href="#display_opt">display option</a> <b>Contour in
        MOE Window</b> is enabled, a single click in the <b>Mutation List</b>
        will display the contours for the selected residues in the list. A
        double click will toggle the selection state of the associated atoms of
        the selected residues.</p>
      </td>
    </tr>
  </table>

<h5>Mutation Editor</h5>

<img src="prodesign/panel_edit2.png" alt=
  "Protein Design Edit Options" class="center" />

<p>The <a href="#MultipleMutations">second variation</a> of the <b>Mutation
  List Editor</b> is used for Sequence Design.</p>

<h5>Generation Options</h5>
  <table class="noborder">
    <tr>
      <td><b>Site Limit</b></td>
      <td>A site refers to the
      residues being mutated. <b>Site Limit</b> gives the maximum number of
      residues which may constitute a mutation site. For Sequence Design, all
      sites of sizes 1 to <b>Site Limit</b> will be generated.</td>
    </tr>
    <tr>
      <td><b>Mutation
      Space</b></td>
      <td>
        This is the number of mutations that will be searched. The value will
        change color depending on the size of the space:
        <ul>
          <li>Yellow = 0</li>
          <li>Green &gt; 0 and &lt; 10,000,000</li>
          <li>Red &ge; 10,000,000</li>
          <li>Cyan: Upper estimate with enabled <b>Pair Constraint</b>.
        </ul>
      </td>
    </tr>
    <tr>
      <td><b>Design</b></td>
      <td>
      <ul>
        <li><b>Maximum Number</b>. The maximum number of unique
        sequences that will be output per mutation site.</li>
        <li><b>Energy Window</b>. Specifies the cutoff energy. Unique
        sequences whose energy for a mutation site is greater than this
        value (above the global minimum) will be discarded.</li>
      </ul>
<img src="prodesign/sequence_design_option.png" alt=
"Sequence Design Options" class="center" />
      </td>
    </tr>
    <tr>
      <td><b>Settings</b></td>
      <td>
<a id="settings_seqdesign"></a>
              Pressing the Settings icon <img src="prodesign/settings.png" alt=
              "settings" style="vertical-align: middle" /> will raise the popup
              menu shown below.
<img src="prodesign/config_sequence_design.png" alt="options" class="center" />
              <ul>
                <li><b>Repack environment</b>. When enabled, residues within
                <b>Cutoff</b> of the mutation will be included in the
                conformational search.</li>
                <li><b>Cutoff</b>. The distance (in &Aring;) used to determine
                which residues are added to the conformational search in
                addition to the residue being mutated.</li>
                <li><b>Alternate protomers</b>. When enabled, alternate
                protonation states of residues to be mutated are considered.
                For example, when enabled and mutating to Asp both the charged
                and neutral forms are allowed to compete against each other.
                Disabling would allow only the charged form. By default, this
                option is disabled.</li>
                <li><b>pH</b>. Specifies the <i>pH</i> of the system when
                performing the mutation. This, along with <a href=
                "../apps/residuepka.htm">PROPKA</a>, is used for scoring
                alternate protonation states to account for local
                <i>pK<sub>a</sub></i> shifts.</li>
                <li><b>Refine Mutations</b>. When enabled, the mutations will
                be refined, using UQO, following the conformational search. For
                more information on the refinement protocol, see <a href=
                "#Searching">Searching Conformation and Sequence Space</a>. and 
                <a href="#Refinement">Refining Mutations Generated by UQO</a>.
                This option is by default enabled.</li>
                <li><b>RMS Gradient</b>. Specifies the cutoff RMS of the
                gradient of the potential energy. If <b>Refine Mutations</b> is
                enabled, the final refinement will be performed until the RMS
                of the gradient of the potential energy is less than this
                specified value.</li>
                <li><b>Preserve Restraints</b>.
                Try to preserve existing fixed states, tethers, and restraints
                during refinement and ensemble calculations:
                Hold all-atom-fixed evironment residues fixed,
                apply tethers of all-tethered environment sidechains,
                and apply existing force field restraints.
                The all-fixed, all-tethered, and restrained residues
                are excluded from the repacking of the environment.
                See further details in the
                <a href="#Refinement">refinement</a> and
                <a href="#Ensemble">ensemble</a> section.
                </li>
                <li><b>Enumerate Site Limit</b>. When enabled, the number of
                mutated sites will be enumerated from 1 to N (where N = <b>Site
                Limit</b>). When disabled, only mutations of size N will be
                created.</li>
                <li><b>Pair Constraint</b>. If enabled, multi-site mutations
                residue pairs must satisfy distance constraints to be included.
                This will reduce the number of mutations
                created if the <b>Site Limit</b> is larger than 1.
                </li>
                <li><b>Pair Cutoff</b>. Specifies the maximum distance allowed
                between &alpha; carbon atoms of residues pairs in a multi-site
                mutation.
                </li>
                <li><b>Inter-Chain Only</b>. If enabled, multi-site mutation
                residue pairs are only included, if the residues are in
                different chains. This option is useful to restrict mutations
                to inter-chain interactions in protein interfaces.
                </li>
                <li><b>Strict Pair Cutoff</b>. If on, all residues of a
                multi-site mutation must be within the cutoff distance.
                If off, each residue must have another residue
                within the cutoff distance, but the pairs can
                be farther away.
                </li>
              </ul>
      </td>
    </tr>
  </table><!-- End of Protein Variants -->
<a id="panel_input"></a>
<h3>Input/Output Options</h3>

<p>The Protein Design application uses the system currently in the MOE Window
  for running the simulation. Before running, ensure the system has been
  <a href="#Preparing_the_Data">prepared properly</a>.</p>

<img src="prodesign/panel_input.png" alt=
  "Protein Design Input Options" class="center" />
  <table class="noborder">
    <tr>
      <td><b>Chains</b></td>
      <td>The collection of chains
      which the Protein Design application will use. All chains containing only
      dummy atoms will be ignored. The default is Active Chains.</td>
    </tr>
    <tr>
      <td><b>MDB File</b></td>
      <td>
        It is also possible to use an MDB file containing a conformational
        ensemble generated from such applications as <a href=
        "../apps/confsrch.htm#LowModeMD">LowMode MD</a> or <a href=
        "../apps/md.htm">Molecular Dynamics</a>. When an MDB file is defined,
        The Protein Design application will loop over the MDB performing the
        desired mutations defined in the mutation list.

<p>To specify a MOE database (MDB) file, enter a filename in the
        <span class="menu">Protein Design | MDB File</span> text field.
        Alternatively, use the <b>Browse</b> button to select a file through a
        file browser. For Protein Design to run correctly, the system currently
        loaded in MOE must correspond to the molecule defined in the first
        molecule field in the database. Note that the first molecule field of
        an MDB file will be used irrespective of the name, and ignoring all
        other molecule fields.</p>
      </td>
    </tr>
    <tr>
      <td><b>Output</b></td>
      <td>MDB file output
      name.</td>
    </tr>
  </table>
<a id="display_opt"></a>
<h3>Display Options</h3>

<p>The Protein Design application <b>Display Options</b> work with the
  <b>Mutation List</b> that is displayed for the different <a href=
  "#Variants">Protein Design Variants</a>.</p>

<img src="prodesign/panel_display_options.png" alt=
  "Protein Design Display Options" class="center" />
  <table class="noborder">
    <tr>
      <td><b>Display</b></td>
      <td>
        This option filters which residues are displayed in the <b>Mutation
        List</b>. Residues not shown in the list will not be mutated.
        <b>Display</b> has the following options:
        <ul>
          <li>All Residues</li>
          <li>Selected Atoms</li>
          <li>Selected Residues</li>
          <li>Selected Chains</li>
          <li>Pocket</li>
          <li>Interface</li>
          <li><i>Sets</i></li>
        </ul>The initial value for <b>Display</b> is dependent on what is
        currently loaded. For example, if atoms are selected in the system,
        <b>Display</b> will default to <i>Selected Atoms</i>.
      </td>
    </tr>
    <tr>
      <td style="vertical-align:top"><b>Contour in MOE Window</b></td>
      <td style="vertical-align:top">When enabled, a transparent surface is
      created for the residues selected in the <b>Mutation List</b>. A green
      contour indicates that the associated residues will be mutated, while a
      magenta color indicates otherwise. By default, <b>Contour in MOE
      Window</b> is disabled.</td>
    </tr>
  </table>
<a id="ensemble_opt"></a>
<h3>Ensemble Options</h3>

<p>The Ensemble options listed below are the same for all Protein Design
  variants.</p>
  <div class="floatleft"><img src="prodesign/ensemble_opt.png" class="floatleft" /></div>

<p>If the site being mutated is flexible, then generating an ensemble of
  conformations may yield more accurate results. To enable generation of a
  conformational ensemble, enable the <b>Ensemble</b> checkbox in the Protein
  Design panel. Once enabled, select one of the <a href="#Ensemble">ensemble
  methods</a>. Press the Settings icon to open a pop-up window for ensemble
  options.</p>
  <table class="noborder">
    <tr>
      <td width="250">
        <b>Rotamer Explorer</b>

<img src="prodesign/config_rotamerexplorer.png" class="center" />
      </td>
      <td>
        <ul>
          <li><b>RMSD Limit</b>. Check for duplicates in the generated
          ensemble. Two conformations are considered duplicates if the RMSD
          between them, taking molecular symmetry into account, is less than
          the RMSD Limit parameter.</li>
          <li><b>Energy Window</b>. Energy minimized conformations will be
          discarded if their potential energy is greater than <span class=
          "nobr"><i>E<sub>min</sub></i> + <i>s</i></span>, where
          <i>E<sub>min</sub></i> is the lowest energy among the minimized
          conformations and <i>s</i> is the value specified for Energy Window,
          in kcal/mol.</li>
          <li><b>Conformation Limit</b>. Specifies the limit for the resulting
          number of conformations. If a value <i>k</i> is specified, then only
          the <i>k</i> lowest energy conformations will be retained and the
          rest will be discarded.</li>
          <li><b>Fix Residues Farther than</b>. Residues farther than this
          distance (&Aring;) will be fixed. Residues within this distance will
          be searched using the Rotamer Explorer.</li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>
        <b>LowMode</b>

<img src="prodesign/config_lowmode.png" class="center" />
      </td>
      <td>
        <ul>
          <li><b>Iteration Limit</b>. Maximum number of LowMode
          iterations.</li>
          <li><b>RMSD Limit</b>. Check for duplicates in the generated
          ensemble. Two conformations are considered duplicates if the RMSD
          between them, taking molecular symmetry into account, is less than
          the RMSD Limit parameter.</li>
          <li><b>Energy Window</b>. Energy minimized conformations will be
          discarded if their potential energy is greater than <span class=
          "nobr"><i>E<sub>min</sub></i> + <i>s</i></span>, where
          <i>E<sub>min</sub></i> is the lowest energy among the minimized
          conformations and <i>s</i> is the value specified for Energy Window,
          in kcal/mol.</li>
          <li><b>Conformation Limit</b>. Specifies the limit on the resulting
          number of conformations. If a value <i>k</i> is specified then only
          the <i>k</i> lowest energy conformations will be retained and the
          rest discarded.</li>
          <li><b>Fix Residues Farther than</b>. Residues farther than this
          distance (&Aring;) will be fixed. Residues within this distance will
          be searched using LowMode MD.</li>
          <li><b>Tether Sidechains</b>. A tether weight of this value will be
          applied to side chain atoms of residues within the above cutoff. A
          flat bottom tether is used with a maximum deviation of 0.25.</li>
          <li><b>Tether Backbone</b>. A tether weight of this value will be
          applied to backbone atoms of residues within the above cutoff. A flat
          bottom tether is used with a maximum deviation of 0.25.</li>
          <li><b>Add Explicit Solvent</b>. Add explicit water molecules when
          performing the ensemble generation. The system will be neutralized by
          adding the minimum number of Na+ and Cl- ions. The solvent will not
          be saved in the output MDB file.</li>
        </ul>
      </td>
    </tr>
    <tr>
      <td>
        <b>Dynamics</b>

<img src="prodesign/config_dynamics.png" class="center" />
      </td>
      <td>
        <ul>
          <li><b>Simulation Length (ps)</b>: The length of time in picoseconds
          of the Molecular Dynamic production stage.</li>
          <li><b>Energy Window</b>. Energy minimized conformations will be
          discarded if their potential energy is greater than <span class=
          "nobr"><i>E<sub>min</sub></i> + <i>s</i></span>, where
          <i>E<sub>min</sub></i> is the lowest energy among the minimized
          conformations and <i>s</i> is the value specified for Energy Window,
          in kcal/mol.</li>
          <li><b>Conformation Limit</b>. Specifies the limit for the resulting
          number of conformations. If a value <i>k</i> is specified, then only
          the <i>k</i> lowest energy conformations will be retained and the
          rest discarded.</li>
          <li><b>Fix Residues Farther than</b>. Residues farther than this
          distance (&Aring;) will be fixed. Residues within this distance will
          be allowed to move during the Molecular Dynamics simulation.</li>
          <li><b>Tether Sidechains</b>. A tether weight of this value will be
          applied to side chain atoms of residues within the above cutoff. A
          flat bottom tether is used with a maximum deviation of 0.25.</li>
          <li><b>Tether Backbone</b>. A tether weight of this value will be
          applied to backbone atoms of residues within the above cutoff. A flat
          bottom tether is used with a maximum deviation of 0.25.</li>
          <li><b>Add Explicit Solvent</b>. Add explicit water molecules when
          performing the ensemble generation. The system will be neutralized by
          adding the minimum number of Na+ and Cl- ions. The solvent will not
          be saved in the output MDB file.</li>
          <li><b>Save Trajectories</b>. All trajectory files will be saved in a
          folder of the same name root name as the output MDB file. Each
          mutation entry will have its own trajectory file
          (<i>rootname</i>/<i>rootname</i>_<i>mutation</i>.mdb).</li>
        </ul>
      </td>
    </tr>
  </table>
<a id="affinity_opt"></a>
<h3>Affinity Options</h3>

<p>The Affinity options listed below are the same for all Protein Design
  variants.</p>

<p>When affinity is enabled, the <a href=
  "../apps/docking.htm#GBVI_WSA_dG_Scoring">GBVI/WSA dG scoring function</a>
  or MM/GBVI is
  used to calculate the affinity of the molecule defined by <i>Atoms</i> with
  the rest of the system. MM/GBVI is used when the molecule defined by
  <i>Atoms</i> contains more than 10 residues. This is because GBVI/WSA dG is
  appropriate only for Protein:Ligand complexes, and not Protein:Protein ones.
  In both cases the units for affinity are kcal/mol.</p>
  <table class="noborder left">
    <tr>
      <td><b>Atoms</b></td>
      <td>Used to defined the atoms
      of binding partner (e.g. a ligand in a Protein:Ligand complex or an
      antigen in an Antibody:Antigen complex).</td>
    </tr>
    <tr>
      <td><b>?</b></td>
      <td>Flash the selection state
      of the atoms defined by <i>Atoms</i>.</td>
    </tr>
  </table>
<a id="properties_opt"></a>
<h3>Properties Options</h3>

<p>The Properties options listed below are the same for all Protein Design
  variants.</p>

<p>When Properties is enabled, then <a href="../proteins/ensprop.htm#MolProp">
  protein properties</a> will be calculated for the set defined by
  <i>Chains</i>.</p>
  <table class="noborder left">
    <tr>
      <td><img src="prodesign/settings.png" alt=
              "settings" class="icon" /></td>
      <td>Select to calculate protein property values,
      protein patch surface values, or both for specified
      protein chain(s).</td>
    </tr>
    <tr>
      <td><b>Chains</b></td>
      <td>Used to define the
      protein chain(s) for which the protein properties will be
      calculated.</td>
    </tr>
    <tr>
      <td><b>?</b></td>
      <td>Briefly flash the atoms
      defined by <i>Chains</i>.</td>
    </tr>
  </table>
<a id="Output"></a>
<h2>Working with Output</h2>

<p>The Protein Design application will write the mutations to a database
  containing some or all of the fields described below. Please note that for
  all mutations the Protein Design application automatically performs a
  mutation to self for the wild type protein to calculate relative changes to
  the stability and other predicted properties.
<a id="Output_Database"></a>
</p>

<h3>The Output Database</h3>

<p class="center"></p>
  <table class="titlebar">
    <tr>
      <th>field</th>
      <th>Description</th>
    </tr>
    <tr>
      <td><b><tt>mol</tt></b></td>
      <td>The output mutation. When
      an ensemble is generated, the lowest energy conformation of the ensemble
      is saved in this field.</td>
    </tr>
    <tr>
      <td>
      <b><tt>mol:conf</tt></b></td>
      <td>Packed conformational
      ensemble of the mutant. This field is only present if an ensemble was
      generated.</td>
    </tr>
    <tr>
      <td><b><tt>mseq</tt></b></td>
      <td>The molecule sequence
      number. If an MDB is being used as input, the mseq will correspond to the
      entry number in the input database. If the input was from the system
      loaded MOE, then the mseq will be 1. When examining the output database,
      entries with different mseq numbers arise from different input
      structures.</td>
    </tr>
    <tr>
      <td><b><tt>sseq</tt></b></td>
      <td>The site sequence number
      corresponds to the unique mutation expression used to generate the
      entry's mutation.</td>
    </tr>
    <tr>
      <td>
      <b><tt>mutseq</tt></b></td>
      <td>The mutation sequence
      number. This is used to identify the id of the mutation within an
      individual mutation expression. A mutseq of 0 corresponds to the wild
      type.</td>
    </tr>
    <tr>
      <td>
      <b><tt>mutation</tt></b></td>
      <td>
        The mutation from the input required to produce the structure in mol.
        The mutation is defined as follows for the example of 1:Y61N.
        <ul>
          <li><i>1</i> = Chain number</li>
          <li><i>Y</i> = Wild Type Single letter residue name</li>
          <li><i>61</i> = Residue UID and insertion code</li>
          <li><i>Asn</i> = Single letter mutation name</li>
        </ul>Multiple point mutations will have mutations defined by a comma
        separated list.
      </td>
    </tr>
    <tr>
      <td>
      <b><tt>Stability</tt></b></td>
      <td>The absolute thermostability of the mutation.
      When generating an ensemble, Stability is equal to the Boltzmann
      average of the stabilities of the ensemble. Note that the Stability value
      is specific to the specified mutation site and only used to calculate
      the difference values between the wild-type and mutations at the
      same site. It is not a value to quantify the overall Stability of
      the protein structure. It can not be compared to the Stability 
      value of other, non-related proteins.
      The units of stability are kcal/mol.</td>
    </tr>
    <tr>
      <td>
      <b><tt>dStability</tt></b></td>
      <td>The relative
      thermostability of the mutation with respect to the wild type protein.
      When generating an ensemble, dStability is the difference of
      the Boltzmann averages of the mutant and wild type stabilities
      of the ensemble.
      A more negative value indicates a more stable mutation.
      The units of stability are kcal/mol.<br />
      A color gradient is applied to this field marking generally favorable
      mutations blue and disfavorable ones red.</td>
    </tr>
    <tr>
      <td>
      <b><tt>Stability_all</tt></b></td>
      <td>A vector of numbers,
      where element <i>i</i> corresponds to the Stability of conformation
      <i>i</i> in the ensemble. This field is only present if an ensemble was
      generated.</td>
    </tr>
    <tr>
      <td>
      <b><tt>Affinity</tt></b></td>
      <td>The absolute binding affinity of the mutation.
      This field is only present if the affinity calculation is enabled.
      When generating an ensemble, Affinity is equal to the Boltzmann
      average of the affinities of the ensemble.
      A more negative value indicates a mutation with better affinity.
      The units of affinity
      are kcal/mol.</td>
    </tr>
    <tr>
      <td>
      <b><tt>dAffinity</tt></b></td>
      <td>The relative
      binding affinity of the mutation to the wild type protein.
      This field is only present if the affinity calculation was enabled,
      which is the case by default for Resistance Scanning.
      When generating an ensemble, dAffinity is the difference of
      the Boltzmann averages of the mutant and wild type affinities
      of the ensemble.
      A more negative value indicates a mutation with better affinity.
      The units of affinity are kcal/mol.<br />
      A color gradient is applied to this field marking generally favorable
      mutations blue and disfavorable ones red.</td>
    </tr>
    <tr>
      <td>
      <b><tt>Affinity_all</tt></b></td>
      <td>A vector of numbers,
      where element <i>i</i> corresponds to the Affinity of conformation
      <i>i</i> in the ensemble. This field is only present if an ensemble was
      generated.</td>
    </tr>
    <tr>
      <td><b><tt>S-S</tt></b></td>
      <td>The absolute number of
      disulfide bonds.</td>
    </tr>
    <tr>
      <td><b><tt>dS-S</tt></b></td>
      <td>The relative number of
      disulfide bonds to the wild type. By default, this field is always
      written when performing a Disulfide Scan. This field makes it possible
      to quickly identify entries where a new disulfide bond has been
      created.<br />
      This field is colored. If the number of disulfide bonds
      increases it is marked blue and red if the number decreases.</td>
    </tr>
    <tr>
      <td><b><tt>S-S
      ChiX</tt></b></td>
      <td>The torsion angles
      <i>Chi1</i>, <i>Chi2</i>, <i>Chi3</i>, <i>Chi2*</i>, and <i>Chi3*</i> of
      disulfide bridges (in degrees). They allow comparison of the geometry and
      stability of the disulfide bridge. These fields are only written if a
      disulfide scan was performed and only populated if the specific mutation
      resulted in the formation of a disulfide bridge.</td>
    </tr>
    <tr>
      <td>
      <b><tt>mutexp</tt></b></td>
      <td>The mutation expression
      that the mutation is derived from.</td>
    </tr>
    <tr>
      <td>
      <b><tt>dpro_<i>XXX</i></tt></b></td>
      <td>
        The relative difference between the mutation and wild type for various
        <a href="../proteins/ensprop.htm#MolProp">protein properties</a>,
        where XXX is the following:
        <ul>
          <li><i>mass</i>: Protein Mass in kDa.</li>
          <li><i>debye</i>: Debye Screening Length.</li>
          <li><i>pI_seq</i>: Sequence-based pI Prediction.</li>
          <li><i>pI_3D</i>: Structure-based pI Prediction.</li>
          <li><i>coeff_280</i>: Extinction coefficient at 280 nm.</li>
          <li><i>coeff_fric</i>: Frictional Coefficient.</li>
          <li><i>coeff_diff</i>: Diffusion Coefficient.</li>
          <li><i>r_gyr</i>: Radius of Gyration.</li>
          <li><i>r_solv</i>: Hydrodynamic Radius.</li>
          <li><i>sed_const</i>: Sedimentation Constant.</li>
          <li><i>eccen</i>: Protein Eccentricity.</li>
          <li><i>asa_vdw</i>: Accessible Surface Area (Water Probe).</li>
          <li><i>asa_hyd</i>: Hydrophobic Surface Area.</li>
          <li><i>asa_hph</i>: Hydrophilic Surface Area.</li>
          <li><i>volume</i>: Protein Volume.</li>
          <li><i>mobility</i>: Protein Mobility.</li>
          <li><i>helicity</i>: Protein Helix Ratio.</li>
          <li><i>henry</i>: Henry's Function f(ka).</li>
          <li><i>net_charge</i>: Protein Net Charge.</li>
          <li><i>app_charge</i>: Protein Charge at Debye Length.</li>
          <li><i>dipole_moment</i>: Protein Dipole Moment.</li>
          <li><i>hyd_moment</i>: Hydrophobicity Moment.</li>
          <li><i>zeta</i>: Zeta Potential at Debye Length.</li>
          <li><i>zdipole</i>: Zeta Dipole Moment.</li>
          <li><i>zquadrupole</i>: Zeta Quadruple Moment.</li>
        </ul>
        <p>
        The relative difference between the mutation and wild type for
        <a href="../proteins/ensprop.htm#ProPatch">protein patch surface areas</a>
        in &Aring;<sup>2</sup> where XXX is the following:</p>
        <ul>
          <li><i>patch_hyd</i>:
                 Total area of hydrophobic protein patch(es).</li>
          <li><i>patch_pos</i>:
                 Total area of positive protein patch(es).</li>
          <li><i>patch_neg</i>:
                 Total area of negative protein patch(es).</li>
          <li><i>patch_cdr_hyd</i>: Total area of hydrophobic protein patch(es)
                 near the antibody CDRs.</li>
          <li><i>patch_cdr_pos</i>: Total area of positive protein patch(es)
                 near the antibody CDRs.</li>
          <li><i>patch_cdr_neg</i>: Total area of negative protein patch(es)
                 near the antibody CDRs.</li>
        </ul>
      </td>
    </tr>
    <tr>
      <td><b><tt>proprop</tt></b></td>
      <td>Vector of all absolute protein property values.</td>
    </tr>
     <tr>
      <td><b><tt>proprop_all</tt></b></td>
      <td>Vector of all absolute protein property values for
      all ensemble conformations. This field is only present
      if an ensemble was generated.</td>
    </tr>
 </table>
<a id="Browsing_the_Output_Database"></a>
<h3>Browsing the Output Database</h3>

<p>To use the custom browser for the Protein Design application, choose
  <span class="menu">DBV | File | Browse</span>. The following browser will
  then appear.</p>

<img src="prodesign/browser.png" class="center" />

  <table class="noborder">
    <tr>
      <td><b>dStability</b></td>
      <td>The relative stability
      compared to the wild type protein.</td>
    </tr>
    <tr>
      <td><b>dAffinity</b></td>
      <td>The relative binding affinity
      compared to the wild type protein (only shown if calculated).</td>
    </tr>
    <tr>
      <td><b>Keep</b></td>
      <td>Normally, browsing
      through the database replaces the currently browsed entry by another
      entry. Pressing <b>Keep</b> causes the current entry to be retained
      in MOE.</td>
    </tr>
    <tr>
      <td><b>Isolate</b></td>
      <td>
        Similar to <span class="menu">MOE | RHS | SiteView</span>.
        <b>Isolate</b> will perform the following rendering changes:
        <ol>
          <li>Isolate the mutation by hiding distant atoms. Distant backbone
          ribbons will also be hidden.</li>
          <li>Render the mutation in stick mode; all other atoms will be
          rendered in line mode.</li>
          <li>Hide non-polar hydrogens and show polar hydrogens.</li>
        </ol>
      </td>
    </tr>
    <tr>
      <td><b>Extend</b></td>
      <td>Extend the visible isolated site,
      if <b>Isolate</b> is toggled on.</td>
    </tr>
    <tr>
      <td><b>Label</b></td>
      <td>Label the mutated residue.</td>
    </tr>
    <tr>
      <td><b>Center</b></td>
      <td>Center view on the mutation site.</td>
    </tr>
    <tr>
      <td><b>Overlay Wild
      Type</b></td>
      <td>Overlay wild type protein
      with current entry.</td>
    </tr>
    <tr>
      <td><b>Color</b></td>
      <td>When enabled, the carbon
      color of the mutant and wild-type structures are controlled using the
      <b>Mutant</b> and <b>Wild Type</b> color widgets. When disabled, the atom
      coloring will be controlled using, for example, the <span class=
      "menu">MOE | Footer | Atom</span> or the <a href=
      "../moe/gui/system.htm"><b>System Manager</b></a>.</td>
    </tr>
    <tr>
      <td><b>Mutant/Wild Type
      Color</b></td>
      <td>Allows you to control the
      color of the wild type and mutant carbon atoms.</td>
    </tr>
    <tr>
      <td><b>Conformation</b></td>
      <td>If an ensemble calculation was performed, the <b>Conformation</b>
      slider allows to browse through the ensemble conformations
      of the mutated residue (only available if ensemble data is present).</td>
    </tr>
    
    <tr>
      <td><b>Select</b></td>
      <td>Toggle selection state of
      current entry being viewed.</td>
    </tr>
    <tr>
      <td><b>Browse Only
      Selected</b></td>
      <td>Browse only selected
      entries.</td>
    </tr>
  </table>

<a id="Using_MOEsmp"></a>
<h2>Running Protein Design with MOE/cluster (MOE/smp)</h2>

<p>Installing and running MOE in SMP mode is described in the <a href=
  "../install/smp_inst.htm">MOE/smp</a> manual. If MOE is being run on a
  multiprocessor workstation, then this can be setup by invoking MOE with the
  appropriate command line <a href=
  "../install/install.htm#RunSwitch">switches</a>, e.g.:</p>

<pre class="code oneline">
cmd% moe -mpu 4
</pre>

<p>The above example will run MOE with four SMP nodes, all on the same
  workstation. When MOE/cluster (MOE/smp) is being used, multiple mutation
  sites will be searched simultaneously, which speeds up the calculation
  tremendeously.</p>

<p>If one uses the <b>Batch</b> option of the Protein Design application
to submit the job to a queuing system connected to the MOE 
<a href="../moe/hpc.htm">HPC framework</a> one can specify the number
of SMP workers directly in the Batch submit panel. In the following
example one requests to use 32 MOE workers in parallel:</p>

<img src="prodesign/batch_mpu.png"
 alt="Submit Protein Design Job to 32 workers" class="center" />
 
<a id="Using_GPU"></a>
<h2>Running Protein Design with GPU</h2>

<p>The conformational search and sidechain repacking algorithm can take
advantage of the GPU cores of modern graphics cards, such as
CUDA&reg; capable graphics cards from Nvidia.
If MOE detects a suitable graphics card
(see <a href="../install/install.htm#GPUInfo">GPU Info</a>)
it will automatically use it which greatly speeds up the calculation time.
Note that the optional ensemble calculations do not take advantage of the GPU.

<a id="Continuing"></a>
<h2>Continuing an Incomplete Protein Design Job</h2>

<p>Sometimes it is necessary to cancel a computational experiment for some
  unforeseen reason such as server maintenance, priority given to another job,
  or an unexpected computer shut down. When this occurs and a
  Protein Design job was stopped, it is possible to continue the Protein Design
  experiment. To do this, start the SVL Commands window.</p>

<p class="blockquote"><span class="menu">MOE | SVL</span></p>

<p>In the SVL window type the following function:</p>

<pre class="code oneline">
svl&gt; ProteinDesign '<i>prodesign_outputfile</i>'
</pre>

<p>where <i>prodesign_outputfile</i> is the previously running job. This will
  automatically continue the Protein Design job specified by
  <i>prodesign_outputfile</i>. Additionally, it will load the structure into
  the MOE window.</p>

<p>To start the job from the beginning and delete all of the current data in
  the MDB file, the following option can be used:</p>

<pre class="code oneline">
svl&gt; ProteinDesign ['<i>prodesign_outputfile</i>', [restart:0]]
</pre>

<p>This will start the Protein Design job specified in
  <i>prodesign_outputfile</i> from the beginning, overwriting all data
  currently in the output. When using the <tt>restart:0</tt> option, all
  other Protein Design options can be used to modify the parameters of the
  Protein Design experiment.</p>

<p>When using the Sample Sequence variant of Protein Design, it is possible
  to add more mutants to the database even if the run was completed. This
  provides the ability to augment the search space if it was not searched
  adequately. This can be done using the following option:</p>

<pre class="code">
svl&gt; ProteinDesign ['<i>prodesign_outputfile</i>', [
    seq_sample_iter : <i>number_of_total_mutants</i>
]]
</pre>

<p>This option will force Protein Design to add additional mutants so that
  the total (previous + new mutants) is equal to
  <i>number_of_total_mutants</i>. This option is similar to the <b><a href=
  "#SampleSequence">Sequences</a></b> option of the Sample Sequence variant.
<a id="Reloading"></a>
</p>

<h2>Reload/Start from a Protein Design Output File</h2>

<p>To enable the ability to see which parameters where used when conducting
  the Protein Design experiment, it is possible to reload the Protein Design
  application with the options originally used. To do this, start the SVL
  Commands window.</p>

<p class="blockquote"><span class="menu">MOE | SVL</span></p>

<p>In the SVL window type the following command:</p>

<pre class="code oneline">
svl&gt; run ['prodesign_ui.svl', '<i>prodesign_outputfile</i>']
</pre>

<p>This will load the input structure used for the Protein Design experiment
  into the MOE window. It will automatically create atom sets that initialize
  the following:</p>
  <ul>
    <li><i>MOEProDes_Env</i> is used to define the <b>Chains</b> that were
    specified in the <a href="#panel_input">Input/Output Options</a>.</li>
    <li><i>MOEProDes_Mut</i> is used for residues defined by the <b>Display</b>
    that was specified in the Display Options of the <a href=
    "#panel_variants">Protein Design Variant</a>.</li>
    <li><i>MOEProDes_Lig</i> is used to define the Ligand <b>Atoms</b> that
    were used when the <a href="#affinity_opt">Affinity Options</a> were
    enabled.</li>
    <li><i>MOEProDes_Prop</i> is used to define the Property <b>Chains</b> that
    were used when the <a href="#properties_opt">Properties Options</a> were
    enabled.</li>
  </ul><!-- =========================== Examples ========================= -->
<a id="Examples"></a>
<h1>Examples</h1><!-- Example 1 -->
<a id="Example1_Data_Preparation"></a>
<h2>Example 1: Data Preparation for Protein Design</h2>
  <div class="example">

<p>This example demonstrates how to prepare a protein for any of the
    Protein Design Variants. Subtilisins are protein-digesting enzymes which
    are commonly found in laundry detergents to allow for efficient cleaning of
    clothes at cold temperatures.</p>
    <ol>
      <li>
        <i>Load the structure into MOE.</i>

<p class="blockquote"><span class="menu">MOE | File | Open |
        <tt>$MOE/sample/mol/2st1.pdb.gz</tt></span></p>In the Load PDB File
        panel, press <b>OK</b>. The molecular data comprises the crystallized
        protein.
      </li>
      <li>
        <i>Delete the solvent chain.</i>

<p class="blockquote"><span class="menu">MOE | RHS | Select |
        Solvent</span><br />
        <span class="menu">MOE | RHS | Delete</span></p>
        <div class="floatright"><img src="prodesign/example1_2st1_prep.png"
        alt="Preparation of 2ST1" class="floatright" /></div>
      </li>
      <li>
        <i>Setup the forcefield and assign partial charges.</i>

<p class="blockquote"><span class="menu">MOE | Footer | Potential
        Setup</span></p>If the currently loaded forcefield is not Amber10EHT,
        select it from the <b>Load</b> menu. The change is effective
        immediately. To assign charges to both the protein and the ligand,
        click on <span class="menu">Potential Setup | Fix Charges</span> Close
        the Potential Setup panel.
      </li>
      <li>
        <i>Correct Structural Issues.</i><br />

<p class="blockquote"><span class="menu">MOE | Compute | Prepare |
        Structure Preparation</span> or <span class="menu">MOE | Protein |
        Structure Preparation</span></p>Press <b>Correct</b> in the panel to
        add hydrogens and correct structural issues in the protein-ligand
        complex.
      </li>
      <li>
        <i>Optimize hydrogen bond network.</i><br />

<p class="blockquote"><span class="menu">Structure Preparation |
        Protonate3D</span></p>Press <b>OK</b> in the Protonate3D panel to
        optimize the hydrogen positions of the protein. The calculation may
        take a moment. Once done, you may close the Structure Preparation
        panel.
      </li>
      <li>
        <i>Save current system as MOE file.</i>

<p class="blockquote"><span class="menu">MOE | File | Save...|
        <tt>2st1_prep.moe</tt></span></p>Save the current system as
        2st1_prep.moe for future use in other examples.
      </li>
    </ol><br clear="all" />
  </div><!-- Example 2 -->
<a id="Example2_DisulfideScan"></a>
<h2>Example 2: Disulfide Scan on 2ST1</h2>
  <div class="example">

<p>This example demonstrates the use of Disulfide Scan. Subtilisins are
    protein-digesting enzymes which are commonly found in laundry detergents to
    allow for efficient cleaning of clothes at cold temperatures.</p>
    <ol>
      <li>
        <i>Load the prepared structure from Example 1, Data Preparation.</i>

<p class="blockquote"><span class="menu">MOE | File | Open |
        <tt>2st1_prep.moe</tt></span></p>
      </li>
      <li>
        <i>Open the Disulfide Scan Panel</i>.

<p class="blockquote"><span class="menu">MOE | Protein | Design |
        Disulfide Scan ...</span></p>
        <div class="floatright"><img src="prodesign/example2_2st1_site.png"
        alt="2ST1 Mutation Site" class="floatright" /></div>
      </li>
      <li>
        <i>Identify residues to be mutated.</i><br />

<p>By default since there are no atoms or residues selected, no
        ligands, and no multiple protein chains, all residues are displayed in
        the Mutation List. To reduce the list to only the desired residues, the
        <b>Display</b> controls should be used. Scroll down in the Mutation
        List to <tt>SER87</tt> and double click; this will select the atoms in
        the MOE Window. Select <span class="menu">Display | Selected
        Atoms</span> in the panel.</p>
      </li>
      <li>
        <i>Isolate mutation site.</i>

<p class="blockquote"><span class="menu">MOE | RHS |
        SiteView</span></p>SiteView is used to show only selected mutation
        sites and nearby residue atoms. Enable <b>Contour in MOE Window</b> to
        view the residues to be mutated. In green is SER87, the selected
        residue, while in yellow are the residues whose
        <i>C<sub>&beta;</sub></i> is within 5.0&Aring;. Each possible pair of
        residues in yellow with SER87 will be sequentially mutated to Cys and
        the application will attempted to create a disulfide bond.
      </li>
      <li>
        <i>Start the Disulfide Scan.</i><br />

<p>Press <b>OK</b>. The output database opens automatically into a
        Database Viewer. This database will contain the mutations along with
        the associated wild type protein. The Disulfide Scan progress is
        reported to the <a href="../moe/gui/svlcommands.htm">SVL Commands
        window</a> as well as to the top left corner of the MOE Window.</p>
      </li>
    </ol>
  </div>

<h5>Analyzing Results</h5>
  <div class="example">

<p>For more information on all the columns of the output database, please
    see <a href="#Output_Database">Output Database</a>. The main columns of
    interest in this example are:</p>
    <ul>
      <li><i>mutseq</i>, which is the id of the mutation within an individual
      mutation expression, where 0 indicates the wild type protein.</li>
      <li><i>mutation</i>, which identifies the exact mutation made for the
      corresponding mdb entry.</li>
      <li><i>dStability</i> which represents the change in stability over the
      wild-type protein.</li>
      <li><i>dS-S</i> which is the number of disulfide bonds compared to the
      wild type protein.</li>
    </ul>

<p>Of the three mutations made, two have created disulfide bonds. To easily
    analyze the resulting structures, you can browse the mutants by
    <span class="menu">DBV | File | Browse ...</span>, this will start the
    customized <a href="#Browsing_the_Output_Database">DB Browser for Protein
    Design</a>.</p>

<p>The disulfide bonds created in this example have been created in
    Subtilisin and reported in [Katz&nbsp;1986].</p>
    <div class="floatright"><img src="prodesign/example2_2st1_mutation.png"
    alt="2ST1 Mutation" class="floatright" /></div>
    <ol>
      <li>
        <i>Start the Protein Design Browser</i>.

<p class="blockquote"><span class="menu">DBV | File | Browse
        ...</span></p>
      </li>
      <li>
        <i>Hide the original 2st1 structure</i>.

<p class="blockquote"><span class="menu">MOE | System | 2ST1</span></p>
      </li>
      <li>
        <i>Isolate the mutation</i>.<br />

<p>Go to entry 2 in the MDB by clicking the next arrow in the browser.
        Follow this by enabling <b>Center Mutation</b> and <b>Overlay Wild
        Type</b>. This will isolate the mutation in the center of the screen,
        similar to <a href="../apps/ligx.htm#SiteView">SiteView</a> and allow
        you to easily compare the wild type conformation shown in gray and the
        mutant shown in green.</p>
      </li>
    </ol><br clear="all" />
  </div>
  <!-- ======================== Example 3 ============================== -->
<a id="Example3_ResidueScan"></a>
<h2>Example 3: Residue Scan on 1VFB</h2>
  <div class="example">

<p>1VFB is a crystal structure of the Fv fragment of an anti-hen egg white
    lysozyme antibody D1.3 antigen bound form. This example will be use to
    highlight how residue scanning can be used to modulate antibody affinity.
    This example requires that 1VFB (found in
    <tt>$MOE/sample/mol/1vfb.pdb.gz</tt>) be prepared using similar steps
    as in <a href="#Example1_Data_Preparation">Example 1</a>.</p>
    <div class="floatright"><img src="prodesign/example3_1vfb_h1loop.png" alt=
    "1VFB H1 Loop" class="floatright" /></div>
    <ol>
      <li>
      <i>Load and prepare the structure:</i>
      <p class="blockquote">
        <span class="menu">MOE | File | Open |</span>
          <b><tt>$MOE/sample/mol/ 1vfb.pdb.gz</tt></b>
        <ul>
          <li>Delete the solvent.</li>
          <li>Setup the forcefield and assign partial charges.</li>
          <li>Correct Structural Issues.</li>
          <li>Optimize hydrogen bond network.</li>
        </ul>
      </p>
      </li>
      <li>
        <i>Render Structure to help identify key mutations.</i><br />

<p>To help reduce the number of residues to mutate, it is beneficial to
        render the protein in such a way so that the antigen and antibody are
        clearly distinguished.</p>
        <ol type="i">
          <li>
            <i>Annotate secondary structure using antibody detection.</i>

<p class="blockquote"><span class="menu">SE | Annotate | Antibody |
            CCG</span></p>
          </li>
          <li>
            <i>Color atoms by chain.</i>

<p class="blockquote"><span class="menu">MOE | Footer | Atoms |
            Color by Chain</span></p>
          </li>
          <li>
            <i>Select H1 loop.</i>

<p class="blockquote"><span class="menu">MOE | Popup | Select | Set
            | <i>CDR_H1</i></span></p>
          </li>
          <li>
            <i>Color H1 Loop atoms Green
           (they will become visible in the next step).</i>

<p class="blockquote"><span class="menu">MOE | Footer | Atoms |
            Green</span></p>
          </li>
          <li>
            <i>Isolate H1 loop.</i>

<p class="blockquote"><span class="menu">MOE | RHS |
            SiteView</span></p>
          </li>
          <li>
            <i>Unselect Atoms.</i>

<p class="blockquote"><span class="menu">MOE | POPUP | Select Clear</span></p>
          </li>
        </ol><br clear="all" />
        <div class="floatright"><img src="prodesign/example3_1vfb_panel.png"
        alt="Residue Scan Panel" style="padding: 1em;" class="floatright" /></div>
      </li>
      <li>
        <i>Open the Residue Scan Panel</i>

<p class="blockquote"><span class="menu">MOE | Protein | Design |
        Residue Scan ...</span></p>
      </li>
      <li>
        <i>Identify residues to be mutated.</i><br />

<p>By default, since there are no atoms or residues selected and no
        ligands, no residues are shown in the Mutation List.
        Since this example focuses on the H1 CDR loop region,
        change the <b>Display</b> option:</p>

<p class="blockquote"><span class="menu">Residue Scan | Display |
        <i>CDR_H1</i></span></p>

<p>There should be 10 residues shown in the Mutation list. Select all
        residues and click on <b>Clear</b> in the Mutation Editor. By removing
        the mutations for these residues, they will not be searched. Most of
        the residues in the H1 loop do not interact with the antigen and
        therefore will be excluded in this example. We will focus only on
        Gly31, since there is space for a larger mutation to fit. Examination
        of the selected mutation site reveals that there are many charged
        residues present in the antigen (e.g. Arg112 and Lys116). To limit the
        mutation space, we will focus on creating either novel salt bridges by
        mutating to Glu or Asp, or Pi-Cation interactions by mutating to an
        aromatic residue. Select Gly31 in the list, and in the mutation editor
        box type <b>DEFWY</b> and click <b>Apply</b> to limit mutations to Asp,
        Glu, Phe, Trp and Tyr.</p>
      </li>
      <li>
        <i>Enable binding affinity calculation.</i>

<p>Since the antigen contains more than 10 residues, the affinity
        calculation will use MM/GBVI scoring. This scoring function should only
        be used as a pass/fail type score, where the user will define the
        pass/fail mark.</p>

<p class="blockquote"><span class="menu">Residue Scan |
        Affinity</span></p>Enable the checkbox to allow the affinity
        calculation. Select the antigen (the 1VFB.C chain) in the Sequence
        Editor.

<p class="blockquote"><span class="menu">Residue Scan | Affinity |
        Atoms | Selected Chains</span></p>To calculate protein properties for
        only the antibody chain, ensure first that Properties is enabled. With
        Properties enabled, the Properties Chains pulldown becomes available;
        choose Unselected Chains (the antigen chain was selected in the
        previous step).

<p class="blockquote"><span class="menu">Residue Scan | Properties |
        Chains | Unselected Chains</span></p>
      </li>
      <li>
        <i>Start the Residue Scan.</i><br />

<p>Press <b>OK</b>. The output database opens automatically into a
        Database Viewer. This database will contain the mutations along with
        the associated wild type protein. The Residue Scan progress is reported
        to the <a href="../moe/gui/svlcommands.htm">SVL Commands window</a> as
        well as to the top left corner of the MOE Window. This calculation will
        take a few minutes.</p>
      </li>
    </ol>
  </div>

<h5>Analyzing Results</h5>
  <div class="example">

<p>For more information on all the columns of the output database please
    see <a href="#Output_Database">The Output Database</a>. The main columns of
    interest in this example are:</p>
    <ul>
      <li><i>mutseq</i>, which is the id of the mutation within an individual
      mutation expression where 0 indicates the wild type protein.</li>
      <li><i>mutation</i>, which identifies the exact mutation made for the
      corresponding mdb entry.</li>
      <li><i>dStability</i> which represents the change in stability over the
      wild-type protein.</li>
      <li><i>dAffinity</i> which is the relative affinity of the mutation
      compared to the wild-type protein.</li>
    </ul>

<p>To better understand the gains or losses in affinity, the <a href=
    "#Browsing_the_Output_Database">Protein Design Browser</a> can be used.</p>
    <div class="floatright"><img src="prodesign/example3_1vfb_mutation.png"
    alt="1vfb Mutation" class="floatright" /></div>
    <ol>
      <li>
        <i>Start the Protein Design Browser</i>.

<p class="blockquote"><span class="menu">DBV | File | Browse
        ...</span></p>
      </li>
      <li>
        <i>Hide the original 1VFB structure</i>.

<p class="blockquote"><span class="menu">MOE | System | 1VFB</span></p>
      </li>
      <li>
        <i>Isolate the mutation</i>.<br />

<p>Enable <b>Isolate</b> and <b>Overlay Wild Type</b>. This
        will isolate the mutation in the center of the screen, similarly to
        <span class="menu">MOE | RHS | <a href=
        "ligx.htm#SiteView">SiteView</a></span>, and allow you to easily
        compare the wild type conformation shown in gray to the mutant shown in
        green.</p>

<p>These options allow you to easily view and assess the
        mutations and whether any new interactions were formed.</p>
      </li>
    </ol><br clear="all" />
  </div>
<a id="References"></a>
<h1>References</h1>
  <table class="ref">
    <tr>
      <td valign="top">[Dehouck&nbsp;2009]</td>
      <td valign="top">Dehouck, Y., Grosfils, A., Folch, B., Gilis, D.,
      Bogaerts, P., Rooman, M.; Fast and accurate predictions of protein
      stability changes upon mutations using statistical potentials and neural
      networks: PoPMuSiC-2.0; <i>Bioinformatics 25</i> (<b>2009</b>)
      2537&ndash;2543.</td>
    </tr>
    <tr>
      <td valign="top">[Goldstein&nbsp;1994]</td>
      <td valign="top">Golstein, R.F.; Efficient rotamer elimination applied to
      protein side chains and related spin glasses; <i>Biophys J. 66</i>
      (<b>1994</b>) 1335&ndash;1340.</td>
    </tr>
    <tr>
      <td valign="top">[Guerois&nbsp;2002]</td>
      <td valign="top">Guerois, R., Nielsen, J.E., Luis Serrano, L.; <i>J. Mol.
      Biol. 320</i> (<b>2002</b>) 369&ndash;387.</td>
    </tr>
    <tr>
      <td valign="top">[Katz&nbsp;1986]</td>
      <td valign="top">Katz, B.A. and Kossiakoff, A.; <i>J. Biol. Chem. 261</i>
      (<b>1986</b>) 15480&ndash;15485.</td>
    </tr>
    <tr>
      <td valign="top">[Labute&nbsp;2009]</td>
      <td valign="top">Labute, P.; Protonate3D: Assignment of Ionization States
      and Hydrogen Coordinates to Macromolecular Structures; <i>Proteins 75</i>
      (<b>2009</b>) 187&ndash;205.</td>
    </tr>
  </table>
<a id="See_Also"></a>
<h1>See Also</h1>

<p><a href="structprep.htm">Structure Preparation</a><br />
  <a href="../proteins/ensprop.htm">Protein Properties</a><br />
  <a href="../apps/mutanal.htm">Mutation Analysis</a><br />
  <a href="../proteins/probuild.htm">Protein Builder and Rotamer Explorer</a>

<!-- START MOE_FOOTER -->
  <div class="MOE_FOOTER">
    <img src="../images/ccgicon.png" /> <a href="../index.htm"></a> <a href=
    "../legal.htm"></a> &copy;<span class="versionyear"></span> <a href="http://www.chemcomp.com"></a>. All
    rights reserved.<br />
    <a href="mailto:info@chemcomp.com"></a>
  </div><!-- END MOE_FOOTER -->
</div>
</div>
</body>
</html>

