<!DOCTYPE html>
<html>


<!--
!!    MOE On-Line Manuals
!!    COPYRIGHT (C) CHEMICAL COMPUTING GROUP ULC.  ALL RIGHTS RESERVED.
!!-->
<head>
  <meta http-equiv="x-ua-compatible" content="IE=9" />
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <link rel="stylesheet" type="text/css" href="../../include/manstyle.css" />
  <link rel="icon" type="image/png" href="../../images/ccgicon.png" />
  <link rel="shortcut icon" type="image/png" href="../../images/ccgicon.png" />
  <meta name="keywords" content="panel: MOE" />
  <meta name="keywords" content="panel: Atoms (Atom Rendering)" />
  <meta name="keywords" content="panel: Ribbon (Ribbon Rendering)" />
  <meta name="keywords" content="panel: Delete Atoms" />
  <meta name="keywords" content="panel: Sketch View" />
  <meta name="keywords" content="panel: Dials" />
  <meta name="keywords" content="panel: Suspend" />
  <meta name="keywords" content="panel: 2D Overlay" />
  <meta name="keywords" content="panel: Superpose Molecules" />
  <meta name="keywords" content="panel: Interactive Superposition" />
  <title>MOE Window</title>
  <meta name="keywords" content=
  "keywords: select bond, system list, meter, restraint, constraint" />
  <meta name="keywords" content=
  "keywords: stereochemistry, contacts, footer, fog, clip, chain tag" />
  <meta name="keywords" content=
  "keywords: quick surface, ligand, right button bar, trackball, view" />
  <meta name="keywords" content=
  "keywords: ligx, render, copy, paste, clipboard, auto-type" />
  <meta name="keywords" content=
  "keywords: chirality, CLI, stereo, mouse, measure, distance" />
  <meta name="keywords" content="keywords: angle, dihedral, style" />
  <meta name="keywords" content="keywords: sketcher, drag and drop" />
  <meta name="keywords" content="keywords: interactive superposition, superpose" />
  <meta name="keywords" content="keywords: Atom Selector, Selection Prompter" />
  <meta name="keywords" content="keywords: picking, select atoms" />
  <meta name="keywords" content="keywords: select ribbons, auto-advance" />
  <meta name="keywords" content="keywords: 2D overlay, overlay" />
</head>
<body>
<div class="center-page">
  <!-- START MOE_HEADER -->
  <div class="MOE_HEADER">
    <a href="../../index.htm"><img src="../../images/ccglogo.png" /></a>
    <a href="../../index.htm"><div class="moeversion"></div></a>
    MOE Window
  </div>
<!-- END MOE_HEADER -->
<div class="content">

<p>This document introduces the MOE Window and explains how to select, render
  and edit molecular systems. It does not cover all the commands that are
  available in the MOE Window. If you are looking for information on a specific
  menu or command, please refer to the <a href="#MenuBar">MOE Menu Bar</a> and
  to the <a href="../../apps/ligx.htm#ButtonBar">QuickPrep: MOE Right Button
  Bar</a> chapter.</p>
  <ul>
    <li>
      <a href="#Introduction">Introduction</a>
    </li>
    <li>
      <a href="#MOEWindow">The MOE Window</a>
      <ul>
        <li>
          <a href="#MenuBar">MOE Menu Bar</a>
        </li>
        <li>
          <a href="#CommandLine">Command Line</a>
        </li>
        <li>
          <a href="#3DRenderingArea">3D Rendering Area</a>
        </li>
        <li>
          <a href="#BubbleHelp">Atom Popup Information</a>
        </li>
        <li>
          <a href="#Footer">Footer</a>
        </li>
        <li>
          <a href="#MOEPopup">MOE Window Popup Menu</a>
        </li>
      </ul>
    </li>
    <li>
      <a href="#Selecting">Selecting Molecular Objects</a>
      <ul>
        <li>
          <a href="#UsingMouse">Selecting Atoms Using the Mouse</a>
        </li>
        <li>
          <a href="#SelectRibbon">Selecting Ribbons Using the Mouse</a>
        </li>
        <li>
          <a href="#UsingMenu">Selecting Atoms Using the Select Menu</a>
        </li>
        <li>
          <a href="#SelectionPrompter">Selecting Atoms Using the Selection
          Prompter</a>
        </li>
      </ul>
    </li>
    <li>
      <a href="#Viewing">Rotating, Translating and Zooming the View</a>
    </li>
    <li>
      <a href="#CenterRotation">Changing the Center of Rotation</a>
    </li>
    <li>
      <a href="#Rendering">Rendering Molecular Objects</a>
      <ul>
        <li>
          <a href="#HideShow">Hiding and Showing Atoms</a>
        </li>
        <li>
          <a href="#RenderingAtoms">Rendering Atoms and Bonds</a>
        </li>
        <li>
          <a href="#Color">Coloring Atoms</a>
        </li>
        <li>
          <a href="#Label">Labeling Atoms</a>
        </li>
        <li>
          <a href="#RenderingBackbone">Rendering Ribbons and Backbones</a>
        </li>
        <li>
          <a href="#Contacts">Rendering Nonbonded Contacts</a>
        </li>
        <li>
          <a href="#Draw">Draw Settings</a>
        </li>
        <li>
          <a href="#DisplayModes">Stereo Mode</a>
        </li>
        <li>
          <a href="#DuplicateWindow">Duplicate 3D Window</a>
        </li>
        <li>
          <a href="#Rock&Roll">Rock and Roll</a>
        </li>
      </ul>
    </li>
    <li>
      <a href="#Clipping">Clipping Planes and Depth Cueing</a>
    </li>
    <li>
      <a href="#2dLigandOverlay">2D Ligand Overlay</a>
    </li>
    <li>
      <a href="#Editing">Editing Molecular Objects</a>
      <ul>
        <li>
          <a href="#Undo">Undo and Redo</a>
        </li>
        <li>
          <a href="#EditingAtomsUsingTheMouse">Editing Atoms Using the
          Mouse</a>
        </li>
        <li>
          <a href="#RotatingAboutABond">Rotating about a Bond</a>
        </li>
        <li>
          <a href="#EditingAtomsUsingTheEditMenu">Editing Atoms Using the Edit
          Menu</a>
        </li>
        <li>
          <a href="#2DEditing">2D Editing: Sketchers</a>
        </li>
        <li>
          <a href="#Copy">Copy and Paste</a>
        </li>
        <ul>
          <li>
            <a href="#DnD">Drag and Drop</a>
          </li>
        </ul>
        <li>
          <a href="#Duplicate">Duplicate</a>
        </li>
        <li>
          <a href="#SketchView">Sketch View</a>
        </li>
        <li>
          <a href="#DeletingAtoms">Deleting Atoms</a>
        </li>
        <li>
          <a href="#Hydrogens">Hydrogens and Lone Pairs</a>
        </li>
        <li>
          <a href="#Automatic">Automatic Bonding and Typing</a>
        </li>
        <li>
          <a href="#Chirality">Chirality Constraints</a>
        </li>
        <li>
          <a href="#EditPotential">Potential Energy Attributes</a>
        </li>
        <li>
          <a href="#Measure">Creating and Deleting Meters</a>
        </li>
        <li>
          <a href="#InteractiveSuperposition">Interactive Superposition
	  (Superpose Molecules)</a>
        </li>
      </ul>
    </li>
    <li>
      <a href="#ProteinAnnotation">Protein Annotation</a>
    </li>
    <li>
      <a href="#Priority">System Priority</a>
    </li>
    <li>
      <a href="#ClosingTheSystem">Closing the System</a>
    </li>
    <li>
      <a href="#Cancel">Canceling Tasks</a>
    </li>
    <li>
      <a href="#SVL">SVL Commands</a>
    </li>
    <li>
      <a href="#Tutorials">Tutorials and Courses</a>
    </li>
    <li>
      <a href="#References">References</a>
    </li>
  </ul>

<h1><a id="Introduction">Introduction</a></h1>

<p>The MOE Window is where molecular systems are rendered and manipulated at
  the atomic level. It always reflects the current state of the system, and
  animates calculations such as energy minimization and molecular dynamics. The
  3D rendering area provides different renderings of molecular objects such as
  atomic nuclei, bonds and backbone ribbons. Graphic objects such as
  electrostatic and water-accessible surfaces (or other graphical objects
  specified in SVL) can be rendered and displayed with the molecular system
  under study. Dynamically calculated distance meters and angle measurements
  can also be drawn, and general visualization settings can be changed at any
  time.</p>

<p>The <a href="#MenuBar">menu bar</a> at the top of the MOE Window provides
  direct access to commonly used commands, applications and tools. The MOE
  Window includes a <a href="#CommandLine">command line interface (CLI)</a>
  (raised by pressing the <tt>Space</tt> bar while the MOE Window has the
  focus) which is used to enter SVL commands and also serves as a prompt area
  for atom editing. Both the menu and the
  <a href="../../apps/ligx.htm#ButtonBar">right button bar</a> are fully
  <a href="../../install/moeconfig.htm#CustomMenus">customizable</a>. The
  <a href="#Cancel">Cancel menu</a> in the top right corner of the window allows you to
  halt any tasks running in MOE.</p>
  <img src="mainwin/mainwin.png" alt="MOE Window" class="center" />

<p>MOE also offers a collection of molecular builders that build and edit
  systems of varying complexity, from small molecules and carbohydrates to
  proteins and crystals:</p>
  <ul>
    <li>
      <a href="../molsystems/build.htm">Builder</a>
    </li>
    <li>
      <a href="../../apps/carbo.htm">Carbohydrate Builder</a>
    </li>
    <li>
      <a href="../../proteins/probuild.htm">Protein Builder</a>
    </li>
    <li>
      <a href="../../proteins/dnabuild.htm">DNA/RNA Builder</a>
    </li>
    <li>
      <a href="../../apps/crystal.htm">Crystal Parameters</a>
    </li>
    <li>
      <a href="../../apps/polybld.htm">Polymer Builder</a>
    </li>
  </ul>

<p>To exit MOE, choose <span class="nobr"><b>MOE | File | Quit</b></span>.
  MOE will display a warning if molecular data has not been saved. You can
  either cancel the Quit operation or quit MOE and discard unsaved data. If
  nothing has been modified, MOE will terminate.</p>

<p>Errors in MOE are signaled by an error box which will prompt you for
  acknowledgement (e.g. when a syntactically incorrect command is entered at
  the CLI, or when atoms are deleted from the system while a calculation
  involving atoms is on-going). MOE will be suspended until you press <b>OK</b>
  in the error box, or until you press <tt>Enter</tt> or <tt>Esc</tt> in any of
  the MOE windows.</p>

<a id="The_MOE_Window"></a>
<a id="MOE_Window"></a>

<h1>
<a id="MOEWindow">The MOE Window</a></h1>

<a id="MOE_Menu_Bar"></a>
<h2>
<a id="MenuBar">MOE Menu Bar</a></h2>

<p>The menu bar in the MOE Window is the entry point to many of MOE's
  operations. The menu bar is fully programmable, allowing you to add new menus
  or to extend an existing menu. For more information on how to customize the
  menu bar, see <a href="../../install/moeconfig.htm">Customizing Menus in
  MOE</a>.</p>

<p>The following is a list of the standard built-in menu items in the MOE
  menu bar. For direct access to information on a specific command, click on
  the command name. Certain commands are described in this document whereas
  others are described in separate files.</p>
  <table class="noborder width-full">
    <tr>
      <td nowrap="nowrap" width="25%">
        <fieldset>
          <legend><b>File</b></legend>
		  New<br />
&nbsp;&nbsp;&nbsp;<a href="../../mdb/db.htm">Database</a><br />
&nbsp;&nbsp;&nbsp;<a
	    href="../../apps/ph4_query.htm">Pharmacophore&nbsp;Query</a><br />
&nbsp;&nbsp;&nbsp;<a href="ted.htm">Text&nbsp;Editor</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/moevie.htm">MOEvie</a><br />
&nbsp;&nbsp;&nbsp;<a
	    href="../../moesaic/overview.htm">MOEsaic</a><br />
          <a href="./fileopen.htm">Open</a><br />
          <a href="../../apps/filesave.htm">Save</a><br />
          <a href="#ClosingTheSystem">Close</a><br />
          <a href="../../apps/print.htm">Print</a><br />
          <a href="../hpc.htm#JobManager">Job Manager</a><br />
          <a href="../../install/install.htm#InstallStatus">Install
          Status</a><br />
          <a href="#System_Priority">Suspend</a><br />
          Quit
        </fieldset><br />
        <fieldset>
          <legend><b><a href="#Editing">Edit</a></b></legend>
          <a href="#Undo">Undo</a><br />
          <a href="#Undo">Redo</a><br />
          <a href="#Copy">Copy As</a><br />
          <a href="#Copy">Copy</a><br />
          <a href="#Copy">Paste</a><br />
          <a href="#Duplicate">Duplicate</a><br />
          <a href="#DeletingAtoms">Delete</a><br />
          Build<br />
&nbsp;&nbsp;&nbsp;<a href="#Hydrogens">Hydrogens</a><br />
&nbsp;&nbsp;&nbsp;<a href="../molsystems/build.htm">Molecule</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/carbo.htm">Carbohydrate</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/polybld.htm">Repeat&nbsp;Unit</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/polybld.htm">Polymer</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../proteins/probuild.htm">Protein</a><br />
&nbsp;&nbsp;&nbsp;<a href="#PTM">PTM</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../proteins/dnabuild.htm">DNA/RNA</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../proteins/buildrotlib.htm">Rotamer&nbsp;Library</a><br />
&nbsp;&nbsp;&nbsp;<a href="#Automatic">Automatic</a><br />
&nbsp;&nbsp;&nbsp;<a href="#Chirality">Chirality</a><br />
          <a href="#EditPotential">Potential</a><br />
          <a href="../../apps/solvate.htm">Solvate</a><br />
          <a href="#Measure">Measure</a><br />
          <a href="#SettingsMenu">Preferences</a><br />
        </fieldset>
      </td>
      <td nowrap="nowrap" width="25%">
        <fieldset>
          <legend><b><a href="#Selecting">Select</a></b></legend>
          <a href="#Invert">Invert</a><br />
          <a href="#Clear">Clear</a><br />
          <a href="#All">All</a><br />
          <a href="#Extend">Extend</a><br />
          <a href="#Hydrogens">Hydrogens</a><br />
          <a href="#Ligand">Ligand</a><br />
          <a href="#Pocket">Pocket</a><br />
          <a href="#Receptor">Receptor</a><br />
          <a href="#Interface">Interface</a><br />
          <a href="#Solvent">Solvent</a><br />
          <a href="#Dummy">Dummy</a><br />
          <a href="#Set">Set</a><br />
          <a href="#SetCreate">Set&nbsp;Create</a><br />
          <a href="#SetDelete">Set&nbsp;Delete</a><br />
          <a href="#Selector">Selector</a><br />
          <a href="#SelectGeo">Stereo</a><br />
          <a href="#Potential">Potential</a><br />
          <!--
        <a href="#Protein">PTM&nbsp;Sites</a><br />
-->
           <a href="#Protein">Protein</a><br />
          <!--
        <a href="#Attachment">Attachment</a><br />
        <a href="#Element">Element</a><br />
-->
           <a href="#Save">Save</a><br />
          <a href="#Restore">Restore</a><br />
          <a href="#Synchro">Synchronize</a><br />
        </fieldset><br />
        <fieldset>
          <legend><b><a href="#Rendering">Render</a></b></legend>
          <a href="#Viewing">Center</a><br />
          <a href="../../apps/ligx.htm#SiteView">SiteView</a><br />
          <a href="../../proteins/pdb_mdb.htm#RenderProtein">Protein</a><br />
          <a href="#HideShow">Hide</a><br />
          <a href="#HideShow">Show</a><br />
          <a href="#RenderingAtoms">Atoms</a><br />
          <a href="#RenderingBackbone">Ribbon</a><br />
          <a href="#Contacts">Contacts</a><br />
          <a href="label.htm">Atom Labels</a><br />
          <a href="../../appendix/viewsetup.htm">Setup</a><br />
          <a href="#DisplayModes">Stereo</a><br />
          <a href="#Rock&Roll">Rock&nbsp;and&nbsp;Roll</a><br />
          <a href="#FullScreen">Full&nbsp;Screen&nbsp;(F12)</a><br />
        </fieldset><br />
        <fieldset>
          <legend><b>Protein</b></legend> Search<br />
&nbsp;&nbsp;&nbsp; <a href="../../proteins/prosearch.htm">PDB&nbsp;Search</a><br />
&nbsp;&nbsp;&nbsp; <a href="../../proteins/motifsearch.htm">Domain&nbsp;Motif</a><br />
&nbsp;&nbsp;&nbsp; <a href="../../proteins/projsrch.htm">Project&nbsp;MDB</a><br />
          <a href="../../apps/structprep.htm">Structure&nbsp;Preparation</a><br />
          <a href="../../proteins/alignsup.htm">Align/Superpose</a><br />
          <a href="../../proteins/procore.htm">Consensus</a><br />
          <a href="#ProteinAnnotation">Annotate</a><br />
          <a href="../../proteins/probuild.htm">Protein&nbsp;Builder</a><br />
          <a href="../../proteins/dnabuild.htm">DNA/RNA&nbsp;Builder</a><br />
          <a href="../../proteins/promodel.htm">Homology&nbsp;Model</a><br />
          <a href="../../proteins/antibody.htm">Antibody&nbsp;Modeler</a><br />
          <a href="../../apps/proloop.htm">Loop&nbsp;Modeler</a><br />
          <a href="../../apps/proloop.htm">Linker&nbsp;Modeler</a><br />
          <a href="../../apps/prodesign.htm">Design</a><br />
          <a href="../../proteins/propatch.htm">Patch&nbsp;Analyzer</a><br />
          <a href="../../proteins/ensprop.htm">Properties</a><br />
          <a href="../../apps/progeom.htm">Geometry</a><br />
          <a href="../../proteins/procontacts.htm">Contacts</a><br />
          <a href="../../apps/docking.htm#Protein_Method">Protein-Protein
          Dock</a>
        </fieldset><br />
      </td>
      <td nowrap="nowrap" width="25%">
        <fieldset>
          <legend><b>Compute</b></legend> Prepare<br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/ligx.htm#ButtonBar">QuickPrep</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/structprep.htm">Structure&nbsp;Preparation</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/protonate3d.htm">Protonate&nbsp;3D</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/protoview.htm">Protomers</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/residuepka.htm">Residue pKa</a><br />
&nbsp;&nbsp;&nbsp;<a href="../pot.htm">Partial Charges</a><br /><a href="../pot.htm">Potential Energy</a><br />
          <a href="../../apps/emin.htm">Energy Minimize</a><br />
		  Conformations<br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/confsrch.htm">Search</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/confgui.htm">Import</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/mogul.htm">Mogul Analysis</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/torprofile.htm">Torsion Profile</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/dihedralcontour.htm">Dihedral Contour Plot</a><br />
          <a href="../../apps/flex.htm">Flexible Alignment</a><br />
          <a href="#InteractiveSuperposition">Superpose Molecules</a><br />
          <a href="../../apps/sitefind.htm">Site Finder</a><br />
          <a href="../../apps/docking.htm">Dock</a><br />

          <a href="../../apps/surfmap.htm">Surfaces&nbsp;and&nbsp;Maps</a><br />
          <a href="../../apps/prolig2d.htm">Ligand Interactions</a><br />
          <a href="../../apps/view2d.htm">2D Molecules</a><br />
          <a href="../../apps/ph4_guide.htm">Pharmacophore</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/ph4_query.htm">Query Editor</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/ph4_elu.htm">Elucidation</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/ph4_search.htm">Search</a><br />
          Fragments<br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/scaffold.htm">Scaffold Replacement</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/scaffold.htm">Link Multiple Fragments</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/scaffold.htm">Add Group to Ligand</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/shaman.htm">MedChem Transformations</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/breed.htm">BREED</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/frag_combi.htm">Combinatorial Builder</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/mfss.htm">MultiFragment Search</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/recap.htm">RECAP Analysis</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/recap.htm">RECAP Synthesis</a><br />
		  <a href="../../quasar/qsar.htm">QuaSAR</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/sar.htm">SAReport</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/combitools.htm">Combinatorial Library</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../quasar/modcomp.htm">Model-Composer</a><br />
          <!--
        &nbsp;&nbsp;&nbsp;
        <a href="../../appendix/qcombi.htm">QuaSAR-CombiDesign</a><br />
-->
&nbsp;&nbsp;&nbsp;<a href="../../quasar/hole.htm">QuaSAR-HoleFiller</a><br />
          Simulations<br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/md.htm">Dynamics</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/md.htm#Simulation_Analysis">Dynamics
	Player</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/ti.htm">Thermodynamic
	Integration</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/rism.htm">Solvent Analysis</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/scf.htm">SCF Calculation</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/diffract.htm">Diffraction</a><br />
&nbsp;&nbsp;&nbsp;<a href="../../apps/polymer.htm">Polymer Properties</a><br />
          <a href="#Priority">Priority</a><br />
        </fieldset><br />
      </td>
      <td nowrap="nowrap" width="25%">
        <fieldset>
          <legend><b>Window</b></legend> <a href="#Introduction">MOE<br />
          Windows</a><br />
&nbsp;&nbsp;&nbsp;Raise All<br />
&nbsp;&nbsp;&nbsp;<a href="../../mdb/db.htm">Database Viewers</a><br />
&nbsp;&nbsp;&nbsp;<a href="ted.htm">Text Editors</a><br />
          <a href="se.htm">Sequence Editor</a><br />
          <a href="system.htm">System Manager</a><br />
          <a href="atommgr.htm">Atom Manager</a><br />
          <a href="gobjman.htm">Graphics Objects</a><br />
          <a href="svlcommands.htm">Commands</a><br />
        </fieldset><br />
        <fieldset>
          <legend><b>Help</b></legend> Contents<br />
          Panel Index<br />
          Function Index<br />
          <a href="mouse.htm#MouseReference">Mouse</a><br />
          Getting Started<br />
          <a href="#Tutorials">Tutorials</a><br />
          <a href="#Tutorials">Courses</a><br />
          <a href="../../rnotes.htm">Release Notes</a><br />
          <a href="../../install/install.htm#LicenseInfo">License
          Info</a><br />
          <a href="../../appendix/viewsetup.htm#GraphicsCardInformation">OpenGL
          Info</a><br />
        </fieldset><br />
        &nbsp;<b><a href="#SVL_Button">SVL</a></b><br />
        &nbsp;<b><a href="#SVL_Button">DBV</a></b><br />
        &nbsp;<b><a href="se.htm">SEQ</a></b><br />
        <br />
        &nbsp;<img src="mainwin/gear_wheel.png" class="icon" /> <b>
        <a href="#SettingsMenu">Settings</a></b><br />
        <br />
        &nbsp;<b><a href="#Cancel">Cancel</a></b>
      </td>
    </tr>
  </table>

<a id="Command_Line"></a>
<h2>
<a id="CommandLine">Command Line</a></h2>
  <div class="floatright"><img src="mainwin/cli.png" alt="CLI in MOE" class="floatright" /></div>

<p>The <a href="svlcommands.htm">Command Line Interface (CLI)</a> is where
  SVL commands are typed and issued. The CLI is hidden by default, but becomes
  accessible when <tt>Space</tt> is pressed and the focus is in the MOE Window.
  While the MOE Window has the focus, all keystrokes are automatically directed
  to the CLI regardless of where the mouse is positioned within the window.</p>

<p>Non-ASCII characters are displayed in the CLI as empty squares. When such
  characters are entered at the CLI, the CLI content is flashed to warn of
  their presence.</p>

<p>Text can be cut from the CLI using <tt>Ctrl+X</tt>. The key combination
  <tt>Ctrl+A</tt> causes all the text in the CLI to become selected.</p>

<p>Commands entered in the CLI are executed when <tt>Return</tt> is pressed.
  On successful execution the CLI is closed and can be reopened with pressing
  <tt>Space</tt> again. When using <tt>Shift+Return</tt> to execute the
  command, the CLI is kept open and additional SVL commands can be issued right
  away.</p>

<p>The command line area is also used as a prompt area for several MOE
  applications and functions. For example, the <a href="sel_ui.htm">Selection
  Prompter</a> and the <a href="../../apps/moevie.htm">MOEvie</a> maker
  interfaces occupy this area. The prompt area may also be for entering data or
  simply for displaying messages. In all cases, the prompt may be canceled by
  pressing either the red 'X' or <tt>Esc</tt>. While the MOE
  Window's CLI is occupied by a prompt, the CLIs in other windows will
  function as usual.</p>

<p>For more information on the Command Line Interface, see the
  <a href="svlcommands.htm">SVL Commands window</a>.</p>

<a id="SVL_Button"></a>
<h2>SVL, DBV, and SEQ Buttons</h2>

<p>The <b>SVL</b>, <b>DBV</b>, and <b>SEQ</b> buttons are shortcuts for
  raising the <a href="svlcommands.htm">SVL Commands window</a>, a list of
  currently open <a href="../../mdb/db.htm">Database Viewers</a>, and the
  <a href="se.htm">Sequence Editor</a>, respectively. The shortcut
  <tt>Ctrl+Space</tt> will also open the SVL Commands window. When
  <tt>&lt;Ctrl&gt;</tt> is held down, the <b>SVL</b> button also serves as a
  shortcut for raising the <a href="modtasks.htm">SVL Modules &amp; Tasks</a>
  panel. Furthermore, when <tt>&lt;Alt&gt;</tt> is held down, the <b>SVL</b>
  button also serves as a shortcut for raising the
  <a href="modtasks.htm#SVLCrashHistory">Crash History</a> panel.</p>
<a id="SettingsMenu"></a>
<h2>Settings</h2>

<p>The Settings menu collects together configuration and customization tools.
  These are described in greater detail in
  <a href="../../appendix/settings.htm">MOE Settings Menu</a>.</p>
  <center>
    <table class="titlebar">
      <tr>
        <th>Command</th>
        <th>Description</th>
      </tr>
      <tr>
        <td>
<a id="Appearance"><b>Appearance</b></a>
        </td>
        <td>Toggles the appearance of MOE between MOE-Blanc (light mode) and
        MOE-Noir (dark mode). The current setting becomes the default.</td>
      </tr>
      <tr>
        <td><b>Builder</b><br />
        <b>QuickPrep</b><br />
        <b>Protein&nbsp;Defaults</b><br />
        <b>PSILO/RCSB</b><br />
        <b>Java/JDBC</b><br />
        <b>Proxy&nbsp;Settings</b><br />
        <b>MOE/web&nbsp;Settings</b><br />
        <b>Function&nbsp;Keys</b><br />
        <b>Miscellaneous</b><br />
        <b>User&nbsp;Interface</b></td>
        <td>
          Opens the <a href="../../appendix/settings.htm">Configuration
          Options</a> panel to the appropriate page.
        </td>
      </tr>
      <tr>
        <td><b>Mouse</b></td>
        <td>
          Opens the <a href="../../appendix/settings.htm#MainMouse">Mouse
          Configuration</a> panel.
        </td>
      </tr>
      <tr>
        <td>
          <a href="../../appendix/settings.htm#DrawMenu"><b>Draw
          Settings</b></a>
        </td>
        <td>Opens a submenu for setting fonts, drawing coordinate axes, and
        setting the quality of rotation and anti-aliasing.</td>
      </tr>
      <tr>
        <td>
          <a href="../../appendix/settings.htm#StereoMenu"><b>Stereo
          Mode</b></a>
        </td>
        <td>
          Opens a submenu for setting the default stereo mode. The
          <a href="../../appendix/viewsetup.htm#StereoSettings">Stereo Settings</a> and
          <a href="zspace.htm#zSpace_configuration_settings">zSpace Settings</a>
          panels can be opened from here.
        </td>
      </tr>
    </table>
  </center>

<a id="3D_Rendering_Area"></a>
<h2>
<a id="3DRenderingArea">3D Rendering Area</a></h2>

<p>The 3D rendering area is MOE's main visualization area for viewing the
  current molecular system. The mouse is the main vehicle for atom manipulation
  in this area. A quick reference for all the default mouse actions can be
  found in <a href="mouse.htm">Using the Mouse</a>.</p>

<p>The right mouse button brings up the <a href="#MOEPopup">popup</a> menu,
  which contains a selection of commands that overlap both the
  <a href="#MenuBar">menu bar</a> and the <a href="../../apps/ligx.htm#ButtonBar">right
  button bar</a>.</p><!--
    <div class="floatright">
    <img src="mainwin/font.png" alt="MOE Messages" class="floatright" />
    </div>
-->

<p>At the top of the window, messages are often displayed during
  calculations. <!--
    For example, while running the GizMOE minimizer you might see a message
    such as the one shown here.
-->
   The font used for these messages may be customized and saved in the MOE
  resource configuration file using the <span class="menu">MOE | Settings |
  Draw Settings</span> font toggle buttons. To save the current values for
  future MOE sessions use the <span class="menu">Save As Default</span> option
  located at the bottom of this menu. For more information on customizing MOE
  see the <a href="../../appendix/settings.htm">Settings Menu</a>.</p>

<div class="floatright">
<img src="mainwin/bubblehelp.png" alt="Atom Info" class="floatright" />
</div>
<h2><a id="BubbleHelp">Atom Popup Information</a></h2>

<p>Hovering the mouse over an atom, bond, or ribbon segment while
the <tt>Shift</tt> key is pressed will pop up a bubble help
with information about the specific atom, bond or residues associated
with the ribbon segment. The information includes atom name, charge,
chirality, force field type, residue name and number, etc.
The modifier key which needs to be pressed to pop up the
information can be configured in the
<a href="../../appendix/settings.htm#Miscellaneous">Settings Menu</a>.</p>

<a id="Button_Bars"></a>
<h2>
<a id="ButtonBars">Button Bars</a></h2>

<p>The MOE Window includes a button bar to the right of the rendering area.
  Abbreviated <b>RHS</b> (for &quot;right-hand side&quot;), the right button
  bar includes shortcuts to the commands in the menus and is entirely
  customizable. It is also possible to program and display a left button bar in
  the MOE Window. See
  <a href="../../install/moeconfig.htm#CustomMenus">Customizing your Menus</a> for
  information on adding commands. A full description of the MOE right button bar
  is given in the chapter <a href="../../apps/ligx.htm#ButtonBar">QuickPrep:
  MOE Right Button Bar</a>.
</p>

<h2>
<a id="Footer">Footer</a></h2>

<p>The footer located at the bottom of the MOE Window provides a trackball, a
  zoom dial, clipping and fogging sliders, atom and ribbon rendering options,
  and forcefield and periodic systems controls. Additionally, the footer area
  provides system-dependent information and, depending on the atom selection
  set, may also provide controls for editing atom properties, interatomic
  distances and angles. Bubble help in the footer area gives information about
  the current system.</p>

<p>The footer area comprises the following elements:</p>
  <ul>
    <li>
      <a href="#StereoIcon"><i>Stereo</i></a>. Button for turning on and off the
      <a href="../../appendix/settings.htm#StereoMenu">default stereo mode</a>.
    </li>
    <li>
      <a href="#Themes"><i>Themes</i></a>. Menu for changing themes and for
      copying and saving pictures.
    </li>
    <li>
      <a href="#RSL"><i>R</i>, <i>S</i>, <i>L</i>, &#8680;</a>. Buttons for
      controlling mouse selection in the MOE Window.
    </li>
    <li>
      <a href="#Trackball"><i>Trackball</i></a>. Trackball for rotating,
      translating, and zooming the view; <b>View</b> popup menu.
    </li>
    <li>
      <a href="#Dial"><i>Dial</i></a>. Zoom dial.
    </li>
    <li>
      <a href="#Fog"><i>Fog</i></a>. Depth Cue settings.
    </li>
    <li>
      <a href="#ZClip"><i>ZClip</i></a>. Clipping plane settings.
    </li>
    <li>
      <a href="#2D"><i>2D</i></a>. Two-dimensional ligand overlay.
    </li>
    <li>
      <a href="#Contacts"><i>Contacts</i></a>. Non-bonded contacts controls.
    </li>
    <li>
      <a href="#RenderingAtoms"><i>Atoms</i></a>. Atom render panel.
    </li>
    <li>
      <a href="#RenderingBackbone"><i>Ribbon</i></a>. Backbone ribbon render
      panel.
    </li>
    <li>
      <a href="#Forcefield"><i>Forcefield</i></a>. Opens the
      <a href="../pot.htm#Forcefield">Potential Setup</a> panel for selecting
      forcefields and solvation models.
    </li>
    <li>
      <a href="#CrysParm"><i>Crystal Parameters</i></a>. Opens the
      <a href="../../apps/crystal.htm">Crystal Parameters</a> panel for enabling
      periodic boundary conditions and installing crystal parameters.
    </li>
    <li>
      <a href="#FooterEditing"><i>Footer Editing</i></a>. Selection-sensitive
      area for reporting information about the system and for:
      <ul>
        <li>Editing atom name.</li>
        <li>Measuring/editing interatomic distances.</li>
        <li>Measuring/editing atom angles.</li>
        <li>Measuring/editing atom dihedral angles.</li>
      </ul>
    </li>
  </ul>

<a id="footer_image"></a>
<p class="center"><img src="mainwin/moefooter.png" alt="MOE Footer" /></p>
<a id="StereoIcon"></a>
  <dl>
    <dt><b>Stereo</b> <img src="mainwin/stereo_icon.png" style=
    "vertical-align:middle" /></dt>
    <dd>
    <p>The <b>Stereo</b> radio button in the footer of the right button bar
    turns on and off the
    <a href="../../appendix/settings.htm#StereoMenu">default stereo mode</a>.
    Right-click to set the stereo mode and access the
    <a href="../../appendix/viewsetup.htm#StereoSettings">Stereo Settings</a>.
    panel.
    </dd>
  </dl>
  <dl>
    <dt id="Themes"><b>Themes</b> <img src="mainwin/themes_icon.png" style=
    "vertical-align:middle" /></dt>
    <dd>

<p>The <b>Themes</b> menu in the footer of the right button bar provides
      quick access to a variety of built-in themes and to the <b>Copy
      Picture</b> (to clipboard) and <b>Save Picture</b> commands. Custom
      <a href="../../appendix/viewsetup.htm#Themes">Themes</a> will
      automatically be added to the menu. The
      <a href="../../appendix/viewsetup.htm">Visualization Setup</a> panel can also be
      opened from here.</p>
    </dd>
  </dl>
<a id="RSL"></a>
<dl>
    <dt><b>RSL&#8680;</b></dt>
    <dd>

<p>The <b>R</b>, <b>S</b>, <b>L</b>, and <b>&#8680;</b> buttons provide
      atom selection masking in the MOE Window, affecting mouse picking of
      atoms. Successive button presses toggle between enabling and disabling
      the ability to alter the selection state of the respective atoms
      (<b>R</b> &ndash; receptor, <b>S</b> &ndash; solvent, <b>L</b> &ndash;
      ligand, <b>&#8680;</b> &ndash; ribbon) using the mouse. For example, in
      the following setting,</p>
      <img src="mainwin/RSL_lig.png" alt="RSL Buttons" class="center" />

<p>only ligand atoms are susceptible to selection/deselection by mouse
      picking. A red background indicates that mouse picking has been disabled.
      Note that these buttons have no effect on other means of atom selection,
      e.g. via the <span class="menu">MOE | RHS | Select</span> menu.</p><!--

<p>
Double-clicking on one of the <b>R</b>, <b>S</b>, or <b>L</b> buttons
    is a shortcut to
    enable picking of the selected class of atoms and disable picking of
    the other two classes of atoms.  For example, double-clicking on
    <b>L</b> would result in the RSL setting of the figure above.

</p>
-->

<p>The <b>R</b>, <b>S</b>, <b>L</b>, and <b>&#8680;</b> buttons provide
      additional feedback as to the selection state of atoms of the
      corresponding class:</p>
      <table class="titlebar">
          <tr>
            <th>RSL&#8680;</th>
            <th>Select&nbsp;Atoms&nbsp;in&nbsp;RSL&#8680;&nbsp;Class</th>
            <th>Description</th>
          </tr>
          <tr>
            <td valign="top"><img src="mainwin/rsl_normal.png" style=
            "vertical-align:middle; margin: 0 5px 5px 0;" /></td>
            <td valign="top">Enabled</td>
            <td valign="top">No atoms of the corresponding class are
            selected.</td>
          </tr>
          <tr>
            <td valign="top"><img src="mainwin/rsl_pinkfg.png" style=
            "vertical-align:middle; margin: 0 5px 5px 0;" /></td>
            <td valign="top">Enabled</td>
            <td valign="top">Some atoms of the corresponding class are
            currently selected.</td>
          </tr>
          <tr>
            <td valign="top"><img src="mainwin/rsl_redbg.png" style=
            "vertical-align:middle; margin: 0 5px 5px 0;" /></td>
            <td valign="top">Disabled</td>
            <td valign="top">No atoms of the corresponding class are currently
            selected.<br />
            <br />
            <i>Action:</i> Press the appropriate RSL&#8680; button.</td>
          </tr>
          <tr>
            <td valign="top"><img src="mainwin/rsl_pinkbg.png" style=
            "vertical-align:middle; margin: 0 5px 5px 0;" /></td>
            <td valign="top">Disabled</td>
            <td valign="top">Atoms of the corresponding class are currently
            selected.<br />
            <br />
            <i>Action:</i> Hold <tt>Shift</tt> while pressing the appropriate
            button; selected atoms of this class remain selected.</td>
          </tr>
        </table>

<p>Note that double-clicking no longer has any effect. The former
      double-click behavior, to enable selection of the selected class and
      disable selection of the others, is achieved by holding down the
      <tt>Ctrl</tt> key while pressing the appropriate button.
      <tt>Ctrl+Shift</tt> additionally preserves the current atom
      selection.</p>
    </dd>
    <dd>
<a id="Trackball"></a>
</dd>
    <dt><b>Trackball</b></dt>
    <dd>

<p>The trackball changes the view onto the molecular system. Dragging
      left over the trackball results in rotational motion. At lower left is an
      icon indicating the current mode of the trackball: translation or
      rotation. Pressing this icon toggles between modes.</p>

<p>Rotation can be restricted to one of the X or Y axes by dragging close
      to the edge of the trackball widget. Dragging from the corner of the
      widget restricts rotation to the Z axis. Dragging anywhere else in the
      widget produces unrestricted rotational motion. Similarly, rotation can
      be restricted to the X-Y plane, or to the Z-direction (zoom). All these
      special modes are indicated by additional icons within the trackball
      area.</p>

<p>If there are selected atoms, then holding down the <tt>Alt</tt> key
      changes the positions of only those atoms, rotating and translating them
      just as if <a href="#Editing">key modifiers</a> had been used.</p>

<p>There is a popup menu on the trackball widget. Pressing the right
      mouse button brings up the <a href="#Viewing"><b>View</b></a> menu.</p>
    </dd>
    <dd>
<a id="Dial"></a>
</dd>
    <dt><b>Dial</b></dt>
    <dd>

<p>The dial beneath the trackball is used for visual zoom (translation of
      the view in Z). Holding down the <tt>Ctrl</tt> while dragging the mouse
      gives finer control (slower zoom). Zooming the view can also be achieved
      by holding down the <tt>Ctrl</tt> key while dragging over the trackball.
      The zoom operation affects only the view &mdash; holding down the
      <tt>Alt</tt> key has no effect.</p>
    </dd>

    <dd>
<a id="2D"></a>
</dd>
    <dt><b>2D Overlay</b></dt>
    <dd>
      The <a href="#2dLigandOverlay">2D Ligand Overlay</a> provides a
      persistent two-dimensional view of the ligand in the MOE window.
    </dd>

    <dd>
<a id="ZClip"></a>
</dd>
    <dt><b>ZClip</b></dt>
    <dd>
      The <b>ZClip</b> settings control the position of the front
      and back clipping planes. All atoms and objects closer than the
      front and farther than the back clipping planes are hidden from view.
      Additional settings control the positions of the surface-only clipping
      plane, which only clips graphics objects such as molecular surfaces.
      More details are given in the section on
      <a href="#Clipping">Clipping Planes and Depth Cueing</a>.
    </dd>

    <dd>
<a id="Fog"></a>
</dd>
    <dt><b>Fog</b></dt>
    <dd>
      The <b>Fog</b> settings control where fading of the image begins
      and the strength of the fading beyond that point.  The purpose of
      fogging is to give depth cues: objects farther away will appear dimmer.
      More details are given in the section on
      <a href="#Clipping">Clipping Planes and Depth Cueing</a>.
    </dd>

    <dt><b>Contacts</b></dt>
    <dd>

<p>Render nonbonded contacts:
      <a href="#Contacts">hydrogen bond and van der Waals contacts</a>.</p>
    </dd>
    <dt><br />
    <b>Atoms</b><br />
    <b>Ribbon</b><br />
    <br /></dt>
    <dd>

<p>The <b>Atoms</b> and <b>Ribbon</b> buttons are shortcuts for
      <a href="#RenderingAtoms"><span class="nobr"><b>MOE | Render |
      Atoms</b></span></a> and <a href="#RenderingBackbone"><span class=
      "nobr"><b>MOE | Render | Ribbon</b></span></a>.</p>
    </dd>
    <dd>
<a id="Forcefield"></a>
<a id="CrysParm"></a>
</dd>
    <dt><br />
    <b>Forcefield</b><br />
    <b>Crystal Parameters</b><br />
    <br /></dt>
    <dd>

<p>On the left side of the footer, the button labeled with the current
      forcefield opens the <a href="../pot.htm#Forcefield">Potential Setup</a>
      panel when pressed. The button labeled with the space group of the
      current system or, if periodic boundary conditions are not currently
      active, <tt>PBC</tt>, opens the <a href="../../apps/crystal.htm">Crystal
      Parameters</a> panel when pressed.</p>
    </dd>
    <dd>
<a id="FooterEditing"></a>
</dd>
    <dt><br />
    <b>Footer Editing Area</b><br />
    <br /></dt>
    <dd>

<p>The footer editing area provides information about the current system
      and is also sensitive to current atom selection, providing controls
      for:</p>
      <ul>
        <li>Editing atom name.</li>
        <li>Measuring/editing interatomic distances.</li>
        <li>Measuring/editing atom angles.</li>
        <li>Measuring/editing atom dihedral angles.</li>
      </ul>

<p>depending on the number of atoms selected.</p>

<p><b>One atom selected.</b> When a single atom is selected, the footer
      provides a text field for editing the atom name.</p>
      <img src="mainwin/footer_1sel.png" alt="Footer: 1 Atom Selected" class="center" />
<a id="DihedralProfile"></a>
<p><b>Two atoms selected.</b> When two atoms are selected, the footer
      provides a text field that reports the distance between them. If the atoms
      are bonded and free to move (not in a ring, not fixed), the interatomic
      distance can be modified by entering a new number (in angstroms) in the
      text field and pressing <b>Apply</b>. The side that moves is specified
      using the star button, which toggles between the two atoms. In the MOE
      Window, the respective atom will be rendered with a starry halo. If the
      bond is rotatable and atoms are free to move, one side of the bond can be
      rotated with respect to the other using the <b>Rotate</b> wheel. The side
      that moves is also specified by the star button. The <b>Profile</b>
      button pops open a plot of energy (kcal) vs. dihedral angle (degrees).
      The current value is indicated by the green line and the original value
      (if changed) is indicated by a yellow dotted line. If a protein is
      present, the energy profile of the isolated ligand is drawn with a dotted
      line. A green/orange/red bar at the top shows the dihedral quality of the
      bond as determined by the <a href="../../apps/ligx.htm#Torsions">
      Ligand Torsions</a> application.
      If <a href="../../apps/mogul.htm">Mogul</a> is available, the torsion
      quality is shown as a blue histogram instead. Clicking in the plot sets
      the dihedral angle accordingly.</p>
      <img src="mainwin/footer_distance.png" alt=
        "Footer: 2 Bonded Atoms Selected" class="center" />
<a id="DihedralContour"></a>

The behavior of the Mogul Histogram can be altered by right-clicking
on the Dihedral Profile.  This brings up a configuration panel similar
to the <a href="../../apps/mogul.htm#Settings">Mogul Analysis Settings</a>
panel:

<img src="mainwin/histo_config.png" class="center" alt=
    "Footer: Mogul Histogram Config" />

<table class="noborder">
<tr>
<td><b>Mogul Histogram</b></td>
<td>This specifies how the Mogul Histogram will be constructed.
<span class="nobr"><b>Multiple Torsions</b></span> will use all of the torsion
angles that contain the selected bond, shifting the individual histograms
as appropriate.
<span class="nobr"><b>Single Torsion</b></span> will show the histogram for a
single torsion.  If only two atoms are selected, the additional angles of
the torsion will be highlighted in MOE.
<span class="nobr"><b>Auto</b></span> will generate a multi-torsion histogram
if two atoms are selected and a single-torsion histogram if four atoms
are selected.</td>
</tr>
<tr>
<td><b>Exclude Solvent</b></td>
<td>If selected, Mogul will ignore any fragments that come from known
solvents.</td>
</tr>
<tr>
<td><b>Exclude Organometallics</b></td>
<td>If selected, Mogul will ignore any fragments that contain a transition
metal or similar heavy atoms.</td>
</tr>
<tr>
<td><b>Relevance</b></td>
<td>Specify the minimum fragment relevance used when generating histograms.
</tr>
<td><b>Ligand Torsions</b></td>
<td>This option controls the colored bar at the top of the
plot representing the <a href="../../apps/ligx.htm#Torsions">Ligand Torsions</a>
quality of the bond. <b>Auto</b> shows the bar if Mogul is unavailable.
<b>On</b> and <b>Off</b> show and hide the bar regardless of Mogul.
</tr>

</table>

<p>Note that <span class="menu">MOE | Compute | Conformations | Dihedral
      Contour Plot</span> performs a similar calculation, except that the
      potential energy is calculated as a function of two dihedral angles, thus
      yielding an energy surface that is plotted in 2D as a contour plot.</p>

<p><b>Three atoms selected.</b> When three chained atoms are selected,
      the footer provides a text field that reports the angle between them. If
      the atoms are free to move, the text field is active, and the angle can be
      modified by entering a new number (in degrees) and pressing <b>Apply</b>.
      The atom that moves is chosen using the star button, which toggles
      between the two sides. In the MOE Window, the respective atom will be
      rendered with a starry halo.</p>
      <img src="mainwin/footer_angle.png" alt=
        "Footer: 3 Bonded Atoms Selected" class="center" />

<p><b>Four atoms selected.</b> When four chained atoms are selected, the
      footer provides a text field that reports the dihedral angle between them.
      If atoms are free to move, the text field is active, and the angle can be
      modified by entering a new number (in degrees) and pressing <b>Apply</b>
      or by using the wheel. The side that moves is chosen using the star
      button, which toggles between the two sides. In the MOE Window, the
      respective atom will be rendered with a starry halo. Similarly to when
      two bonded atoms are selected, the <b>Profile</b> button pops open a plot
      of energy (kcal) vs. dihedral angle (degrees). The current value is
      indicated by the green line and the original value (if changed) is
      indicated by the yellow dotted line. If a protein is present, the energy
      profile of the isolated ligand is drawn with a dotted line. A
      green/orange/red bar at the top shows the dihedral quality of the
      bond as determined by the <a href="../../apps/ligx.htm#Torsions">
      Ligand Torsions</a> application.
      If <a href="../../apps/mogul.htm">Mogul</a> is available, the torsion
      quality is shown as a blue histogram instead. Clicking in the plot sets
      the dihedral angle accordingly.</p>

<p>If parts of the molecule are not free to move, because of fixed atoms,
      this will be indicated by a red sign over the <b>F</b> button. Pressing
      the <b>F</b> will switch the footer editing controls in override mode and
      move the atoms despite their fixed atom status. This will be indicated by
      a green ring around the <b>F</b>. This override mode only affects the
      footer controls, all other tools treat the fixed atoms as such.</p>
      <div class="floatright"><img src="mainwin/footer_singlesel.png" alt=
      "Footer: Multiple Atoms Selected" class="floatright" /></div>
      <div class="floatright"><img src="mainwin/footer_manysel.png" alt=
      "Footer: Multiple Atoms Selected" class="floatright" /></div>

<p>In all other cases, when atoms are selected, the reported information
      may include the number of selected atoms, whether they belong to the
      ligand or receptor, and the total number of atoms in the system.</p>

<p>Additional information is provided in the footer bubble help.
  The bubble help over the footer of a single atom selection will show
  pertinent atom information, such as the name, element, residue, chirality,
  B-factor, force field type, etc. If multiple atoms are selected, the
  bubble help contains a summary of the selection: how many atoms and
  residues are selected; how many atoms, residues, chains are present, etc.
</p>
    </dd>
  </dl>

<h2>
<a id="MOEPopup">MOE Window Popup Menu</a></h2>

<p>The MOE Window popup menu is opened by clicking the right mouse button in
  the <a href="#3DRenderingArea">3D rendering area</a>. It contains the
  commands as found in the <a href="#Rendering">Rendering menu</a>
  <nobr>(<b>MOE | Render</b>).</nobr> Rendering options will be applied to
  selected atoms only or to all atoms if none are selected. The popup
  menu also provides access to the <a href="#UsingMenu">Select menu</a>
  as well as the <a href="system.htm">System Manager</p></p>

<a id="Selecting_Molecular_Objects"></a>
<h1>
<a id="Selecting">Selecting Molecular Objects</a></h1>

<img src="mainwin/selatom.png" alt="Selected Atom in MOE" class="floatright" />

<p>In many instances, operations are performed on only part of the molecular
  system. MOE lets you select the specific molecular objects that you wish to
  manipulate in the rendering area. Selected atoms are identified by a change
  in color.</p>

<p>There are three distinct selection sets in MOE: an atom selection set, a
  residue selection set, and a chain selection set. Atom selection can be
  accomplished in the MOE Window, <a href="atommgr.htm">Atom Manager</a>, or
  <a href="se.htm">Sequence Editor</a>. Residue and chain selection is
  primarily performed in the Sequence Editor.</p>

<p>There are several basic methods used for selecting atoms. You can select
  atoms directly in the 3D rendering area using the mouse, you can select atoms
  using the <b>Select</b> menu in the menu bar, you can use the
  <a href="sel_ui.htm">Selection Prompter</a>, or you can type your own SVL commands at
  the command line to select atoms based on some criteria.</p>

<p>The following sections explain each of these atom selection methods.</p>

<a id="Selecting_Atoms_Using_the_Mouse"></a>
<h2>
<a id="UsingMouse">Selecting Atoms Using the
  Mouse</a></h2><!-- WARNING: This table is repeated in mouse.htm -->

<p>This section lists the default MOE mouse selection actions on atoms. Note
  that, since mouse actions can be customized (using the
  <a href="mouse.htm#MouseConfiguration">Mouse Configuration</a> panel), these may not
  correspond to what is currently in any given particular instance of MOE.</p>

<p><b>Note:</b> Mouse selections are subject to the state of the
  <a href="#RSL"><b>R</b>, <b>S</b>, <b>L</b>, and <b>&#8680;</b></a> buttons.</p>

<table class="titlebar">
  <tr>
	<th width="50%">Operation</th>
	<th>Left Mouse Button</th>
  </tr>
  <tr>
	<td>Select an atom, deselecting all other atoms</td>
	<td>Click on the atom</td>
  </tr>
  <tr>
	<td>Select a bond (select two atoms bonded together)</td>
	<td>Click on the bond joining the atoms</td>
  </tr>
  <tr>
	<td>Select all atoms of the same class (ligand, receptor, solvent) in
	the same connected component as the atom</td>
	<td>Double-click on the atom</td>
  </tr>
  <tr>
	<td>Toggle selection state of an atom</td>
	<td><tt>Shift</tt>-click on the atom</td>
  </tr>
  <tr>
	<td>Select all atoms of parent residue</td>
	<td><tt>Ctrl</tt>-click on the atom</td>
  </tr>
  <tr>
	<td>Toggle selection state of parent residue atoms</td>
	<td><tt>Shift+Ctrl</tt>-click on the atom</td>
  </tr>
  <tr>
	<td>Select a group of atoms, deselecting all other atoms</td>
	<td>Drag over atoms</td>
  </tr>
  <tr>
	<td>Add more atoms to a group of selected atoms</td>
	<td><tt>Shift</tt>-drag over atoms</td>
  </tr>
  <tr>
	<td>Select a group of atoms AND atoms of parent residues</td>
	<td><tt>Ctrl</tt>-drag over atoms</td>
  </tr>
  <tr>
	<td>Toggle selection state of parent residue atoms</td>
	<td><tt>Shift+Ctrl</tt>-drag over atoms</td>
  </tr>
  <tr>
	<td>Select atoms near an atom, deselecting all other atoms
	</td>
	<td>
	    <img src="mainwin/sphere_selection.png" style=
        	"vertical-align:top; float:right" />
	    Click on the atom, then click again and drag the selection sphere
	    with the center of the first atom to select all atoms inside
	    the sphere.
	</td>
  </tr>
  <tr>
	<td>Toggle selection state of atoms near an atom</td>
	<td>
	    Click on a selected atom and <tt>Shift</tt>-drag the
	    selection sphere. If all atoms inside the sphere are
	    selected then they will be deselected, otherwise the
	    unselected atoms in the sphere will be selected as well.
	</td>
  </tr>
  <tr>
	<td>Select atoms AND atoms of parent residues near an atom</td>
	<td>
	    Click on the atom, then click again and <tt>Ctrl</tt>-drag
	    the selection sphere.
	</td>
  </tr>
  <tr>
	<td>Toggle selection state of parent residue atoms near an atom</td>
	<td>
	   Click on a selected atom and <tt>Shift+Ctrl</tt>-drag
	   the selection sphere.
	</td>
  </tr>
  <tr>
	<td>Select atoms near multiple selected atoms
	</td>
	<td>
	   Click on one of the selected atom and <tt>Alt+Shift</tt>-drag
	   selection spheres over all selected atoms.
	</td>
  </tr>
  <tr>
	<td>Clear selection of atoms</td>
	<td>Click away from any atoms</td>
  </tr>
</table>

<a id="Selecting_Ribbons_Using_the_Mouse"></a>
<h2>
<a id="SelectRibbon">Selecting Ribbons Using the
  Mouse</a></h2><!-- WARNING: This table is repeated in mouse.htm -->

<p>This section lists the default MOE mouse selection actions on ribbons.
  When the ribbon segment of a residue is selected or unselected with the
  mouse, all the atoms of that residue are selected/unselected as well. Note
  that, since mouse actions can be customized (using the
  <a href="mouse.htm#MouseConfiguration">Mouse Configuration</a> panel), these may not
  correspond to what is currently in any given particular instance of MOE.</p>

<p><b>Note:</b> Mouse selections are subject to the state of the
  <a href="#RSL"><b>R</b>, <b>S</b>, <b>L</b>, and <b>&#8680;</b></a> buttons. With the
  <b>&#8680;</b> button one can enable/disable the selectability of ribbons in
  general.</p>
  <center>
    <table class="titlebar">
      <tr>
        <th>Operation</th>
        <th>Left Mouse Button</th>
      </tr>
      <tr>
        <td width="50%">Select ribbon segment of a residue, deselecting all
        other ribbons</td>
        <td>Click on the ribbon segment</td>
      </tr>
      <tr>
        <td>Select all ribbons in the same connected component as the picked
        ribbon</td>
        <td>Double-click on the ribbon</td>
      </tr>
      <tr>
        <td>Toggle selection state of a ribbon</td>
        <td><tt>Shift</tt>-click on the ribbon</td>
      </tr>
      <tr>
        <td>Select a group of ribbons, deselecting all other ribbons/atoms</td>
        <td>Drag over ribbons</td>
      </tr>
      <tr>
        <td>Add more ribbons to a group of selected ribbons</td>
        <td><tt>Shift</tt>-drag over ribbons</td>
      </tr>
      <tr>
        <td>Clear selection of ribbons</td>
        <td>Click away from any ribbons</td>
      </tr>
    </table>
  </center>

<a id="Selecting_Atoms_Using_the_Select_Menu"></a>
<h2>
<a id="UsingMenu">Selecting Atoms Using the Select
  Menu</a></h2>

<p>For quick access to a variety of selection commands, you can use the
  <b>Select</b> menu which provides basic selections such as selecting all
  atoms of a specified element type. All selections performed from this menu
  are added to the current atom selection set. The following options are
  available from the menu:</p>
  <center>
    <table class="titlebar">
      <tr>
        <th>Command</th>
        <th>Description</th>
      </tr>
      <tr>
        <td>
          <a id="Invert"><b>Invert</b></a>
        </td>
        <td>Toggles the atom selection state for each atom.</td>
      </tr>
      <tr>
        <td>
          <a id="Clear"><b>Clear</b></a>
        </td>
        <td>Deselects all atoms.</td>
      </tr>
      <tr>
        <td>
          <a id="All"><b>All</b></a>
        </td>
        <td>Selects all atoms.</td>
      </tr>
      <tr>
        <td>
          <a id="Extend"><b>Extend</b></a>
        </td>
        <td><b>Bonded To</b>. Extends selection to all atoms that are bonded to
        any currently selected atom.<br />
        <b>H and LP</b>. Extends selection to all Hydrogen and Lone Pair atoms
        that are bonded to any currently selected atom.<br />
        <b>Molecule</b>. Extends selection to all atoms that belong to the same
        connected component as any currently selected atom.<br />
        <b>Near (4.5A)</b>. Extends selection to all atoms within a 4.5
        angstroms radius of any currently selected atom.<br />
        <b>Near Residues</b>. Extends selection to all atoms of residues
        containing at least one atom within 4.5 angstroms of any currently
        selected atom.<br />
        <b>Residue</b>. Extends selection to all atoms that belong to the same
        residue as any currently selected atom.<br />
        <b>Chain</b>. Extends selection to all atoms that belong to the same
        chain as any currently selected atom.<br />
        <b>Domain</b>. Extends selection to all atoms that belong to the same
        protein domain as any currently selected atom.<br />
        <b>Chain Tag</b>. Extends selection to all atoms belonging to a chain
        which has the same chain tag as any currently selected atom.<br /></td>
      </tr>
      <tr>
        <td>
          <a id="Hydrogens"><b>Hydrogens</b></a>
        </td>
        <td>Selects all hydrogen atoms.</td>
      </tr>
      <tr>
        <td>
          <a id="Ligand"><b>Ligand</b></a>
        </td>
        <td>Ligand atoms are defined as all non-solvent molecules with &lt;= 10
        residues. The calculation is done strictly within each chain tag. Atoms
        in the set &quot;Cofactor&quot; are then subtracted. Atoms in the set
        &quot;Ligand&quot; are then added. H and LP are added to the sets at
        the end. <b>Note:</b> The selection state of the ligand atoms of
        individual molecular systems can be controlled using the system list
        (<span class="menu">MOE | RHS |
        <a href="system.htm">System</a></span>).</td>
      </tr>
      <tr>
        <td>
          <a id="Pocket"><b>Pocket</b></a>
        </td>
        <td>Pocket atoms are defined as all atoms of receptor residues
        containing at least one atom within 4.5 angstroms of the ligand atoms.
        The calculation is done strictly within each chain tag. Atoms in the
        set &quot;Pocket&quot; are then added. H and LP are added to the sets
        at the end.</td>
      </tr>
      <tr>
        <td>
          <a id="Receptor"><b>Receptor</b></a>
        </td>
        <td>Receptor atoms are defined as all non-solvent, non-ligand atoms.
        The calculation is done strictly with each chain tag. H and LP are
        added to the sets at the end. <b>Note:</b> The selection state of the
        receptor atoms of individual molecular systems can be controlled using
        the system list (<span class="menu">MOE | RHS |
        <a href="system.htm">System</a></span>).</td>
      </tr>
      <tr>
        <td>
          <a id="Interface"><b>Interface</b></a>
        </td>
        <td>Select atoms of receptor chain interface and ligand/solvent
        atoms which are near these residues. The interface residues are
        receptor residues which are near (within 4.5 angstroms)
        other receptor residues within the same tag but belong to
        a different chain.</td>
      </tr>
      <tr>
        <td>
          <a id="Solvent"><b>Solvent</b></a>
        </td>
        <td>Solvent atoms are defined as all molecules with no torsions among
        the heavy atoms. Metals are solvent only if they have no non-solvent
        contacts (within 4.5 angstroms). The calculation is done strictly with
        each chain tag. Atoms in the set &quot;Solvent&quot; are then added.
        Atoms in the sets &quot;Receptor&quot; and &quot;Ligand&quot; are then
        subtracted. H and LP are added to the sets at the end. <b>Note:</b> The
        selection state of the receptor atoms of individual molecular systems
        can be controlled using the system list (<span class="menu">MOE | RHS |
        <a href="system.htm">System</a></span>).</td>
      </tr>
      <tr>
        <td>
          <a id="Dummy"><b>Dummy</b></a>
        </td>
        <td>
          Selects dummy atoms. A Dummy atom is defined to be a lone pair (LP)
          atom with no bonds. Dummy atoms may be created through the
          <a href="../molsystems/build.htm">Builder</a> or
          <a href="../../apps/sitefind.htm">Site Finder</a> panels.
        </td>
      </tr>
      <tr>
        <td>
          <a id="Set"><b>Set</b></a>
        </td>
        <td>Select atoms of a specified named set.</td>
      </tr>
      <tr>
        <td>
          <a id="SetCreate"><b>Set Create</b></a>
        </td>
        <td>Create a new named set from the selected atoms, or add them to an
        existing set.</td>
      </tr>
      <tr>
        <td>
          <a id="SetDelete"><b>Set Delete</b></a>
        </td>
        <td>Delete specified named set definition.</td>
      </tr>
      <tr>
        <td>
          <a id="Selector"><b>Selector</b></a>
        </td>
        <td>
          Open the <a href="#SelectionPrompter">Selection Prompter</a>.
        </td>
      </tr><!--
        <tr>
            <td><a id="Attachment"><b>Attachment</b></a></td>

            <td>Select atoms names as attachment points for
            combinatorial library generation. Select by a specific attachment
            number, <b>A0</b>,...,<b>A9</b>, or <b>Any</b> attachment
            number.</td>
        </tr>

        <tr>
            <td><a id="Element"><b>Element</b></a></td>

            <td>
                Selects all atoms of the specified element type.
                Note that
                a Dummy atom is defined to be a lone pair (LP) atom with no
                bonds. Dummy atoms may be created through the
                <a href="../molsystems/build.htm">Builder</a> or
                <a href="../../apps/sitefind.htm">Site Finder</a>
                panels.
            </td>
        </tr>
-->
      <tr>
        <td>
          <a id="SelectGeo"><b>Stereo</b></a>
        </td>
        <td>Selects all atoms of the specified stereo geometry. Note that the
        stereochemistry is defined by the current atom coordinates and not by
        any chiral or stereo constraints. (To select stereochemically
        constrained atoms, use <span class="menu">MOE | Select |
        Potential</span>.)</td>
      </tr>
      <tr>
        <td>
          <a id="Potential"><b>Potential</b></a>
        </td>
        <td>
          Select atoms with the specified property:
          <ul>
            <li><b>Fixed</b>. Atoms anchored to their current positions.</li>
            <li><b>Inert</b>. Atoms marked as non-interacting.</li>
            <li><b>Forced Stereo</b>. Atoms assigned the corresponding chiral or
            stereo constraints, independent of atom coordinates. (To select
            atoms based on atom coordinates, use <span class="menu">MOE |
            Select | Stereo</span>.)</li>
            <li>
              <b>Tethered</b>. Atoms
              <a href="../pot.htm#Restraints">tethered</a> to their current position to
              prevent gross movement.
            </li>
            <li>
              <b>Restrained</b>. Atoms restrained by distance, angle, and/or
              dihedral angle <a href="../pot.htm#Restraints">restraints</a>.
            </li>
            <li>
              <b>Walled</b>. Atoms subject to <a href="../pot.htm#Wall">wall
              restraints</a>.
            </li>
            <li><b>Untyped</b>. Atoms with missing parameters under the current
            forcefield.</li>
          </ul>
        </td>
      </tr>
      <tr>
        <td>
          <a id="Protein"><b>Protein</b></a>
        </td>
        <td>
          Select protein <b>Alpha Carbons</b> and <b>Beta Carbons</b>,
          <b>Backbone Atoms</b> (atoms with backbone bit set), and <b>Sidechain
          Atoms</b> (group attached to alpha-carbon) atoms, as well as
          <b>Buried</b> and <b>Exposed</b> atoms. Also allows selection of
          potential Post-Translational Modification sites (<b>PTM Sites</b>).
          These are atoms on amino acid sidechains where PTMs may be attached,
          and will only be selected if the chosen modification can be
          chemically attached there and if there is enough space for the
          additional atoms. A simple van der Waals potential is used to
          determine the latter, and solvent atoms are ignored. Modifications
          permitted are:
          <ul>
            <li>fucose</li>
            <li>glucose</li>
            <li>galactose</li>
            <li>mannose</li>
            <li>n-acetylglucosamine</li>
            <li>n-acetylgalactosamine</li>
            <li>xylose</li>
            <li>acetylation</li>
            <li>methylation</li>
            <li>nitrosylation</li>
            <li>phosphorylation</li>
            <li>sulfation</li>
          </ul>
        </td>
      </tr><!--
        <tr>
            <td>
            <a id="Protein"><b>PTM Sites</b></a></td>

            <td>
            Select potential Post-Translational Modification sites.
            These are atoms on amino acid sidechains where PTMs may be attached,
            and will only be selected if the chosen modification can be
            chemically attached there and if there is enough space
            for the additional atoms.  A simple van der Waals potential is
            used to determine the latter, and solvent atoms are ignored.
            Modifications permitted are:
            <ul>
            <li>Fucose</li>
            <li>Glucose</li>
            <li>Galactose</li>
            <li>Mannose</li>
            <li>N-acetylglucosamine</li>
            <li>N-acetylgalactosamine</li>
            <li>Xylose</li>
            <li>Acetylation</li>
            <li>Methylation</li>
            <li>Nitrosylation</li>
            <li>Phosphorylation</li>
            <li>Sulfation</li>
            </ul>
            </td>
        </tr>
-->
      <tr>
        <td>
          <a id="Save"><b>Save</b></a>
        </td>
        <td>Saves the current atom selection set. You can save up to 16
        selection sets at any one time. Simply select one or more atoms and
        choose the Save command. To view the saved selection, choose Restore
        (described next).</td>
      </tr>
      <tr>
        <td>
          <a id="Restore"><b>Restore</b></a>
        </td>
        <td>Restores the last saved selection set. Selections are saved on a
        last-in, first-out basis. The last saved selection is the first to be
        restored. The selection is then removed from the stack and cannot be
        restored again, unless first saved.</td>
      </tr>
      <tr>
        <td>
          <a id="Synchro"><b>Synchronize</b></a>
        </td>
        <td>
          Synchronizes the selection of atoms in the MOE rendering area with
          the selection of residues in the Sequence Editor.

<p>Selecting an atom causes its parent residue to be selected (seen
          in the Sequence Editor), and selecting a residue causes all its
          children to become selected (seen in the MOE Window). The exception
          is when a residue becomes selected because the monitor has detected
          that one of the residue's child atoms has been selected: the rest
          of the residue's children do not immediately become selected. A
          residue remains selected as long as any of its child atoms are
          selected. Deselecting a residue deselects its atoms. Deselecting all
          atoms of a residue deselects the residue.</p>
        </td>
      </tr>
    </table>
  </center>

<p><font color="#FF0000"><b>Tip</b></font>&nbsp;&nbsp; If you accidentally
  click in the 3D rendering area and lose your current selection, you can use
  the <span class="nobr"><b>MOE | Select | Restore</b></span> command to
  restore the last selection set. <a id="Selecting_Atoms_Using_the_Selection_Prompter"></a>
</p>

<h2>
<a id="SelectionPrompter">Selecting Atoms Using
  the Selection Prompter</a></h2>

<p>For more complex selection criteria, MOE provides a
  <a href="sel_ui.htm">Selection Prompter</a>, which is opened with <span class=
  "nobr"><b>MOE | Select | Selector</b></span>. When active, the Selection
  Prompter appears in the MOE Window immediately below the menu bar. The
  Selection Prompter allows for specifying atom selections using MOE's
  <a href="../fcnref/aselect.htm">selection language</a>. This language has
  built-in keywords representing common sets, e.g. <tt>asel</tt> for the
  current atom selection, and <tt>ligand</tt> for all ligand atoms, as well as
  logical, comparison, property, substructure, and extension operations.</p>

<p><span class="tip">Note:</span> Beginning in 2013, the <a id="AtomSelector"></a>
<a href="select.htm">Atom Selector</a>
  panel is invoked through the <b>UI</b> button in the Selection Prompter.</p>
<a id="Rotating_Translating_and_Zooming_the_View"></a>
<h1>
<a id="Viewing">Rotating, Translating and Zooming the
  View</a></h1>

<p>You can use the mouse, menu <b>View</b> commands, and the trackball and
  zoom dial in the <a href="#Footer">Footer</a> to rotate, translate or zoom
  the molecular system in real time in the MOE Window. The SVL <a class="svl"
  href="../fcnref/view.htm">View</a> function can also be used. Note that
  changing the view of the molecular system does not alter the positions of the
  atoms. To <a href="#EditingAtomsUsingTheMouse">change atom coordinates</a>,
  select the atoms to move and drag mouse middle (in conjunction with
  <tt>Shift</tt> for translation) while holding down the <tt>Alt</tt> key.</p>

<p>The most common ways of changing the view are:</p>
  <ul>
    <li>Dragging the mouse while holding down the middle mouse button. This
    produces in rotational motion. When combined with <tt>Shift</tt>,
    translation results. When combined with <tt>Ctrl</tt>, zooming results.
    More details can be found in the chapter on <a href="mouse.htm">Using the
    Mouse</a>.
    </li>
    <li>Using the <a href="#Footer"><span class="nobr"><b>MOE | Footer |
    Trackball</b></span></a> and dial.
    </li>
    <li>Using <span class="nobr"><b>MOE | Render | Center</b></span> or
    <span class="nobr"><b>MOE | RHS | Center</b></span>. These commands operate
    on selected atoms only and adjust the view so that the atoms are centered
    in the rendering area. If no atoms are selected then the commands will
    apply to all visible atoms, or, if no atoms are visible then to all
    atoms.</li>
    <li>Using the <span class="nobr"><b>MOE | Footer | Trackball Popup |
    View</b></span> menu. The <b>Reset</b> command returns the view to the
    default setting (view look-at point <tt>[0,0,0]</tt>, view orientation
    vectors <tt>[0,1,0]</tt>, and <tt>[0,0,1]</tt>).

<p>The <b>Save</b> and <b>Load</b> commands allow you to save and load up
      to 8 view settings. Views saved into the viewing registers are preserved
      only for the duration of the current session. A view saved in one session
      cannot be retrieved in a subsequent session of MOE.</p>

<p>Views can also be saved as part of molecule files; see
      <a href="../../apps/filesave.htm">Saving the System</a>.</p>
    </li>
  </ul>

<p>The view may also be modified through other panels or SVL functions. For
  instance, the Sequence Editor will perform a View operation after isolating
  the atoms of a specified chain. The view is specified by its look-at point as
  well as its y- and z-direction vectors. (For more information on the view
  look-at point, orientation and scale, see the <a class="svl" href=
  "../fcnref/view.htm">View</a> functions.)</p>

<p><b>Note:</b> It is also possible to <a href="#RotatingAboutABond">rotate
  atoms about a bond</a>.</p>

<a id="Changing_the_Center_of_Rotation"></a>
<h1>
<a id="CenterRotation">Changing the Center of
  Rotation</a></h1>

<p>When the view is rotated, the atoms in the 3D rendering area appear to
  rotate. The center of rotation is, by default, the center of mass of the
  molecular system. To display x, y and z axes, select <b>Coordinate Axes</b>
  from <span class="menu">MOE | Settings | Draw Settings</span>. If the center
  of rotation is changed, the axes will be moved to the new center
  position.</p>

<p>It is possible to rotate the view with respect to a particular atom, i.e.
  the center of rotation can be made to coincide with any atom.</p>

<p>To change the center of rotation, click the middle mouse button on the
  atom that will be the new center of rotation. Then, rotate the system by
  dragging the middle mouse button.</p>

<p>To reset the center of rotation to the default (center of mass of the
  system), click the middle mouse button away from any atoms.</p>

<a id="Rendering_Molecular_Objects"></a>
<h1>
<a id="Rendering">Rendering Molecular Objects</a></h1>

<p>The <b>Render</b> menu is used to control molecule rendering. Different
  rendering modes are available for displaying molecular objects. Atomic nuclei
  can be rendered as points or spheres, and bonds as lines or cylinders.
  Backbone ribbons for proteins and nucleic acids can be rendered with smooth
  splined surfaces in flat or cylindrical styles, and hydrogen bonds can be
  displayed dynamically as dashed lines.</p>

<p>MOE does not support a &quot;current rendering mode&quot; that is applied
  to all subsequent additions to the molecular system. Rendering options apply
  only to atoms already in the system. The same is true for atom coloring,
  labeling and hiding. One exception is additions to the molecular system that
  are made using the Builder. Groups that are substituted onto a molecule
  inherit the atom nucleus and bond rendering modes of the atom being
  substituted.</p>


  <a id="HideShow"></a>
<a id="Hiding_and_Showing_Atoms"></a>
<h2>Hiding and Showing Atoms</h2>

<p>Many MOE panels offer the ability to work on visible, hidden or all atoms.
  Atoms may be hidden using the <b>MOE | Render | Hide</b> menu which is also
  available as <b>MOE | RHS | Hide</b>. The complementary <b>Show</b> menus are
  located next to the <b>Hide</b> menus. Hiding and showing individual
  molecular subsystems is most easily done using
  <a href="system.htm"><span class="menu">MOE | RHS | System</span></a>. The right
  button bar also contains a <b>Hydrogens</b> button dedicated to the setting
  the visibility of hydrogens. For proteins, the visibility of backbone and
  sidechain atoms can also be controlled using the
  <a href="#RenderingBackbone">Ribbon</a> panel. The <b>Hide</b> and <b>Show</b> menus
  provide the following options:</p>

<table class="titlebar">
  <tr>
	<th>Command</th>
	<th>Hide or Show</th>
  </tr>
  <tr>
	<td><b>All Atoms</b></td>
	<td>All atoms in the molecular system.</td>
  </tr>
  <tr>
	<td><b>Selected</b></td>
	<td>Selected atoms.</td>
  </tr>
  <tr>
	<td><b>Unselected</b></td>
	<td>Unselected atoms.</td>
  </tr>
  <tr>
	<td><b>Hydrogens</b></td>
	<td>Hydrogen atoms.</td>
  </tr>
  <tr>
	<td><b>Non-Polar<br />
	Hydrogens</b></td>
	<td>Hydrogen atoms that are bonded to an element of column IV of the
	periodic table.</td>
  </tr>
  <tr>
	<td><b>Polar<br />
	Hydrogens</b></td>
	<td>Hydrogen atoms that are bonded to an electronegative element (i.e.
	O, N).</td>
  </tr>
  <tr>
	<td><b>Ligand</b></td>
	<td>
	  Atoms belonging to a <a href="#Ligand">ligand</a>.
	</td>
  </tr>
  <tr>
	<td><b>Pocket</b></td>
	<td>
	  Atoms belonging to a <a href="#Pocket">pocket</a>.
	</td>
  </tr>
  <tr>
	<td><b>Receptor</b></td>
	<td>
	  <a href="#Receptor">Receptor</a> atoms.
	</td>
  </tr>
  <tr>
	<td><b>Solvent</b></td>
	<td>
	  <a href="#Solvent">Solvent</a> atoms.
	</td>
  </tr>
  <tr>
	<td><b>Dummy</b></td>
	<td>
	  Dummy atoms. A Dummy atom is defined to be a lone pair (LP) atom with
	  no bonds. Dummy atoms may be created through the
	  <a href="../molsystems/build.htm">Builder</a> or
	  <a href="../../apps/sitefind.htm">Site Finder</a> panels.
	</td>
  </tr>
  <tr>
	<td><b>Set</b></td>
	<td>Atoms of a specified named set.</td>
  </tr>
  <tr>
	<td><b>Backbone</b></td>
	<td>Any atoms with the backbone bit set; e.g. in proteins:
	alpha-carbon, peptide C and N atoms, and carbonyl (C=O) oxygen.</td>
  </tr>
  <tr>
	<td><b>Sidechain</b></td>
	<td>Group attached to the alpha-carbon.</td>
  </tr>
</table>

<p>The <b>Show</b> submenu also contains a <span class="menu">More
  (4.5A)</span> option. This will extend the visible atom set to include atoms
  within 4.5 angstroms of any currently visible atom.</p>
<a id="Rendering_Atoms_and_Bonds"></a>
<h2>
<a id="RenderingAtoms">Rendering Atoms and
  Bonds</a></h2>

<p>The <span class="nobr"><b>MOE | Render | Atoms</b></span> panel, which can
  also be opened using <span class="nobr"><b>MOE | Footer | Atoms</b></span>,
  provides different rendering scheme buttons for drawing atoms and bonds.
  These are applied to selected atoms only or, if no atoms are selected, to all
  atoms.</p>

<p class="center"><img src="mainwin/atom_render_panel.png" alt=
  "Atom Rendering Panel" /></p>

<p>Atom rendering for molecular systems as a whole or in part can be
  controlled using the rendering style buttons in the system list (<span class=
  "menu">MOE | RHS | <a href="system.htm">System</a></span>).</p>

    <table class="titlebar">
      <tr>
        <th>Mode</th>
        <th>Description</th>
        <th>Fig.</th>
      </tr>
       <tr>
        <td>
<a id="HideAtoms"></a>
<span class=
          "nobr"><img src="mainwin/atom_render_hide.png" class="icon" />
          Hide</span>
        </td>
        <td colspan="2">Hide atoms.</td>
       </tr>
       <tr>
        <td>
<a id="ShowAtoms"></a>
<span class=
          "nobr"><img src="mainwin/atom_render_show.png" class="icon" />
          Show</span>
        </td>
        <td colspan="2">Show atoms.</td>
       </tr>
      <tr>
        <td><span class="nobr"><a id="Line"></a>
<img src="mainwin/atom_render_line.png" alt="" class="icon" /> Line</span></td>
        <td>Renders atoms as points and bonds as lines.</td>
        <td><img src="mainwin/atom_line.png" alt="Line Mode" class="top"/></td>
      </tr>
      <tr>
        <td>
<a id="Stick"></a>
<span class="nobr"><img src=
          "mainwin/atom_render_stick.png" alt="" class="icon" />
          Stick</span>
        </td>
        <td>
          Renders atoms as points and bonds as cylinders. The radius of the
          cylinder is determined by the Atom Bond Radius parameter in the
          Dimensions page of the
          <a href="../../appendix/viewsetup.htm">Visualization Setup</a> panel.
        </td>
        <td><img src="mainwin/atom_stick.png" alt="Stick Mode" class="top" /></td>
      </tr>
      <tr>
        <td>
<a id="BallAndLine"></a>
<span class=
          "nobr"><img src="mainwin/atom_render_ball_line.png"  class="icon"/>
          Ball and Line</span>
        </td>
        <td>
          Renders atoms as small spheres and bonds as lines. The size of small
          spheres is determined by the Atom Small Radius parameter in the
          Dimensions page of the
          <a href="../../appendix/viewsetup.htm">Visualization Setup</a> panel.
        </td>
        <td><img src="mainwin/atom_ball_line.png" alt="Ball and Line Mode"
        class="top" /></td>
      </tr>
      <tr>
        <td>
<a id="BallAndStick"></a>
<span class=
          "nobr"><img src="mainwin/atom_render_ballstick.png" class="icon" />
          Ball and Stick</span>
        </td>
        <td>
          Renders atoms as small spheres and bonds as cylinders. The size of
          the small spheres is determined by the Atom Small Radius parameter,
          and the radius of the cylinder by the Atom Bond Radius parameter in
          the Dimensions page of the
          <a href="../../appendix/viewsetup.htm">Visualization Setup</a> panel.
        </td>
        <td><img src="mainwin/atom_ball_stick.png" alt="Ball and Stick Mode"
         class="top" /></td>
      </tr>
      <tr>
        <td>
<a id="SpaceFilling"></a>
<span class=
          "nobr"><img src="mainwin/atom_render_sphere.png" class="icon" />
          Space Filling</span>
        </td>
        <td>Renders atoms as spheres whose radii are displayed at 75% of the
        atomic van der Waals radii.</td>
        <td><img src="mainwin/atom_sphere.png" alt="Space Filling Mode" class="top" /></td>
      </tr>
   </table>
<a id="Coloring_Atoms"></a>
<h2>
<a id="Color"></a>
Coloring Atoms</h2>

<p>By default, MOE colors atoms by element. Other coloring styles are offered
  in the <span class="nobr"><b>MOE | Render | Atoms</b></span> panel.</p>
  <img src="mainwin/atom_render_panel.png" alt="Atom Rendering Panel" class="center" />

<p>Coloring operations are applied to selected atoms only or, if none are
  selected, to all atoms in the system. Coloring only carbon atoms is possible
  by first turning on the <b>C</b> checkbox. The color can be one of the
  predefined set of colors or one of the following:</p>
  <center>
    <table class="titlebar">
      <tr>
        <th>Color</th>
        <th>Description</th>
      </tr>
      <tr>
        <td><img src="mainwin/Color_Drop.png" alt="Color Picker"  class="top" /></td>
        <td>Clicking the down-arrow opens a color picker panel that enables
        selecting a color based on RGB value or HSL value.</td>
      </tr>
      <tr>
        <td>Element</td>
        <td>
          Element type. Default color values may be found in the MOE
          configuration (<tt>moe-rc</tt>) file. Any element type for which a
          default color is not defined will be colored with the Element Color
          Default defined in the
          <a href="../../appendix/viewsetup.htm#MoleculesRibbons">Molecules &amp;
          Ribbons</a> page of the Visualization Setup Panel.
        </td>
      </tr>
      <tr>
        <td>B-Factor</td>
        <td>
          Temperature factor, as determined by the <a class="svl" href=
          "../fcnref/atom.htm">aTempFactor</a> property of each atom. Atoms
          with low temperature are colored blue, high temperature, colored red.
        </td>
      </tr>
      <tr>
        <td>Residue</td>
        <td>
          <a href="se.htm#Coloring_Residues">Residue&nbsp;color</a>, as
          specified in the Sequence Editor. Note that if the residue color is
          the same color as the 3D rendering area's background color then
          the atom will appear to be invisible.
        </td>
      </tr>
      <tr>
        <td>Chain</td>
        <td>
          Chain color. The chain color is displayed in the chain number of the
          <a href="se.htm">Sequence Editor</a> window.
        </td>
      </tr>
      <tr>
        <td>Tag</td>
        <td>
          Chain tag. The tag color is the color of the first chain number in a
          span of consecutive chains (i.e. in <a href="se.htm">Sequence
          Editor</a> order) with the same chain tag.
        </td>
      </tr>
    </table>
  </center>

<p>Atom coloring for molecular systems as a whole or in part can be
  controlled using the coloring style buttons in the system list (<span class=
  "menu">MOE | RHS | <a href="system.htm">System</a></span>).</p>


<a id="Labeling_Atoms"></a>
<h2>
<a id="Label"></a>
Labeling Atoms</h2>

<p>Atoms in the 3D rendering area may be labeled with a variety of
  components. The display of these components is controlled using the
  <span class="nobr"><b>MOE | Render | Atoms</b></span> panel (also available
  from <span class="nobr"><b>MOE | Footer | Atoms</b></span>).</p>
  <img src="mainwin/atom_render_panel.png" alt="Atom Rendering Panel"
  class="center" />

<p>Labels are applied to the <i>selected atoms</i> only. If no atoms are
  selected then the labeling is applied to <i>all</i> atoms. The buttons in the
  panel act as toggles: pressing once adds the associated
  label component; pressing a second time removes it from the label.
  Existing labels can be hidden and shown with the
  <img src="mainwin/atom_render_hide.png"  class="icon"/> and
  <img src="mainwin/atom_render_show.png"  class="icon"/> buttons.
  The <b>X</b> button clears all labels from the applicable
  atoms. With the
  <img src="mainwin/atom_render_labelpanel.png"  class="icon"/> button one
  can open the <a href="label.htm"><b>Label Atoms</b></a> panel for
  custom atom labelling.
  </p>
<a id="Rendering_Ribbons_and_Backbones"></a>
<h2>
<a id="RenderingBackbone">Rendering Ribbons and
  Backbones</a></h2>

<p>The backbone may be rendered in a variety of modes using the <span class=
  "nobr"><b>MOE | Render | Ribbon</b></span> panel, which can also be opened
  with <span class="nobr"><b>MOE | Footer | Ribbon</b></span>.</p>

<p class="center"><img src="mainwin/ribbon_render_panel.png" alt=
  "Backbone and Ribbon Rendering Panel"  class="icon" /></p>

<p>The render commands are applied to selected atoms only. If no atoms are
  selected then the commands will apply to all atoms. The backbone rendering
  style may be one of the following.</p>
  <table class="noborder">
    <tr>
      <td>
        <img src="mainwin/ribbon_render_cartoon.png" alt=""  class="icon" /> &nbsp; <a id="Cartoon"><b>Cartoon</b></a><br />
        <img src="mainwin/ribbon_cartoon.png" alt="Cartoon Backbone"  class="icon" />
      </td>
      <td>
        <img src="mainwin/ribbon_render_ribbon.png" alt=""  class="icon" /> &nbsp; <a id="Ribbon"><b>Ribbon</b></a><br />
        <img src="mainwin/ribbon_ribbon.png" alt="Ribbon Backbone"  class="icon" />
      </td>
      <td>
        <img src="mainwin/ribbon_render_alphatrace.png" alt=""  class="icon" /> &nbsp;
        <a id="AlphaTrace"><b>Alpha Trace</b></a><br />
        <img src="mainwin/ribbon_alphatrace.png" alt="Alpha Trace Backbone"  class="icon" />
      </td>
    </tr>
    <tr>
      <td>
        <img src="mainwin/ribbon_render_line.png" alt=""  class="icon" /> &nbsp; <a id="Line"><b>Line</b></a><br />
        <img src="mainwin/ribbon_line.png" alt="Line Backbone"  class="icon" />
      </td>
      <td>
        <img src="mainwin/ribbon_render_tube.png" alt=""  class="icon" /> &nbsp; <a id="Tube"></a>
<b>Tube</b><br />
        <img src="mainwin/ribbon_tube.png" alt="Tube Backbone"  class="icon" />
      </td>
      <td>
        <img src="mainwin/ribbon_render_none.png" alt=""  class="icon" /> &nbsp;
<a id="None"></a>
<b>None</b>
      </td>
    </tr>
  </table>

<p>Ribbon dimensions may be modified on the
  <a href="../../appendix/viewsetup.htm#MoleculesRibbons">Molecules &amp; Ribbons</a>
  page of the Visualization Setup panel.</p>

<p>The color of the rendered backbone may be set to either one of the set of
  predefined colors or in accordance with one of the following attributes:</p>
  <center>
    <table class="titlebar">
      <tr>
        <th>Color</th>
        <th>Description</th>
      </tr>
      <tr>
        <td>Structure</td>
        <td>
          Macromolecular secondary structure type. Colors are defined for
          helix, strand and loop in the
          <a href="../../appendix/viewsetup.htm#MoleculesRibbons">Molecules &amp;
          Ribbons</a> page of the Visualization Setup panel.
        </td>
      </tr>
      <tr>
        <td>B-Factor</td>
        <td>
          Average of the temperature factors, as given by <a class="svl" href=
          "../fcnref/atom.htm">aTempFactor</a>, of nearby atoms.
        </td>
      </tr>
      <tr>
        <td>Type</td>
        <td>Color by residue hydrophobicity. The classes are: acidic (Asp,
        Glu), basic (Arg, Lys, His), hydrophobic (Ala, Val, Ile, Met, Phe, Pro,
        Trp) and hydrophilic (Gly, Ser, Thr, Asn, Gln, Cys and Tyr).</td>
      </tr>
      <tr>
        <td>Clustal-X</td>
        <td>Clustal-X consensus coloring.</td>
      </tr>
      <tr>
        <td>RMSD</td>
        <td>Color according to the RMSD of alpha-Carbons of the protein chains
        visible in the Sequence Editor.</td>
      </tr>
      <tr>
        <td>Residue</td>
        <td>Residue color, as shown in the Sequence Editor.</td>
      </tr>
      <tr>
        <td>Chain</td>
        <td>The color of the chain number box in the Sequence Editor.</td>
      </tr>
      <tr>
        <td>Tag</td>
        <td>
          The color of the first chain number in a span of consecutive chains
          (i.e. in <a href="se.htm">Sequence Editor</a> order) with the same
          chain tag.
        </td>
      </tr>
      <tr>
        <td>Domain</td>
        <td>
          The color of the protein structural domain as determined by the
          <a class="svl" href="../fcnref/residue.htm">rDomain</a> property of
          each residue.
        </td>
      </tr>
      <tr>
        <td>Terminus</td>
        <td>Smooth transition from N-terminus (blue) to C-terminus (red).</td>
      </tr>
    </table>
  </center>

<p>The <img src="mainwin/ribbon_render_none.png" alt="x"  class="icon" /> button turns off
  the display of backbone ribbon for all applicable atoms.</p>

<p>The Ribbon panel can also be used to turn on and off the display of
  backbone (<b>BB</b>) and sidechain (<b>SC</b>) atoms as well as polar
  (<b>H-X</b>) and non-polar (<b>H-C</b>) hydrogens. Use the buttons labeled
  <b>X</b> to hide specific atoms.</p>


<a id="Rendering_Contacts"></a>
<h2>
<a id="Contacts">Rendering Nonbonded Contacts</a></h2>
  <div class="floatright"><img src="mainwin/contacts.png" alt=
  "Render Contacts" class="floatright" /></div>

<p>Hydrogen bond contacts and van der Waals contacts (clashes) can be
  rendered in the MOE Window in a variety of styles. The <span class="menu">MOE
  | Render | Contacts</span> popup panel, which is also accessible from
  <span class="menu">MOE | Footer | Contacts</span>, provides fine control over
  which interactions are drawn as well the rendering mode. Separate controls
  are provided for hydrogen contacts and clashes.</p>

<p>At the top of the popup panel, the <b>H-Bond</b> and <b>Clash</b>
  checkboxes enable or disable rendering. Next to each checkbox is a button
  which cycles through the contact rendering modes: dashed, dashed with a
  strength bar, dashed with distance label, and dashed with contact strength
  annotation. A text field with arrows provides the threshold (kcal/mol) below
  which contacts will not be displayed.</p>

<p><span class="tip">Tip.</span> The number of precision digits displayed for
  energy and distance labels can be changed with the SVL functions
  <tt>ViewSetup[nb_hbond_meter_precision:[1,2]]</tt> and
  <tt>ViewSetup[nb_clash_meter_precision:[1,2]]</tt>. By default, energy labels
  show 1 digit after the comma, and 2 for distance labels.</p>

<p>If the <b>H-Bond</b> and/or <b>Clash</b> checkboxes are set, the
  <b>Contacts</b> button in the footer is labeled with bold font, providing a
  quick visual cue, whether the nonbonded contacts rendering is enabled.</p>
  <div class="floatright"><img src="mainwin/contacts_cyl.png" alt=
  "Cylinder Mode" class="floatright" /><br />
  <br />
  <img src="mainwin/contacts_strength.png" alt=
  "Contact Strength Annotation"  class="floatright" /></div>

<p>The lower part of the panel provides an interaction matrix for specifying
  which receptor (<b>Rec</b>) / solvent (<b>Sol</b>) / ligand (<b>Lig</b>)
  interactions are displayed. Buttons within the matrix cycle among displaying
  the contacts using the rendering style specified in the top of the panel
  (green), displaying the contacts weakly as thin dashed lines (yellow), and
  not displaying the contacts at all (gray).</p>

<p>For hydrogen contacts, additional checkboxes enable/disable the display of
  backbone-backbone interactions (<b>BB</b>), CH..X contacts involving
  non-standard H-bond donors and acceptors (<b>CH</b>), proton-&pi;
  interactions (<b>H-pi</b>), and &pi;-&pi; interactions (<b>pi-pi</b>).
  Proton-&pi; interactions (between planar rings and protons) and &pi;-&pi;
  interactions (between planar rings) are calculated using a geometric model
  and drawn in a similar fashion to hydrogen bonds;
  &pi;-&pi; interactions display no strength bars or strength annotations.
  These interactions are intended to represent non-standard hydrophobic
  interactions.
  Standard hydrogen bond donor and acceptor definitions are controlled by the
  <tt>$MOE/lib/hbond.svl</tt> configuration file which contains SMARTS pattern
  tables that determine candidates for hydrogen bonds.</p>

<p>For clashes, additional checkboxes enable/disable the display of
  backbone-backbone interactions (<b>BB</b>), van der Waals clashes
  (<b>vdW</b>), and lone pair repulsions (<b>LP</b>).</p>

<p>For both <b>H-Bond</b> and <b>Clash</b> contacts, interactions within and
  among tags can be controlled by the <b>Tag</b> dropdown menu.</p>
  <div class="floatleft"><img src="mainwin/Tag_Drop.png" alt=
  "Tag selection in Contacts Panel" class="floatleft" /></div>
  <ul>
    <li><b>Tag</b> draws contacts only within a tag.</li>
    <li><b>Tag+</b> draws contacts within a tag, and also across tags between
    different atom classes (receptor, solvent, ligand).</li>
    <li><b>All</b> draws contacts within and across tags, regardless of atom
    classes.</li>
  </ul>

<p>Selecting <b>Tag+</b> is useful when visualizing superposed protein ligand
  complexes.</p>

<p>The Contacts panel popup menu, opened by right-clicking in the Contacts
  panel, is used to save the current panel settings and use them as defaults
  for future MOE sessions. The menu also has a command to restore the default
  settings.</p>

<a id="Draw_Options"></a>
<h2>
<a id="Draw">Draw Settings</a></h2>

<p>The Draw menu is used to control the rendering quality and font of the MOE
  Window rendering area. The display of the system coordinate axes is also set
  here.</p>
  <center>
    <table class="titlebar">
      <tr>
        <th>Command</th>
        <th>Description</th>
      </tr>
      <tr>
        <td><b>Coordinate Axes</b></td>
        <td>
          <div >
            <img src="mainwin/axes.png" alt="Coordinate Axes" style=
            "vertical-align:top; float:right" /> Displays X-Y-Z axes. The
            origin of the axes is drawn at the
            <a href="#Changing_the_Center_of_Rotation">center of rotation</a> of the
            system, which is, by default, the center of mass of all atoms in
            the system.
          </div>
        </td>
      </tr>
      <tr>
        <td><b>High Quality Rotation</b></td>
        <td>
          When off, lower quality graphics is used to display the molecular
          system any time the view is translated or rotated, or the zoom is
          increased or decreased, and similarly when atoms of the molecule are
          moved (using the <a class="svl" href=
          "#EditingAtomsUsingTheMouse">Alt</a> key). The use of lower quality
          graphics accelerates view transformations when large numbers of
          graphic objects and space-filling atoms are present. When on, high
          quality graphics rendering is used throughout.
        </td>
      </tr>
      <tr>
        <td><b>High Quality Anti-Alias</b></td>
        <td>
          Enables
          <a href="../../appendix/viewsetup.htm#AntiAliasing">Full-Scene Accumulation
          Anti-Aliasing</a>, which produces high quality graphics with smooth
          lines and polygons. On some low-end graphics cards this option might
          not be supported or very slow.
        </td>
      </tr>
      <tr>
        <td><b>Font</b></td>
        <td>The font used for drawing labels or for displaying messages in the
        MOE Window are controlled by the options here. The font size can be set
        to one of <b>Small</b>, <b>Medium</b> or <b>Large</b>. The default font
        is a proportional font, but can be changed to <b>Fixed Width</b>. The
        font style can also be set to <b>Bold</b>.</td>
      </tr>
    </table>
  </center>

<p>To save the current values for future MOE sessions, use the <span class=
  "nobr"><b>Save As Default</b></span> option. <a id="Display_Modes"></a>
</p>

<h2>
<a id="DisplayModes"></a>
Stereo Mode</h2>

<p>The <b>Stereo</b> and <b>Full Screen (F12)</b> modes control how the MOE
  system will be displayed:</p>
  <center>
    <table class="titlebar">
      <tr>
        <th>Command</th>
        <th>Description</th>
      </tr>
      <tr>
        <td>
<a id="Perspective"><b>Perspective</b></a>
        </td>
        <td><span class="menu">MOE | Render | Stereo | Perspective</span>
        displays an image of the molecular system in which depth is implied
        using perspective (this is the default mode).</td>
      </tr>
      <tr>
        <td>
          <a id="ParallelProjection"></a>
<b>Parallel</b>
        </td>
        <td><span class="menu">MOE | Render | Stereo | Orthographic</span>
        displays the molecular system without any perspective.</td>
      </tr>
      <tr>
        <td>
<a id="Stereo"><b>Stereo</b></a>
        </td>
        <td>
          Displays a stereo image of the molecular system using the current
          default stereo method. The default stereo mode can be set in the
          <a href="../../appendix/settings.htm#StereoMenu"><span class=
          "menu">MOE | Render | Stereo | Stereo Mode</span></a> menu.
        </td>
      </tr>
      <tr>
        <td>
          <a id="FullScreen"></a>
<b>Full Screen (F12)</b>
        </td>
        <td>Displays the MOE Window as the full screen. Once in full screen
        mode, you must press <tt>F12</tt> or <tt>Esc</tt> or select
        <span class="menu">MOE | Render | Full Screen (F12)</span> or
        <span class="menu">MOE | Popup | Full Screen</span> to get back to
        normal viewing. These commands all toggle between the modes.</td>
      </tr>
    </table>
  </center>

<h2>
<a id="DuplicateWindow"></a>
Duplicate 3D Window</h2>

<p>The 3D view of the main window can be duplicated into a separate window
or in fullscreen onto another monitor. This is useful for presentations.
The duplicated window can be set to use a different stereo mode than the main
MOE window.
</p>
<p>To duplicate the 3D window select
<span class="menu">MOE | Render | Stereo | Stereo Mode | Duplicate 3D</span>.
The view of the duplicate 3D window is synchronized with the main MOE 3D window.
To configure the duplicate window right click in it and select
<span class="menu">Popup | Duplicate</span>.
The <b>Duplicate</b> menu allows to switch between window and fullscreen
mode, setting the stereo mode, and closing the duplicate window.
</p>


<a id="Rock&Roll"></a>
<h2>Rock&nbsp;and&nbsp;Roll</h2>

<p>Agitates the view onto the molecular system in space so that the
  three-dimensionality of the system becomes more apparent.</p>

<!--
<a id="Rotating_Translating_and_Zooming_Atoms"></a>
<h1><a id="Rotating">Rotating, Translating and Zooming Atoms</a></h1>

<p>
<a id="DialsFooter"></a> You can use the mouse, the Dials panel or the
    Dials footer to rotate, translate or zoom the molecular system in real
    time. Note that changing the view of the molecular system by rotating,
    translating or zooming in/out does not alter the positions of the atoms in
    the system. To <a href="#EditingAtomsUsingTheMouse">change atom
    coordinates</a> and move them independently of other atoms (unless they are
    fixed), use the <tt>Alt</tt> key, or, in the case of the Dials panel,
    choose the set of atoms to be operated upon.

<p>
To use the <b>Dials Footer</b> choose the <b>Dials</b> page in the
    bottom left corner of the MOE Window:
</p>

    <center>
        <img src="mainwin/foot1.gif" alt="Dials Footer" vspace="10">
    </center>

    <center>
        <table class="noborder">
            <tr>
                <th>To:</th>

                <th>With the Mouse:</th>

                <th>With the Dials Footer:</th>
            </tr>

            <tr>
                <td>Translate view in X or Y</td>

                <td><tt>Shift</tt>-drag the middle mouse
                button</td>

                <td>Drag the left mouse button over the X or Y
                Translate wheels</td>
            </tr>

            <tr>
                <td>Rotate view</td>

                <td>Drag the middle mouse button on molecule</td>

                <td>Drag the left mouse button over the Rotate
                wheels</td>
            </tr>

            <tr>
                <td>Rotate view about the Z axis only</td>

                <td>Drag the middle mouse button anywhere in
                perimeter of 3D rendering area</td>

                <td>Drag the left mouse button over the Z Rotate
                wheel</td>
            </tr>

            <tr>
                <td>Zoom in/out</td>

                <td><tt>Ctrl</tt>-drag the middle mouse
                button</td>

                <td>Drag the left mouse button over the Zoom
                wheel</td>
            </tr>
        </table>
    </center>

<p>
When using the Dials, press <tt>Ctrl</tt> for finer control as you turn
    the dials. Pressing <tt>Ctrl</tt> serves to scale down MOE's response to
    the actual mouse movement. The sensitivity of the mouse may be changed in
    the <a href="../../appendix/settings.htm#MainMouse">
    Mouse Configuration Panel</a>..

<p>
The <a id="DialsPanel"><b>Dials Panel</b></a> is an alternate method
    of rotating, translating and zooming atoms. To open the Dials Panel choose
    <b><span class="nobr">MOE | Window | Dials</span></b>. The following panel appears:
</p>

    <center>
        <img src="mainwin/dials.png" alt="img">
    </center>

<p>
The <b>Apply To</b> menu defines the set of atoms to operate on. By
    default, this menu has <b>View</b> and <b>Selected Atoms</b> options. In
    addition, all system <b>chain tags</b> and any user defined sets will
    appear in the menu, allowing you to operate on a specific set of
    atoms.
</p>

<p>
<b>Warning!</b>&nbsp;&nbsp; When applying the dials operations to a
    restricted set of atoms, instead of the entire system (i.e.
    <b>View</b>), the actual atom positions will be altered.
</p>

<p>
<font color="#ff0000"><b>Tip</b></font>&nbsp;&nbsp; User defined atom
    sets can be created using the <a href="select.htm">Atom Selector</a>.
</p>
-->

<a id="Clipping_Planes_and_Depth_Cueing"></a>
<h1><a id="Clipping">Clipping Planes and Depth Cueing</a></h1>

  <p class="clear-both">
    <img src="mainwin/zclip_def.png"
      alt="Default ZClip/Fog Options" class="floatright" />

    The <a href="#Footer">Footer</a> contains controls for the Z-axis clipping
    region (<b>ZClip</b>) and the Z-axis depth cue shading parameters
    (<b>Fog</b>).  Parts of molecules can be clipped away using 3D front and
    back clipping planes. The global clipping planes affect the complete
    molecular system, including atoms, ribbons, surfaces, graphics objects,
    etc. The placement of these planes is controlled by the ZClip settings.
    Atoms hidden by Z-clipping participate normally in computations.
    The three-dimensional appearance of molecules can be enhanced using
    distance-dependent shading. Fog (depth cue) parameters are controlled
    in a similar way.
  </p>

  <p>
    <b>Note:</b> Hidden atoms cannot be selected by clicking on them with
    the mouse (they can, however, be unselected by clicking with the mouse).
    Selected hidden atoms are otherwise treated normally, and will be
    affected by any editing or movement commands that apply to selected
    atoms.
  </p>

  <p>
    Pressing on either <b>ZClip</b> or <b>Fog</b> will raise a popup
    panel with the corresponding settings.  Either popup will also
    show a symbolic diagram visualizing where the molecule is on the z-axis
    in order to help set the Z-Clipping and Fog parameters.  In this diagram,
    the protein is shown as a green blob, with the binding site shown darker
    and the extent of the surfaces (if any) in red.  The clipping planes
    are shown in blue (red for surfaces) and the fog strength with gray
    shading.  Pressing on the <img src="mainwin/detach.png" class="icon" />
    icon in the upper right of either popup will raise an independent combined
    panel (as shown) regrouping both sets of options which will remain open.
    The standard settings are described below.
  </p>

  <table class="titlebar">
    <tr>
      <th>Command</th>
      <th>Description</th>
    </tr>
    <tr>
      <td><b>Z-Clip</b></td>
      <td>
	The positions of the front and back clipping planes.
	Use the range slider to set the clipping planes:
	The left slider thumb represents the front clipping plane, the right
	thumb the back plane.
	The blue bars show the extent of the molecule along the Z-axis.
	The default position corresponds to the bounding box of the system.
      </td>
    </tr>
    <tr>
      <td><b>Surface Clip</b></td>
      <td>
	The positions of the surface-only front and back clipping planes.
	The red bars show the extent of the surface along the Z-axis.
	The surface clipping planes must be inside the clipping planes
	for the system.  The default position is equal to the system
	clipping planes.
      </td>
    </tr>
    <tr>
      <td><b>Abs.&nbsp;Start</b></td>
      <td>
	The start position of the fog (depth cue) effect. The green bar
	shows the extent of the molecule along the Z-axis.
      </td>
    </tr>
    <tr>
      <td><b>Strength</b></td>
      <td>
	The strength of the fog (depth cue) effect.
      </td>
    </tr>
  </table>

  <p class="clear-both">
    <img src="mainwin/zclip_auto.png"
      alt="Automatic ZClip/Fog Options" class="floatright" />

    The <b>Auto Z-Clip</b> checkbox can be selected to position the clipping
    planes automatically.  If enabled, the Z-Clipping planes will be
    set to be optimally positioned around the selected region (e.g. the
    ligand), adjusting automatically when the molecule is rotated or moved.
    When <b>Auto Z-Clip</b> is enabled, auto <b>Fog Range</b> is also
    enabled by default such that the fog start and strength are adjusted
    to the position of the Z-clipping planes instead of the overall extent
    of the system volume.  Auto <b>Fog Range</b> can be separately
    disabled to use the global fog settings or adjust the fog start and
    strength manually within the Z-Clipped volume.  When using automatic
    Z-Clipping, the following settings are available.
  </p>

  <table class="titlebar">
    <tr>
      <th>Command</th>
      <th>Description</th>
    </tr>
    <tr>
      <td><b>Z-Clip&nbsp;Around</b></td>
      <td>
	Specify set of atoms around which the automatic Z-clipping range
	is defined.
      </td>
    </tr>
    <tr>
      <td><b>Set&nbsp;Selected</b></td>
      <td>
	Press to use currently selected atoms to define Z-clipping range.
      </td>
    </tr>
    <tr>
      <td><b>Addtl.&nbsp;Border</b></td>
      <td>
	Specify additional border around the atoms of the Z-clipping site.
      </td>
    </tr>
    <tr>
      <td><b>Rel.&nbsp;Start</b></td>
      <td>
	The start position of the fog (depth cue) effect within the
	Z-clipping range.
      </td>
    </tr>
    <tr>
      <td><b>Strength</b></td>
      <td>
	The strength of the fog (depth cue) effect within the Z-clipping range.
      </td>
    </tr>
  </table>

<a id="2D_Ligand_Overlay"></a>
<h1 class="clear-both"><a id="2dLigandOverlay">2D Ligand Overlay</a></h1>

  <p>
    The <a href="#Footer">Footer</a> contains a control to activate the 2D
    Ligand Overlay.  When this is active, a 2D ligand diagram is shown in
    the lower right corner of the main window as shown in the example below.
    The diagram is selectable with the left mouse button and the atom
    selection is synchronized between the 2D diagram and the main MOE window.
    If more than one ligand is present in the system, the overlay will
    show one at a time.  Arrow buttons in the lower right corner of the
    overlay can be used to navigate between different ligands. If one
    reaches the last ligand the next button click wraps to the first
    and vice versa. The current ligand chain name is shown under the
    ligand structure, and if the ligand chain name does not match the
    ligand chain tag the tag is shown as well.
  </p>
  <p>
    By default the overlay shows only hydrogens connected to heteroatoms.
    A single click on a heavy atom in the diagram will only select the
    heavy atom, the next click will add the attached hydrogens to the
    selection. The selection state (if only the heavy atom or also
    hydrogens are selected) is indicated by the selection disk size
    in the diagram. A drag/box selection will select the heavy atoms
    inside the box with all attached hydrogens.
  </p>
  <p>
    The <b>H</b> icon in the lower right corner of the overlay panel toggles
    the display of all explicit hydrogens. If the hydrogens
    are shown, they can be selected individually.
    The <b>H</b> is colored pink if hydrogens are selected.
  </p>

  <img src="mainwin/2d_overlay.png"
    alt="2D Ligand Overlay" class="center" />

  <p class="clear-both">
    <img src="mainwin/2d_settings.png"
      alt="2D Ligand Settings" class="floatright" />

    The 2D overlay can be configured by pressing the
    <img src="mainwin/2d_gear.png" class="icon" />
    icon in the lower right corner of the overlay panel.  This will raise
    the 2D Ligand Settings panel with the options described below.
  </p>

  <table class="titlebar">
    <tr>
      <th>Command</th>
      <th>Description</th>
    </tr>
    <tr>
      <td><b>Selection</b></td>
      <td>
	If on, atom selections in the 2D ligand overlay are applied only
	onto the currently shown ligand. The selection states of other
	atoms are not changed.
      </td>
    </tr>
    <tr>
      <td><b>Hydrogens</b></td>
      <td>
	If on, existing hydrogens are shown, otherwise only
	implicit polar hydrogens on hetero atoms are depicted.
	Explicit and implicit chiral hydrogens with neighboring
	chiral centers are always shown.
      </td>
    </tr>
    <tr>
      <td><b>View</b></td>
      <td>
	If on, the 2D ligand diagram orientation is matched on the fly to the
	3D coordinates and orientation on the screen.
      </td>
    </tr>
    <tr>
      <td><b>Chirality<br />R/S&nbsp;Wedges</b></td>
      <td>
	If on, atom RS stereo chemistry is shown in the 2D ligand diagram
	with wedge bonds.
      </td>
    </tr>
    <tr>
      <td><b>Chirality<br />R/S&nbsp;Labels</b></td>
      <td>
	If on, atom RS stereo chemistry labels are shown in the 2D ligand
	diagram.
	Green labels indicate chiral atoms without chiral constraints.
	Atoms with blue labels show that the current geometric chirality
	matches the constraint, and red labels indicate that the
	constraint is violated or set on a non-chiral atom.
      </td>
    </tr>
    <tr>
      <td><b>Chirality<br />E/Z&nbsp;Labels</b></td>
      <td>
	If on, double bond EZ stereo chemistry is shown in the 2D
	ligand diagram.
	Green labels indicate chiral bonds without chiral constraints.
	Bonds with blue labels show that the current geometric	chirality
	matches the constraint, and red labels indicate that the
	constraint is violated.
      </td>
    </tr>
    <tr>
      <td><b>Position</b></td>
      <td>
	Set position of 2D ligand overlay in 3D main window.
      </td>
    </tr>
    <tr>
      <td><b>Scale</b></td>
      <td>
	Set size/scale factor of 2D ligand depiction. This can also be
	changed with the mouse wheel while the mouse cursor is inside
	the 2D overlay in the 3D main window.
      </td>
    </tr>
    <tr>
      <td><b>Opacity</b></td>
      <td>
	Set opacity of 2D ligand depiction overlay. If set to a lower
	value, the molecule in the 3D window will shine through more.
      </td>
    </tr>
    <tr>
      <td><b>Save</b></td>
      <td>
	Save current settings as default.
      </td>
    </tr>
    <tr>
      <td><b>Standard</b></td>
      <td>
	Restore settings to standard values.
      </td>
    </tr>
    <tr>
      <td><b>Export</b></td>
      <td>
	<a href="../../apps/print.htm#GraphicExportPanel">Export</a>
	current 2D ligand depiction as image file. Depictions with white
	and black background will be available in the export panel.
	<p>The 2D Ligand overlay is included in
	<a href="#Copy_and_Paste">snapshots</a>.</p>
      </td>
    </tr>
  </table>




<a id="Editing_Molecular_Objects"></a>
<h1>
<a id="Editing">Editing Molecular Objects</a></h1>

<a id="Undo"></a>
<h2>
<a id="Undo_and_Redo">Undo and Redo</a></h2>

<p>A global undo system exists in MOE to track changes to the currently
  loaded molecular system. Edits to the system can be undone by pressing
  <tt>Ctrl+Z</tt> or, equivalently, <span class="menu">MOE | Edit |
  Undo</span>. The menu item is insensitive if there are no more undo
  states.</p>

<p>When a change is undone, it can be restored by pressing <tt>Ctrl+Y</tt> or
  <tt>Ctrl+Shift+Z</tt>, or by using <span class="menu">MOE | Edit |
  Redo</span>. The menu item is insensitive if there are no more redo
  states.</p>

<p>Note that undo/redo is limited to changes to the MOE molecular system. For
  example, changes to parameter values within panels are not tracked by the
  undo system, and neither are surfaces deemed private to applications (e.g.
  <a href="../../apps/ph4_query.htm">pharmacophore annotations</a>).</p>

<p>Note: The undo system is &quot;reset&quot; when the system is closed (e.g.
  <span class="nobr"><span class="menu">MOE | RHS | Close</span>)</span>,
  meaning that any changes prior to the system's closing cannot be
  recovered, and that the close operation itself cannot be undone. Note also
  that if an undo action has been performed, a change to the system will remove
  and replace the redo state (and any subsequent redo states) so that only this
  new action will be undoable.</p>

<a id="Editing_Atoms_Using_the_Mouse"></a>
<h2>
<a id="EditingAtomsUsingTheMouse">Editing Atoms Using the Mouse</a></h2>
  <table class="titlebar">
    <tr>
      <th>Operation</th>
      <th>Mouse Action</th>
    </tr>
    <tr>
      <td>Translate selected atoms</td>
      <td><tt>Shift</tt>+<tt>Alt</tt>-drag the middle mouse button</td>
    </tr>
    <tr>
      <td>Rotate selected atoms</td>
      <td><tt>Alt</tt>-drag the middle mouse button</td>
    </tr>
    <tr>
      <td>Rotate selected atoms about view Z axis only</td>
      <td><tt>Alt</tt>-drag the middle mouse button in the perimeter of
      rendering window</td>
    </tr>
  </table>

<p>The <a href="#Footer"><span class="nobr"><b>MOE | Footer |
  Trackball</b></span></a> can also be used to modify the positions of atoms by
  holding down the <tt>Alt</tt> key while dragging over the trackball.</p>

<p><span class="tip">Tip!</span> On some systems it is possible that
  <tt>Alt</tt> will not work as described. In these cases, it is necessary to
  edit the <a href="mouse.htm#WindowPreferences">Window Preferences</a>
  manager.</p>

<a id="Rotating_About_a_Bond"></a>
<h3>
<a id="RotatingAboutABond">Rotating About a
  Bond</a></h3>

<p>To rotate atoms about a bond, select the first atom, press <tt>Shift</tt>
  and select the second atom in the bond (or alternatively select both atoms by
  picking on the bond itself). Rotating about the bond can be achieved using
  the controls in the <a href="#FooterEditing">MOE Footer</a> or by pressing
  the <tt>Alt</tt> key and dragging the left mouse button.</p>

<p>Only atoms on one side of the bond will move. The side that moves can be
  specified using the star button in the footer. The default side that moves is
  defined to be the one with fewer atoms or, if there are fixed atoms in the
  system, the one with no fixed atoms. No rotation will occur if:</p>
  <ul>
    <li>More than two atoms are selected</li>
    <li>Selected atoms are not bonded</li>
    <li>One of the selected atoms has no other atoms bonded to it</li>
    <li>The bond is part of a ring</li>
    <li>Fixed atoms are preventing the rotation (possible to override with the
    <b>F</b> button in the footer).</li>
  </ul>

<a id="Editing_Atoms_Using_the_Edit_Menu"></a>
<h2>
<a id="EditingAtomsUsingTheEditMenu">Editing Atoms Using the Edit Menu</a></h2>

<p>The Edit menu contains a variety of commands for editing atom properties.
  Some editing commands open a prompt in the CLI of the MOE Window.
  Modifications are done immediately and are reflected in the entire MOE
  system. Note that, when atom prompting is active, normal atom selection is
  not available.</p>

<p>If a prompt is already opened, and a second prompt is started, the first
  terminates immediately before the second one takes effect. Press <tt>Esc</tt>
  or the red 'X' to remove a prompt, or cancel the editing command by
  choosing it from the Cancel menu.</p>

<a id="2D_Editing"></a>
<h2>
<a id="2DEditing">2D Editing: Sketchers</a></h2>

<p>MOE provides functionality to edit molecules in a 2D environment by
  seamlessly connecting to some popular chemical drawing programs. For more
  information, please see <a href="../../moe/sketcher.htm">2D Sketcher
  Programs</a>.</p>

<a id="Copy_and_Paste"></a>
<h2>
<a id="Copy">Copy and Paste</a></h2>

<p>MOE provides clipboard copy and paste commands to facilitate the transfer
  of data between MOE sessions and between MOE and third party software. For
  example, molecular data can be transferred between MOE and 2D sketchers via
  the clipboard, and high quality images can be placed on the clipboard.</p>

<p>The default <b>Copy</b> command in the MOE Window <span class=
  "nobr">(<tt>Ctrl+C</tt></span> or <span class="nobr"><span class="menu">MOE |
  Edit | Copy</span>)</span> puts 3D molecular data onto the clipboard along
  with 2D molecular data in the case of ligands, suitable for pasting into a 2D
  sketcher, for instance.</p>

<p>The <b><span class="nobr">MOE | Edit | Copy As</span></b> menu provides
  commands for copying data from the MOE Window onto the clipboard. Each item
  in the menu determines a data format that will be placed onto the clipboard:
  molecule formats place molecular data onto the clipboard, the Picture option
  places image data onto the clipboard. If the MOE molecule format is chosen
  and no atoms are selected, graphics, view settings,and crystal parameters (if
  periodic systems are enabled) are also written to the clipboard.</p>

<p><b><span class="nobr">MOE | Edit | Copy As | Picture</span></b> places a
  picture onto the clipboard after setting size and related options in the Copy
  Picture to Clipboard panel.</p>
  <img src="mainwin/copy_pic.png" alt="Copy Picture to Clipboard Panel" class="center" />
  <ul>
    <li>
      <b>Format</b>. Specifies the image format in which the picture will be
      put onto the clipboard. The available formats are summarized in the
      following table:
      <center>
        <table class="titlebar">
          <tr>
            <th>Operating System</th>
            <th>Formats</th>
          </tr>
          <tr>
            <td>Windows</td>
            <td>Portable Network Graphics (PNG)<br />
            Windows Bitmap (BMP)<br />
            Enhanced Windows Metafile (EMF)<br />
            Encapsulated PostScript (EPS)</td>
          </tr>
          <tr>
            <td>Linux</td>
            <td>Portable Network Graphics (PNG)<br />
            Bitmap Image (BMP)<br />
            JPEG Image (JPG)<br />
            Tagged Image File (TIF)<br />
            Encapsulated PostScript (EPS)</td>
          </tr>
          <tr>
            <td>macOS</td>
            <td>Portable Network Graphics (PNG)<br />
            Encapsulated PostScript (EPS)</td>
          </tr>
<!--
          <tr>
            <td>Solaris</td>
            <td>Encapsulated PostScript (EPS)</td>
          </tr>
-->
        </table>
      </center>
    </li>
    <li><b>Dimension</b>. Specifies the picture size in pixels, inches,
    centimeters or percent. Pictures of any size can be created. Units of
    percent are calculated relative to the current MOE Window size.</li>
    <li><b>Resolution</b>. Specifies the picture printing resolution. The
    resolution determines the size of the picture when printed. For example, a
    300 dpi picture of 300x300 pixels will occupy a 1 inch square when printed.
    The resolution is only saved with picture formats that support it (PNG and
    TIFF) otherwise it is used here to calculate the picture size in pixels of
    the picture to be created.</li>
    <li><b>Anti-Alias Picture</b>. This will apply high quality anti-aliasing
    to the output picture.</li>
    <li><b>High Quality</b>. This will apply maximum ribbon and focal blur
      quality to the output picture, otherwise the quality settings from the
      <a href="../../appendix/viewsetup.htm">Visualization Setup</a> panel are
      used.
    </li>
    <li><b>Transparent Background</b>. <i>EMF, PNG and TIFF format.</i> If on,
    the picture pixels matching the color of the MOE Window background are
    marked as transparent. If off, the background color is retained in the
    image file.</li>
    <li><b>White Background</b>. <i>BMP, EPS, and JPG format.</i> If on, the
    picture background color will be white. If off, the background color is
    retained in the image file.<br clear="both" /></li>
    <li><b>2D Ligand</b>. Include 2D ligand overlay in output picture.</li>
  </ul>

<p>The <b><span class="nobr">MOE | Edit | Paste</span></b> menu command
  allows the loading of clipboard data into MOE. If the clipboard contains a
  SMILES string, a molecule file, or an ISIS/Draw or ChemDraw sketch of a
  molecule, the data will be loaded directly into MOE. If it contains an
  ISIS/Draw or ChemDraw sketch of a query molecule (for example with ambiguous
  bond orders) a sketch panel will appear instead (see below). In all other
  cases the Paste Special panel will be raised allowing loading of other
  formats.</p>
  <img src="mainwin/pastespecial.png" alt="Paste Special Panel" class="center" />

<a id="Drag_and_Drop"></a>
<h3>
<a id="DnD">Drag and Drop</a></h3>

<p>A related operation to copy/paste is that of dragging and dropping files
  onto a MOE session. Databases (MDB, SDF), SVL scripts, and molecules in
  various formats (<tt>.moe</tt>, <tt>.pdb</tt>, <tt>.mol</tt>, etc.) can be
  drag-and-dropped onto the MOE Window where they will be automatically loaded
  or opened. Dragging and dropping gzipped and zipped PDB, SDF, and CIF files
  is also supported.</p>

<p>On Windows and Linux the files can be dropped onto the MOE main window,
  the Sequence Editor and Database viewers as well as the MOE program icon on
  the Desktop. On macOS, files can only be dropped onto the MOE application
  icon on the Desktop or in the Dock.</p>

<h2>
<a id="Duplicate">Duplicate</a></h2>

<p><b><span class="nobr">MOE | Edit | Duplicate</span></b> allows
to duplicate a ligand, chain or whole tag: Select an atom of the
molecule object to be duplicated in the MOE Window and then choose in
the <b>Duplicate</b> menu either <b>Ligand</b>, <b>Chain</b>, or <b>Tag</b>.
The molecule atoms, residues and chains are duplicated along with membership
in any atom sets, restraints, tethers, etc. The chain or tag name of the newly created
ligand, chain or tag will have a numeric copy number appended; for example, a copy
of MyLigand will be given the chain name MyLigand (2).
The original molecule object will be hidden and de-activated (made inert) and
the created copy will have all of its atoms selected.
Additional duplication controls are found in
<span class="menu">MOE | Settings
 <img src="mainwin/settings.png" alt="Settings" class="icon" /> |
 <a href="../../appendix/settings.htm#QuickPrep">QuickPrep</a>
</span>.

<a id="Sketch_View"></a>
<h2>
<a id="SketchView">Sketch View</a></h2>

<p>When a query mol file (SKC file, <tt>.skc</tt>) or sketch is pasted from
  the clipboard or loaded into MOE using the <a href="fileopen.htm">Open</a>
  panel, the Sketch View panel will be displayed.</p>
  <img src="mainwin/sketchview.png" alt="Sketch View Panel" class="center" />

<p>Some features will be indicated by a letter (a, b, c, etc.) with a legend
  indicating what the letter represents (a bond type, element list, and so on).
  Several buttons appear below the sketch. If the query is a 3D query molecule,
  a message to this effect will appear. Use the <b>Create</b> button in this
  case to display the query in the MOE Window; constraints, lines, normals and
  planes will be drawn as graphics objects. The representation in the MOE
  Window is strictly for purposes of visualizing the query in 3D.</p>

<p><b>Export...</b> allows the sketch to be saved in various graphical
  formats and <b>Print...</b> outputs the sketch to a printer. <b>Close</b>
  will exit the panel.</p>

<a id="Deleting_Atoms"></a>
<h2>
<a id="DeletingAtoms">Deleting Atoms</a></h2>

<p><b><span class="nobr">MOE | Edit | Delete</span></b> (or, equivalently,
  <span class="menu">MOE | RHS | Delete</span>) deletes the currently selected
  atoms (and possibly the associated residues and chains). Alternatively, the
  keyboard <tt>Del</tt> or <tt>Delete</tt> key can be used.</p>

<p>If no atoms are selected, no action will be taken. If all atoms of a
  residue are selected, then deleting the selected atoms will also result in
  the residue being deleted. If all atoms of a chain are selected, then
  deleting the selected atoms will also result in the chain (and all its
  residues) being deleted.</p><!--
    Otherwise, you will be asked to confirm the deletion in the Delete
    panel.  This panel also provides
    information on the objects slated for deletion
    as well as an option to
    delete resulting emptied residues and chains, if any.

</p>

    <center>
        <img src="mainwin/del_atom.png" alt="Delete Atom Panel">
    </center>
-->

<p>The system list (<span class="menu">MOE | RHS |
  <a href="system.htm">System</a></span>) can be used to delete molecular systems as a
  whole or in part (receptor, solvent, or ligand components).</p>

<h2><a id="Hydrogens_and_Lone_Pairs">Hydrogens and Lone Pairs</a></h2>

<p>The <span class="menu">MOE | Edit | Build | Hydrogens</span> menu provides
  commands for creating and/or deleting hydrogen atoms. The following commands
  apply to selected atoms only or, if no atoms are selected, to all atoms.</p>
  <center>
    <table class="titlebar">
      <tr>
        <th>Command</th>
        <th>Description</th>
      </tr>
      <tr>
        <td>Add Hydrogens</td>
        <td>Adjusts the number of attached hydrogens to conform to
        valence.</td>
      </tr>
      <tr>
        <td>Add Polar Hydrogens</td>
        <td>Adjusts the number of hydrogens attached to polar atoms (N, O, F)
        to conform to valence.</td>
      </tr>
      <tr>
        <td>Add Hydrogens and LPs</td>
        <td>Adjusts the number of hydrogens and Lone Pair pseudo-elements on
        selected atoms only.</td>
      </tr>
      <tr>
        <td>Delete Hydrogens</td>
        <td>Deletes attached hydrogen atoms.</td>
      </tr>
      <tr>
        <td>Delete Nonpolar Hydrogens</td>
        <td>Deletes hydrogen atoms attached to non-polar atoms (not N, O,
        F).</td>
      </tr>
      <tr>
        <td>Delete Lone Pairs</td>
        <td>Deletes attached Lone Pair pseudo-elements.</td>
      </tr>
    </table>
  </center>

<h2><a id="PTM">PTM</a></h2>

<p><span class="menu">MOE | Edit | Build | PTM</span> performs
  Post-Translational Modifications on the currently selected atoms, where
  possible. There attachment of the specified group must be both chemically and
  sterically permitted. A simple van der Waals potential is used to determine
  the latter, and solvent atoms are ignored. The permitted modifications
  are:</p>
  <ul>
    <li>fucose</li>
    <li>glucose</li>
    <li>galactose</li>
    <li>mannose</li>
    <li>n-acetylglucosamine</li>
    <li>n-acetylgalactosamine</li>
    <li>xylose</li>
    <li>acetylation</li>
    <li>methylation</li>
    <li>nitrosylation</li>
    <li>phosphorylation</li>
    <li>sulfation</li>
  </ul>

<a id="Automatic_Bonding_and_Typing"></a>
<h2>
<a id="Automatic">Automatic Bonding and Typing</a></h2>

<p>The <span class="nobr"><b>MOE | Edit | Build | Automatic</b></span> menu
  provides commands for the automatic creation of bonds and for the automatic
  typing of atoms in the currently loaded molecular system. Such commands are
  useful for correcting molecular systems in which, for example, connectivity
  information is missing. This is not uncommon in PDB files, for instance.</p>

<p>The automatic connection and typing commands in MOE operate either on
  selected atoms only or on all atoms if none are selected:</p>
  <center>
    <table class="titlebar">
      <tr>
        <th>Command</th>
        <th>Description</th>
      </tr>
      <tr>
        <td>Connect and Type</td>
        <td>
          Do automatic bond creation and atom typing using the methodology of
          [Labute&nbsp;2005]:

<p><i>Automatic Bonding.</i> Use the atom elements and atom 3D
          coordinates to detect bonds and to connect the atoms. If operating on
          selected atoms, bonds will not be created to atoms outside the
          selection set.</p>

<p><i>Automatic Typing.</i> Detect and assign atom ionization state,
          hint LP state and hybridization from atom element, 3D coordinate, and
          connectivity information alone.</p>
        </td>
      </tr>
      <tr>
        <td>Connect Only</td>
        <td>Do automatic bonding of atoms.</td>
      </tr>
      <tr>
        <td>Reconnect Only</td>
        <td>Similar to Connect except that all bonds between atoms are deleted
        prior to detecting connections. If operating on selected atoms, bond
        deletion and creation is done only on atoms within the selection
        set.</td>
      </tr>
      <tr>
        <td>Type Only</td>
        <td>Do automatic typing of atoms.</td>
      </tr>
    </table>
  </center>

<a id="Chirality_Constraints"></a>
<h2>
<a id="Chirality">Chirality Constraints</a></h2>

<p>The <span class="menu">MOE | Edit | Build | Chirality</span> menu provides
  commands for the assignment of chiral constraints on selected atoms:</p>
  <center>
    <table class="titlebar">
      <tr>
        <th>Command</th>
        <th>Description</th>
      </tr>
      <tr>
        <td>R/Z</td>
        <td>Sets selected chiral atom centers to R and stereo-active double
        bonds to Z.</td>
      </tr>
      <tr>
        <td>S/E</td>
        <td>Sets selected chiral atom centers to S and stereo-active double
        bonds to E.</td>
      </tr>
      <tr>
        <td>Geometry</td>
        <td>Marks selected chiral centers as R or S and stereo-active double
        bonds as E or Z, according to 3D geometry.</td>
      </tr>
      <tr>
        <td>Clear</td>
        <td>Clears stereochemistry constraints.</td>
      </tr>
    </table>
  </center>

<p><b>Note!</b> The chiral annotations are topological in nature; in other
  words, they will not affect the current geometry of the system. <a id="Potential_Energy_Attributes"></a>
</p>

<h2>
<a id="EditPotential">Potential Energy
  Attributes</a></h2>

<p>The <b><span class="nobr">MOE | Edit | Potential</span></b> menu provides
  commands for modifying attributes related to the potential energy model of
  the selected atoms:</p>
  <center>
    <table class="titlebar">
      <tr>
        <th>Command</th>
        <th>Description</th>
      </tr>
      <tr>
        <td>Fix</td>
        <td>Anchors the selected atoms to their current positions. This can
        also be done using <span class="menu">MOE | RHS | Constrain |
        Fix</span>.</td>
      </tr>
      <tr>
        <td>Unfix</td>
        <td>Unfixes selected atoms. This can also be done using <span class=
        "menu">MOE | RHS | Constrain | Free</span>.</td>
      </tr>
      <tr>
        <td>Inert</td>
        <td>Marks atoms as non-interacting. Inert atoms do not participate in
        potential calculations. For entire molecular systems, this can also be
        done in <span class="menu">MOE | RHS |
        <a href="../../apps/ligx.htm#System">System</a></span>.</td>
      </tr>
      <tr>
        <td>Active</td>
        <td>Marks atoms as interacting. This is the opposite state of inert.
        For entire molecular systems, this can also be done in <span class=
        "menu">MOE | RHS |
        <a href="../../apps/ligx.htm#System">System</a></span>.</td>
      </tr>
      <tr>
        <td>Wall</td>
        <td>
          Marks atoms as subject to <a href="../pot.htm#Wall">wall
          restraints</a>.
        </td>
      </tr>
      <tr>
        <td>Unwall</td>
        <td>
          Marks atoms as not subject to <a href="../pot.htm#Wall">wall
          restraints</a>.
        </td>
      </tr>
      <tr>
        <td>State</td>
        <td>
          Marks atoms with a particular <a href="../pot.htm#State">interaction
          state</a>.
        </td>
      </tr>
      <tr>
        <td>Restrain</td>
        <td>
          Raises the prompter for creating distance, angle or dihedral
          <a href="../pot.htm#Restraints">restraints</a>.

<img src="mainwin/restraint_prompt.png" alt=
          "Restraint Prompt" class="center" />The current value of the distance (&Aring;)
          or angle (deg) is displayed under <b>Current</b>, and the target
          minimum and maximum values can be entered at <b>Target</b>. The
          tether weight is specified in the <b>Weight</b> text field. Press
          <b>Create</b> to create the restraint. The restraints will be
          displayed as red lines in the MOE Window.
        </td>
      </tr>
    </table>
  </center>
  <div class="floatright">
    <img src="mainwin/BQ123_constraints.png" alt="Restraints" class="floatright" />

<p class="caption">Distance Restraints</p>
  </div>

<p>Wall and tether parameters can be viewed and edited in the
  <a href="../pot.htm">Potential Setup</a> panel Wall and Restraints pages. Tethers can
  be created using <span class="menu">MOE | RHS | Constrain |
  Tether</span>.</p>

<a id="Creating_and_Deleting_Meters"></a>
<h2>
<a id="Measure">Creating and Deleting Meters</a></h2>

<p>Meters are used to measure bond distances, bond angles and dihedrals of
  atoms. To create meters, use the <span class="menu">MOE | Edit |
  Measure</span> menu (or, equivalently, <span class="menu">MOE | RHS |
  Measure</span>) and choose the type of measurement to perform. In all cases,
  the <b>Measure</b> CLI prompt appears and prompts for the selection of the
  appropriate number of atoms. If atoms are already selected a meter is created
  for them right away.</p>
  <img src="mainwin/mcreate.png" alt="Meters Prompt" class="center" />
  <ul>
    <li><b>Distances</b>. Measure and display the distance between 2
    atoms.</li>
    <li><b>Angles</b>. Measure and display the angle between 3 atoms.</li>
    <li><b>Dihedrals</b>. Measure and display the dihedral angle between 4
    atoms.</li>
  </ul>
  <div class="floatleft">
    <img src="mainwin/tamiflu_meters.png" alt="Meters" class="floatleft" />

<p class="caption">Distance, Angle and Dihedral Meters</p>
  </div>

<p>If a mistake occurs and the wrong atom is selected, it can be deselected
  by simply clicking on it a second time. The prompt in the CLI will update
  automatically to reflect this change. As soon as the required number of atoms
  has been selected, a meter will appear. You can continue selecting atoms and
  creating meters as long as the prompt is up. Press <tt>&lt;Esc&gt;</tt> to
  cancel the prompt.</p>

<p>Meters are displayed in the MOE rendering area as a colored line with a
  measured value. Meters are automatically updated when atom positions are
  altered.</p>

<p>Meters can be selected by clicking on the meter line. The associated atoms
  become selected, and the meter line will be drawn in the selection color.</p>

<p>To delete meters, use the <b>Remove</b> commands from the <span class=
  "menu">MOE | Edit | Measure</span> (or, equivalently, <span class="menu">MOE
  | RHS | Measure</span>) menu. This will allow you to remove all meters or
  only distance, angle, or dihedral meters. If meters are selected, the
  deletion is restricted on those.</p>

<p><span class="tip">Tip.</span> The number of precision digits displayed for
  distance, angle, and dihedral meters can be changed with the SVL function
  <tt>ViewSetup[meter_precision:[2,1,1]]</tt>. By default, the distance meters
  show 2 digits after the comma, and 1 for angles and dihedrals meters.</p>

<a id="Interactive_Superposition"></a>
<h2>
<a id="InteractiveSuperposition">Interactive Superposition</a></h2>

<p><span class="nobr"><b>MOE | Compute | Superpose Molecules</b></span>
  superposes
  any number of molecules or, if <b>By Tag</b> is enabled, tag groups.
  When superposing by tag, all atoms in the specified tags are moved
  during the superposition.  When superposing by molecule, individual
  molecules are superposed.</p>

<p>The superposition is specified by creating
  atom &quot;associations,&quot; or <i>sets</i>,
  each of which is composed of one atom from each molecule or tag in the
  superposition. Atoms in each set are to be superposed exactly (or as
  nearly as possible) in the final optimal solid body rotations and
  translations of the molecules or tag groups.</p>

  <img src="mainwin/isuper.png" alt="Superpose Molecules Prompt" class="center" />

<p>A minimum of 3 and a maximum of 6 atom sets may be defined. The sets
  should all be different from one another, and their atoms must all
  originate from the same molecules or tags. Atom sets need not be
  filled in order, and one set need not be complete before beginning
  another: any set may be added to or otherwise modified at any time by
  first clicking on the appropriate number in the Set button bank, or
  browsing to it using the arrow buttons, to activate that set. Atoms
  already in the currently active set are shown as selected. To remove
  an atom from the set, click the left mouse button on it to toggle its
  selection state. To clear a set, press the <b>X</b> button.  When a
  set has its full complement of atoms (up to a maximum of the number of
  separate molecules or tags present and visible in the system), no more
  atoms can be added.
</p>

<img src="mainwin/isuper_settings.png"
alt="Superpose Molecules Settings" class="floatright" />

<p>In the settings popup, opened with
<img src="mainwin/settings.png" class="icon" />,
the <b>Auto-advance Set</b> feature can be enabled to
automatically advance the set number to the next available set
once a set has been completely specified.  The <b>Draw Lines</b>
option turns on the drawing of distance meters between atoms
in the same set, to help better visualize atom correspondences.
A specific molecule can be chosen to be the <b>Reference</b>.
This molecule will retain its coordinates, and the other molecules
will be superposed onto it.
</p>

<p>When all sets are of the same size, and at least three have been
  filled, superposition is possible. Click on either the <b>Apply</b> or
  <b>OK</b> buttons in the prompt area to perform the superposition.
  <b>Apply</b> will leave the prompter open, whereas <b>OK</b> will
  close the prompter once the superposition has been performed.</p>

<p><a href="../../proteins/alignsup.htm">Protein Superposition</a> is also
  available, and can be invoked from the <a href="se.htm">Sequence Editor</a>
  Homology menu.</p>

<h1><a id="ProteinAnnotation">Protein
  Annotation</a></h1>

<p>The <span class="menu">MOE | Protein | Annotate</span> menu provides
  access to the following functions which aid in highlighting structural
  topologies of the protein chains loaded in the system.</p>

  <center>
    <table class="titlebar">
      <tr>
        <th>Command</th>
        <th>Description</th>
      </tr>
      <tr>
        <td valign="top"><span class="menu">Antibody</span></td>
        <td valign="top">Launch the antibody detection method that scans the
        proteins in the current system for antibody chains. If detected, then
        the three CDR regions of V<sub>L</sub> and V<sub>H</sub> will be
        highlighted according to the one of the antibody numbering schemes
        <i>CCG, Kabat, Chothia</i> and <i>IMGT</i> that may be selected in the
        submenu. In addition to the visual assignment of the CDR regions in the
        SE, a series of dedicated residue sets are generated as well composed
        of the individual CDRs as well as the antibody framework region.
        If one of the numbering schemes is selected from the <b>Renumber</b>
        submenu, then only renumbering will be performed.</td>
      </tr>
      <tr>
        <td valign="top"><span class="menu">Auto-Annotate</span></td>
        <td valign="top">
          Each protein chain in the system is compared to a set of standard
          reference families (databases found in the <b>Project Path</b> of the
          <a href="../../appendix/settings.htm#ProteinDefaults">Protein
          Defaults</a>), which determines its membership in one or more protein
          families based on the
	  <a href="../../proteins/moeprojconfig.htm#MOE_Sequence_Alignment">
	  Z-score cutoff</a> setting in the XML configuration file of each
	  project.  HMMER is used to help estimate domain boundaries on the
          query for each matching family.  The various annotations and
          renderings that are relevant to each family will be copied to the
          Sequence Editor and highlighted.  This makes it easy to compare
          multiple proteins in the system in a topologically meaningful
          fashion. Appropriate sets of residues of interest will also be
          created, and in some cases chains, will be moved to different subunits
          to match the subunit layout of the complex in the family reference
          file.
        </td>
      </tr>
      <tr>
        <td valign="top"><span class="menu">Project</span></td>
        <td valign="top">
	    Similar to <b>Auto-Annotate</b> above, except that now
	    the comparison and annotation is with respect to the reference
	    file of a specific project, chosen from the submenu.  All
	    databases found in the <b>Project Path</b> of the <a href=
	    "../../appendix/settings.htm#ProteinDefaults">Protein Defaults</a>,
	    as well as any remote databases offered via MOE/web, will be
	    available in the submenu.
	    Annotations will be copied only to chains which are determined
	    to match the specified family based on the
	    <a href="../../proteins/moeprojconfig.htm#MOE_Sequence_Alignment">
	    Z-score cutoff (marginal)</a> setting in the XML configuration
	    of the project; this cutoff is usually less restrictive than
	    the cutoff used for <b>Auto-Annotate</b>.
        </td>
      </tr>
    </table>

    <div class="floatright">
      <img src="mainwin/vectors.png" alt="Protein Vector Annotation" class="floatright" />
      <p class="caption">Protein Vector Annotation</p>
    </div>

    <table class="titlebar">
      <tr>
        <td valign="top"><span class="menu">Dipole<br />Vector</span></td>
        <td valign="top">
          Select to show the dipole direction vector in the MOE Window as a
          blue arrow.  A separate vector will be shown for each Tag in the
	  System.  Repeated selection in the menu will toggle the vectors on
          and off.  The display will update to show the dipoles of currently
          active chains.  The vector display can also be removed by selecting
          <b>Protein Vector Annotation</b> from the <b>Cancel</b> button in
          the MOE menu bar.
        </td>
      </tr>
      <tr>
        <td valign="top"><span class="menu">Hydrophobicity<br />Vector</span></td>
        <td valign="top">
          Select to show the hydrophobicity direction vector in the MOE Window
          as a green arrow.  The hydrophobicity vector is calculated as
          described under
          <a href="../../proteins/ensprop.htm#MolProp">Protein Properties</a>.
          The hydrophobicity vector display is controlled as described above
          for the dipole vector.
        </td>
      </tr>
    </table>
  </center>

<a id="System_Priority"></a>
<h1>
<a id="Priority">System Priority</a></h1>

<p>A process priority level is a value used by the operating system to
  determine how CPU time should be allocated. Using the <span class=
  "nobr"><b>MOE | Compute | Priority</b></span> menu options you can choose to
  change the system priority. The priorities range from <b>Highest</b> to
  <b>Lowest</b>. A lower value results in less favorable scheduling and hence
  slower execution. Conversely, a higher value results in more favorable
  scheduling and, as a consequence, faster execution. The number of priority
  levels available and the ability of the user to change the priority levels
  depend on the operating system being used. On some machines, some or even all
  of the buttons found in this menu may be grayed out.</p>

<p>MOE can be completely suspended using <span class="menu">MOE | File |
  Suspend</span>.</p>
  <img src="mainwin/suspend.png" alt="Suspend panel" class="center" />

<p>Doing so will render the current MOE session inactive and return its
  tokens to the license server. MOE may be reactivated by pressing
  <b>Resume</b>, in which case the session once again will consume license
  tokens.</p>
  <ul class="help">
    <li class="help">For more information on licenses, please refer to the
    chapter on <a href="../../install/install.htm">Installation and Running
    MOE</a>.
    </li>
  </ul>

<a id="Closing_the_System"></a>
<h1>
<a id="ClosingTheSystem">Closing the
  System</a></h1>

<p>Deleting atoms can be done using the Delete button in the Edit menu. To
  delete all objects in the system (i.e. to <i>close</i> the system), use the
  <span class="menu">MOE | File | Close</span> command. This command will
  delete all molecular objects in the current system as well as all graphic
  objects. A confirmation dialog is displayed.</p>
  <img src="mainwin/close.png" alt="Close panel" class="center" />

<a id="Canceling_Tasks"></a>
<h1>
<a id="Cancel">Canceling Tasks</a></h1>

<p>The Cancel menu shows all currently running tasks. Tasks are SVL
  processes, such as commands issued from menus or the CLI as well as MOE
  windows and panels. More formally, a task is a single SVL instruction
  execution stream, together with its associated resources, such as open files
  or windows. A new task is started each time you issue a command, press a
  button that triggers an activity or select an option from a MOE menu.</p>

<p>In MOE, more than one task can run at a time. For instance, the Builder
  and the Surfaces and Maps panel can be running concurrently. Running tasks
  are listed in two places:</p>
  <ul>
    <li>Cancel menu in the menu bar (as well as in the Cancel menu of the
    Sequence Editor and Database Viewer windows).

<p>Active tasks are colored black, while those that are waiting or in a
      state of sleeping are red. You can terminate the execution of any of the
      tasks by selecting the task in the menu. This will halt its operation,
      leaving MOE in whatever state it was in at the time of cancellation.</p>
    </li>
    <li>Tasks page of the <a href="modtasks.htm">Modules &amp; Tasks panel</a>
    available from the Window menu.
    </li>
  </ul>

<p>The <b>Cancel All</b> menu item in the Cancel menu allows for all running
  tasks to be terminated at once.</p>

<p><b>Note:</b> The non-graphical version of MOE (<tt>moebatch</tt>) is not
  multi-tasking. For more information, see
  <a href="../../install/install.htm">Installing and Running MOE/batch</a>.</p>

<a id="SVL_Commands"></a>
<h1>
<a id="SVL"></a>
SVL Commands</h1>

<p><a href="../fcnref/atom.htm">Atom Fundamental Properties
  Functions</a><br />
  <a href="../fcnref/atom_d.htm">Atom Display Functions</a><br />
  <a href="../fcnref/atom_g.htm">Atom Geometry Functions</a><br />
  <a href="../fcnref/atom_p.htm">Atom Potential Functions</a><br /></p>

<a id="Tutorials">
<h1>Tutorials and Courses</h1>
</a>

  <p>All MOE courses and tutorials are available from CCG's website to all MOE
  users via the Help menu.  Selecting one of these menu items will open a new
  web browser tab displaying the current list of tutorials or courses available,
  with a brief description of each.  Simply click the one you wish to view and
  it will be downloaded and opened in another browser window or your PDF viewer.
  Any supplemental files, such as PDB files, required by the course or tutorial
  will also be included in the download.  Please note you <b>must</b> have a
  working version of MOE on your computer for the download to complete
  successfully.
  </p>
  <img src="mainwin/course_website.png" alt="Course Website" class="center" />

<a id="References">
<h1>References</h1>
</a>

<table class="ref">
  <tr>
    <td>[Labute&nbsp;2005]</td>
    <td>Labute, P.; On the Perception of Molecules from 3D Atomic
    Coordinates; <i>J. Comp. Inf. Mod. 45</i> (<b>2005</b>) 215-221.</td>
  </tr>
  </table>
<a id="See_Also"></a>
<h1>See Also</h1>

<p><a href="select.htm#AtomSelector">Atom Selector</a><br />
  <a href="mouse.htm">Using the Mouse</a><br />
  <a href="fileopen.htm">MOE Open System</a><br />
  <a href="system.htm">System Manager</a><br />
  <a href="se.htm">Sequence Editor</a><br />
  <a href="../../appendix/settings.htm">MOE Settings</a><br />

<!-- START MOE_FOOTER -->
  <div class="MOE_FOOTER">
    <img src="../../images/ccgicon.png" /> <a href="../../index.htm"></a>
    <a href="../../legal.htm"></a> &copy;<span class="versionyear"></span>
    <a href="http://www.chemcomp.com"></a>. All rights reserved.<br />
    <a href="mailto:info@chemcomp.com"></a>
  </div><!-- END MOE_FOOTER -->
</div>
</div>
</body>
</html>

